{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "19866d49-a479-4a64-afee-b97e20421ab9",
   "metadata": {},
   "source": [
    "# INSULIN\n",
    "-------------\n",
    "## Gene: INS\n",
    "  __Insulin is a protein hormone produced in the _islets of Langerhans_ located in the *beta cells* of the pancreas.__\n",
    "\n",
    "It decreases blood glucose concentration, increases cell permeability to amino acids, fatty acids, and monosaccharides, and accelerates glycogen synthesis, glycolysis, and the pentose phosphate pathway."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 141,
   "id": "ac45bf39-bd25-4c1f-8c38-a11b81cbdf18",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: biopython in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (1.85)\n",
      "Requirement already satisfied: py3Dmol in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (2.5.1)\n",
      "Requirement already satisfied: nglview in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (3.1.4)\n",
      "Requirement already satisfied: numpy in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from biopython) (1.26.4)\n",
      "Requirement already satisfied: ipywidgets>=8 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nglview) (8.1.7)\n",
      "Requirement already satisfied: notebook>=7 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nglview) (7.2.2)\n",
      "Requirement already satisfied: jupyterlab>=3 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nglview) (4.2.5)\n",
      "Requirement already satisfied: jupyterlab_widgets in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nglview) (3.0.15)\n",
      "Requirement already satisfied: comm>=0.1.3 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipywidgets>=8->nglview) (0.2.1)\n",
      "Requirement already satisfied: ipython>=6.1.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipywidgets>=8->nglview) (8.27.0)\n",
      "Requirement already satisfied: traitlets>=4.3.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipywidgets>=8->nglview) (5.14.3)\n",
      "Requirement already satisfied: widgetsnbextension~=4.0.14 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipywidgets>=8->nglview) (4.0.14)\n",
      "Requirement already satisfied: async-lru>=1.0.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (2.0.4)\n",
      "Requirement already satisfied: httpx>=0.25.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (0.27.0)\n",
      "Requirement already satisfied: ipykernel>=6.5.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (6.28.0)\n",
      "Requirement already satisfied: jinja2>=3.0.3 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (3.1.4)\n",
      "Requirement already satisfied: jupyter-core in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (5.7.2)\n",
      "Requirement already satisfied: jupyter-lsp>=2.0.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (2.2.0)\n",
      "Requirement already satisfied: jupyter-server<3,>=2.4.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (2.14.1)\n",
      "Requirement already satisfied: jupyterlab-server<3,>=2.27.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (2.27.3)\n",
      "Requirement already satisfied: notebook-shim>=0.2 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (0.2.3)\n",
      "Requirement already satisfied: packaging in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (24.1)\n",
      "Requirement already satisfied: setuptools>=40.1.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (75.1.0)\n",
      "Requirement already satisfied: tornado>=6.2.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab>=3->nglview) (6.4.1)\n",
      "Requirement already satisfied: anyio in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from httpx>=0.25.0->jupyterlab>=3->nglview) (4.2.0)\n",
      "Requirement already satisfied: certifi in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from httpx>=0.25.0->jupyterlab>=3->nglview) (2024.8.30)\n",
      "Requirement already satisfied: httpcore==1.* in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from httpx>=0.25.0->jupyterlab>=3->nglview) (1.0.2)\n",
      "Requirement already satisfied: idna in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from httpx>=0.25.0->jupyterlab>=3->nglview) (3.7)\n",
      "Requirement already satisfied: sniffio in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from httpx>=0.25.0->jupyterlab>=3->nglview) (1.3.0)\n",
      "Requirement already satisfied: h11<0.15,>=0.13 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from httpcore==1.*->httpx>=0.25.0->jupyterlab>=3->nglview) (0.14.0)\n",
      "Requirement already satisfied: debugpy>=1.6.5 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipykernel>=6.5.0->jupyterlab>=3->nglview) (1.6.7)\n",
      "Requirement already satisfied: jupyter-client>=6.1.12 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipykernel>=6.5.0->jupyterlab>=3->nglview) (8.6.0)\n",
      "Requirement already satisfied: matplotlib-inline>=0.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipykernel>=6.5.0->jupyterlab>=3->nglview) (0.1.6)\n",
      "Requirement already satisfied: nest-asyncio in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipykernel>=6.5.0->jupyterlab>=3->nglview) (1.6.0)\n",
      "Requirement already satisfied: psutil in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipykernel>=6.5.0->jupyterlab>=3->nglview) (5.9.0)\n",
      "Requirement already satisfied: pyzmq>=24 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipykernel>=6.5.0->jupyterlab>=3->nglview) (25.1.2)\n",
      "Requirement already satisfied: decorator in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipython>=6.1.0->ipywidgets>=8->nglview) (5.1.1)\n",
      "Requirement already satisfied: jedi>=0.16 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipython>=6.1.0->ipywidgets>=8->nglview) (0.19.1)\n",
      "Requirement already satisfied: prompt-toolkit<3.1.0,>=3.0.41 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipython>=6.1.0->ipywidgets>=8->nglview) (3.0.43)\n",
      "Requirement already satisfied: pygments>=2.4.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipython>=6.1.0->ipywidgets>=8->nglview) (2.15.1)\n",
      "Requirement already satisfied: stack-data in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipython>=6.1.0->ipywidgets>=8->nglview) (0.2.0)\n",
      "Requirement already satisfied: colorama in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from ipython>=6.1.0->ipywidgets>=8->nglview) (0.4.6)\n",
      "Requirement already satisfied: MarkupSafe>=2.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jinja2>=3.0.3->jupyterlab>=3->nglview) (2.1.3)\n",
      "Requirement already satisfied: platformdirs>=2.5 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-core->jupyterlab>=3->nglview) (3.10.0)\n",
      "Requirement already satisfied: pywin32>=300 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-core->jupyterlab>=3->nglview) (305.1)\n",
      "Requirement already satisfied: argon2-cffi>=21.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (21.3.0)\n",
      "Requirement already satisfied: jupyter-events>=0.9.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.10.0)\n",
      "Requirement already satisfied: jupyter-server-terminals>=0.4.4 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.4.4)\n",
      "Requirement already satisfied: nbconvert>=6.4.4 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (7.16.4)\n",
      "Requirement already satisfied: nbformat>=5.3.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (5.10.4)\n",
      "Requirement already satisfied: overrides>=5.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (7.4.0)\n",
      "Requirement already satisfied: prometheus-client>=0.9 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.14.1)\n",
      "Requirement already satisfied: pywinpty>=2.0.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (2.0.10)\n",
      "Requirement already satisfied: send2trash>=1.8.2 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (1.8.2)\n",
      "Requirement already satisfied: terminado>=0.8.3 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.17.1)\n",
      "Requirement already satisfied: websocket-client>=1.7 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (1.8.0)\n",
      "Requirement already satisfied: babel>=2.10 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (2.11.0)\n",
      "Requirement already satisfied: json5>=0.9.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (0.9.6)\n",
      "Requirement already satisfied: jsonschema>=4.18.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (4.23.0)\n",
      "Requirement already satisfied: requests>=2.31 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (2.32.3)\n",
      "Requirement already satisfied: argon2-cffi-bindings in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from argon2-cffi>=21.1->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (21.2.0)\n",
      "Requirement already satisfied: pytz>=2015.7 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from babel>=2.10->jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (2024.1)\n",
      "Requirement already satisfied: parso<0.9.0,>=0.8.3 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jedi>=0.16->ipython>=6.1.0->ipywidgets>=8->nglview) (0.8.3)\n",
      "Requirement already satisfied: attrs>=22.2.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (23.1.0)\n",
      "Requirement already satisfied: jsonschema-specifications>=2023.03.6 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (2023.7.1)\n",
      "Requirement already satisfied: referencing>=0.28.4 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (0.30.2)\n",
      "Requirement already satisfied: rpds-py>=0.7.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (0.10.6)\n",
      "Requirement already satisfied: python-dateutil>=2.8.2 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-client>=6.1.12->ipykernel>=6.5.0->jupyterlab>=3->nglview) (2.9.0.post0)\n",
      "Requirement already satisfied: python-json-logger>=2.0.4 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (2.0.7)\n",
      "Requirement already satisfied: pyyaml>=5.3 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (6.0.1)\n",
      "Requirement already satisfied: rfc3339-validator in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.1.4)\n",
      "Requirement already satisfied: rfc3986-validator>=0.1.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.1.1)\n",
      "Requirement already satisfied: beautifulsoup4 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (4.12.3)\n",
      "Requirement already satisfied: bleach!=5.0.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (4.1.0)\n",
      "Requirement already satisfied: defusedxml in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.7.1)\n",
      "Requirement already satisfied: jupyterlab-pygments in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.1.2)\n",
      "Requirement already satisfied: mistune<4,>=2.0.3 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (2.0.4)\n",
      "Requirement already satisfied: nbclient>=0.5.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.8.0)\n",
      "Requirement already satisfied: pandocfilters>=1.4.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (1.5.0)\n",
      "Requirement already satisfied: tinycss2 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (1.2.1)\n",
      "Requirement already satisfied: fastjsonschema>=2.15 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from nbformat>=5.3.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (2.16.2)\n",
      "Requirement already satisfied: wcwidth in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from prompt-toolkit<3.1.0,>=3.0.41->ipython>=6.1.0->ipywidgets>=8->nglview) (0.2.5)\n",
      "Requirement already satisfied: charset-normalizer<4,>=2 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from requests>=2.31->jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (3.3.2)\n",
      "Requirement already satisfied: urllib3<3,>=1.21.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from requests>=2.31->jupyterlab-server<3,>=2.27.1->jupyterlab>=3->nglview) (2.2.3)\n",
      "Requirement already satisfied: executing in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from stack-data->ipython>=6.1.0->ipywidgets>=8->nglview) (0.8.3)\n",
      "Requirement already satisfied: asttokens in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from stack-data->ipython>=6.1.0->ipywidgets>=8->nglview) (2.0.5)\n",
      "Requirement already satisfied: pure-eval in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from stack-data->ipython>=6.1.0->ipywidgets>=8->nglview) (0.2.2)\n",
      "Requirement already satisfied: six>=1.9.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from bleach!=5.0.0->nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (1.16.0)\n",
      "Requirement already satisfied: webencodings in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from bleach!=5.0.0->nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (0.5.1)\n",
      "Requirement already satisfied: fqdn in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (1.5.1)\n",
      "Requirement already satisfied: isoduration in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (20.11.0)\n",
      "Requirement already satisfied: jsonpointer>1.13 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (2.1)\n",
      "Requirement already satisfied: uri-template in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (1.3.0)\n",
      "Requirement already satisfied: webcolors>=24.6.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (24.11.1)\n",
      "Requirement already satisfied: cffi>=1.0.1 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from argon2-cffi-bindings->argon2-cffi>=21.1->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (1.17.1)\n",
      "Requirement already satisfied: soupsieve>1.2 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from beautifulsoup4->nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (2.5)\n",
      "Requirement already satisfied: pycparser in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from cffi>=1.0.1->argon2-cffi-bindings->argon2-cffi>=21.1->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (2.21)\n",
      "Requirement already satisfied: arrow>=0.15.0 in c:\\users\\busi4\\onedrive\\documents\\anacondaa3\\lib\\site-packages (from isoduration->jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab>=3->nglview) (1.2.3)\n"
     ]
    }
   ],
   "source": [
    "#install the required packages\n",
    "!pip install biopython py3Dmol nglview"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "id": "49eb22f1-e4ed-412c-8d21-5f23ff0c264b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Downloading PDB structure '4ins'...\n"
     ]
    },
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17539308553244367\"  style=\"position: relative; width: 1000px; height: 300px;\">\n        <p id=\"3dmolwarning_17539308553244367\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.1/build/3Dmol-min.js');\n}\n\nvar viewer_17539308553244367 = null;\nvar warn = document.getElementById(\"3dmolwarning_17539308553244367\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17539308553244367 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17539308553244367\"),{backgroundColor:\"white\"});\nviewer_17539308553244367.zoomTo();\n\tviewer_17539308553244367.addModel(\"HEADER    HORMONE                                 10-JUL-89   4INS              \\nTITLE     THE STRUCTURE OF 2ZN PIG INSULIN CRYSTALS AT 1.5 ANGSTROMS RESOLUTION \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: INSULIN (CHAIN A);                                         \\nCOMPND   3 CHAIN: A, C;                                                         \\nCOMPND   4 ENGINEERED: YES;                                                     \\nCOMPND   5 MOL_ID: 2;                                                           \\nCOMPND   6 MOLECULE: INSULIN (CHAIN B);                                         \\nCOMPND   7 CHAIN: B, D;                                                         \\nCOMPND   8 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: SUS SCROFA;                                     \\nSOURCE   3 ORGANISM_COMMON: PIG;                                                \\nSOURCE   4 ORGANISM_TAXID: 9823;                                                \\nSOURCE   5 MOL_ID: 2;                                                           \\nSOURCE   6 ORGANISM_SCIENTIFIC: SUS SCROFA;                                     \\nSOURCE   7 ORGANISM_COMMON: PIG;                                                \\nSOURCE   8 ORGANISM_TAXID: 9823                                                 \\nKEYWDS    HORMONE                                                               \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    G.G.DODSON,E.J.DODSON,D.C.HODGKIN,N.W.ISAACS,M.VIJAYAN                \\nREVDAT   8   23-OCT-24 4INS    1       LINK                                     \\nREVDAT   7   29-NOV-17 4INS    1       HELIX                                    \\nREVDAT   6   29-FEB-12 4INS    1       JRNL   VERSN                             \\nREVDAT   5   24-FEB-09 4INS    1       VERSN                                    \\nREVDAT   4   01-APR-03 4INS    1       JRNL                                     \\nREVDAT   3   31-JUL-94 4INS    3       HETATM                                   \\nREVDAT   2   15-JUL-93 4INS    1       HEADER                                   \\nREVDAT   1   15-APR-90 4INS    0                                                \\nSPRSDE     15-APR-90 4INS      1INS                                             \\nJRNL        AUTH   E.N.BAKER,T.L.BLUNDELL,J.F.CUTFIELD,S.M.CUTFIELD,E.J.DODSON, \\nJRNL        AUTH 2 G.G.DODSON,D.M.HODGKIN,R.E.HUBBARD,N.W.ISAACS,C.D.REYNOLDS,  \\nJRNL        AUTH 3 K.SAKABE,N.SAKABE,N.M.VIJAYAN                                \\nJRNL        TITL   THE STRUCTURE OF 2ZN PIG INSULIN CRYSTALS AT 1.5 A           \\nJRNL        TITL 2 RESOLUTION.                                                  \\nJRNL        REF    PHILOS.TRANS.R.SOC.LONDON,    V. 319   369 1988              \\nJRNL        REF  2 SER.B                                                        \\nJRNL        REFN                   ISSN 0080-4622                               \\nJRNL        PMID   2905485                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   J.BORDAS,G.G.DODSON,H.GREWE,M.H.J.KOCH,B.KREBS,J.RANDALL     \\nREMARK   1  TITL   A COMPARATIVE ASSESSMENT OF THE ZINC-PROTEIN COORDINATION IN \\nREMARK   1  TITL 2 2ZN-INSULIN AS DETERMINED BY X-RAY ABSORPTION FINE STRUCTURE \\nREMARK   1  TITL 3 (EXAFS) AND X-RAY CRYSTALLOGRAPHY                            \\nREMARK   1  REF    PROC.R.SOC.LONDON,SER.B       V. 219    21 1983              \\nREMARK   1  REFN                   ISSN 0080-4649                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   E.J.DODSON,G.G.DODSON,D.C.HODGKIN,C.D.REYNOLDS               \\nREMARK   1  TITL   STRUCTURAL RELATIONSHIPS IN THE TWO-ZINC INSULIN HEXAMER     \\nREMARK   1  REF    CAN.J.BIOCHEM.                V.  57   469 1979              \\nREMARK   1  REFN                   ISSN 0008-4018                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   N.W.ISAACS,R.C.AGARWAL                                       \\nREMARK   1  TITL   EXPERIENCE WITH FAST FOURIER LEAST SQUARES IN THE REFINEMENT \\nREMARK   1  TITL 2 OF THE CRYSTAL STRUCTURE OF RHOMBOHEDRAL 2-ZINC INSULIN AT   \\nREMARK   1  TITL 3 1.5 ANGSTROMS RESOLUTION                                     \\nREMARK   1  REF    ACTA CRYSTALLOGR.,SECT.A      V.  34   782 1978              \\nREMARK   1  REFN                   ISSN 0108-7673                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   G.BENTLEY,G.DODSON,A.LEWITOVA                                \\nREMARK   1  TITL   RHOMBOHEDRAL INSULIN CRYSTAL TRANSFORMATION                  \\nREMARK   1  REF    J.MOL.BIOL.                   V. 126   871 1978              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  AUTH   E.J.DODSON,N.W.ISAACS,J.S.ROLLETT                            \\nREMARK   1  TITL   A METHOD FOR FITTING SATISFACTORY MODELS TO SETS OF ATOMIC   \\nREMARK   1  TITL 2 POSITIONS IN PROTEIN STRUCTURE REFINEMENTS                   \\nREMARK   1  REF    ACTA CRYSTALLOGR.,SECT.A      V.  32   311 1976              \\nREMARK   1  REFN                   ISSN 0108-7673                               \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  AUTH   D.C.HODGKIN                                                  \\nREMARK   1  TITL   VARIETIES OF INSULIN                                         \\nREMARK   1  REF    J.ENDOCRINOL.                 V.  63     1 1974              \\nREMARK   1  REFN                   ISSN 0022-0795                               \\nREMARK   1 REFERENCE 7                                                          \\nREMARK   1  AUTH   D.C.HODGKIN                                                  \\nREMARK   1  TITL   THE STRUCTURE OF INSULIN                                     \\nREMARK   1  REF    DAN.TIDSSKR.FARM.             V.  46     1 1972              \\nREMARK   1  REFN                   ISSN 0011-6513                               \\nREMARK   1 REFERENCE 8                                                          \\nREMARK   1  AUTH   T.BLUNDELL,G.DODSON,D.HODGKIN,D.MERCOLA                      \\nREMARK   1  TITL   INSULIN. THE STRUCTURE IN THE CRYSTAL AND ITS REFLECTION IN  \\nREMARK   1  TITL 2 CHEMISTRY AND BIOLOGY                                        \\nREMARK   1  REF    ADV.PROTEIN CHEM.             V.  26   279 1972              \\nREMARK   1  REFN                   ISSN 0065-3233                               \\nREMARK   1 REFERENCE 9                                                          \\nREMARK   1  AUTH   T.L.BLUNDELL,J.F.CUTFIELD,E.J.DODSON,G.G.DODSON,D.C.HODGKIN, \\nREMARK   1  AUTH 2 D.A.MERCOLA                                                  \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF RHOMBOHEDRAL 2 ZINC INSULIN         \\nREMARK   1  REF    COLD SPRING HARBOR            V.  36   233 1972              \\nREMARK   1  REF  2 SYMP.QUANT.BIOL.                                             \\nREMARK   1  REFN                   ISSN 0091-7451                               \\nREMARK   1 REFERENCE 10                                                         \\nREMARK   1  AUTH   T.L.BLUNDELL,J.F.CUTFIELD,S.M.CUTFIELD,E.J.DODSON,           \\nREMARK   1  AUTH 2 G.G.DODSON,D.C.HODGKIN,D.A.MERCOLA,M.VIJAYAN                 \\nREMARK   1  TITL   ATOMIC POSITIONS IN RHOMBOHEDRAL 2-ZINC INSULIN CRYSTALS     \\nREMARK   1  REF    NATURE                        V. 231   506 1971              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 11                                                         \\nREMARK   1  AUTH   T.L.BLUNDELL,G.G.DODSON,E.DODSON,D.C.HODGKIN,M.VIJAYAN       \\nREMARK   1  TITL   X-RAY ANALYSIS AND THE STRUCTURE OF INSULIN                  \\nREMARK   1  REF    RECENT PROG.HORM.RES.         V.  27     1 1971              \\nREMARK   1  REFN                   ISSN 0079-9963                               \\nREMARK   1 REFERENCE 12                                                         \\nREMARK   1  AUTH   E.N.BAKER,G.DODSON                                           \\nREMARK   1  TITL   X-RAY DIFFRACTION DATA ON SOME CRYSTALLINE VARIETIES OF      \\nREMARK   1  TITL 2 INSULIN                                                      \\nREMARK   1  REF    J.MOL.BIOL.                   V.  54   605 1970              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 13                                                         \\nREMARK   1  AUTH   M.J.ADAMS,T.L.BLUNDELL,E.J.DODSON,G.G.DODSON,M.VIJAYAN,      \\nREMARK   1  AUTH 2 E.N.BAKER,M.M.HARDING,D.C.HODGKIN,B.RIMMER,S.SHEAT           \\nREMARK   1  TITL   STRUCTURE OF RHOMBOHEDRAL 2 ZINC INSULIN CRYSTALS            \\nREMARK   1  REF    NATURE                        V. 224   491 1969              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 14                                                         \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5   187 1972              \\nREMARK   1  REF  2 AND STRUCTURE (DATA SECTION)                                 \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                                                                \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.50 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PROLSQ                                               \\nREMARK   3   AUTHORS     : KONNERT,HENDRICKSON                                  \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.50                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.153                           \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT/AGREEMENT OF MODEL WITH ALL DATA.                               \\nREMARK   3   R VALUE   (WORKING + TEST SET, NO CUTOFF) : NULL                   \\nREMARK   3   R VALUE          (WORKING SET, NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE                  (NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE TEST SET SIZE (%, NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE TEST SET COUNT   (NO CUTOFF) : NULL                   \\nREMARK   3   TOTAL NUMBER OF REFLECTIONS   (NO CUTOFF) : NULL                   \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 806                                     \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 2                                       \\nREMARK   3   SOLVENT ATOMS            : 350                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   DISTANCE RESTRAINTS.                    RMS    SIGMA               \\nREMARK   3    BOND LENGTH                     (A) : 0.005 ; NULL                \\nREMARK   3    ANGLE DISTANCE                  (A) : NULL  ; NULL                \\nREMARK   3    INTRAPLANAR 1-4 DISTANCE        (A) : NULL  ; NULL                \\nREMARK   3    H-BOND OR METAL COORDINATION    (A) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   PLANE RESTRAINT                  (A) : NULL  ; NULL                \\nREMARK   3   CHIRAL-CENTER RESTRAINT       (A**3) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   NON-BONDED CONTACT RESTRAINTS.                                     \\nREMARK   3    SINGLE TORSION                  (A) : NULL  ; NULL                \\nREMARK   3    MULTIPLE TORSION                (A) : NULL  ; NULL                \\nREMARK   3    H-BOND (X...Y)                  (A) : NULL  ; NULL                \\nREMARK   3    H-BOND (X-H...Y)                (A) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   CONFORMATIONAL TORSION ANGLE RESTRAINTS.                           \\nREMARK   3    SPECIFIED                 (DEGREES) : NULL  ; NULL                \\nREMARK   3    PLANAR                    (DEGREES) : NULL  ; NULL                \\nREMARK   3    STAGGERED                 (DEGREES) : NULL  ; NULL                \\nREMARK   3    TRANSVERSE                (DEGREES) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND               (A**2) : NULL  ; NULL                \\nREMARK   3   MAIN-CHAIN ANGLE              (A**2) : NULL  ; NULL                \\nREMARK   3   SIDE-CHAIN BOND               (A**2) : NULL  ; NULL                \\nREMARK   3   SIDE-CHAIN ANGLE              (A**2) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  SOME RESIDUES ARE APPARENTLY DISORDERED BUT DIFFICULT TO            \\nREMARK   3  DESCRIBE IN TERMS OF ATOMIC POSITIONS.  ALA B 30 IS ONE OF          \\nREMARK   3  THESE RESIDUES.                                                     \\nREMARK   3                                                                      \\nREMARK   3  THE FOLLOWING RESIDUES ARE DISORDERED - GLN B 4, VAL B 12,          \\nREMARK   3  GLU B 21, ARG B 22, ARG D 22, LYS D 29.                             \\nREMARK   4                                                                      \\nREMARK   4 4INS COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179350.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 36.05                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 1.92                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: H 3                              \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z                                                \\nREMARK 290       3555   -X+Y,-X,Z                                               \\nREMARK 290       4555   X+2/3,Y+1/3,Z+1/3                                       \\nREMARK 290       5555   -Y+2/3,X-Y+1/3,Z+1/3                                    \\nREMARK 290       6555   -X+Y+2/3,-X+1/3,Z+1/3                                   \\nREMARK 290       7555   X+1/3,Y+2/3,Z+2/3                                       \\nREMARK 290       8555   -Y+1/3,X-Y+2/3,Z+2/3                                    \\nREMARK 290       9555   -X+Y+1/3,-X+2/3,Z+2/3                                   \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       41.25000            \\nREMARK 290   SMTRY2   4  0.000000  1.000000  0.000000       23.81570            \\nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000       11.33333            \\nREMARK 290   SMTRY1   5 -0.500000 -0.866025  0.000000       41.25000            \\nREMARK 290   SMTRY2   5  0.866025 -0.500000  0.000000       23.81570            \\nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000       11.33333            \\nREMARK 290   SMTRY1   6 -0.500000  0.866025  0.000000       41.25000            \\nREMARK 290   SMTRY2   6 -0.866025 -0.500000  0.000000       23.81570            \\nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       11.33333            \\nREMARK 290   SMTRY1   7  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000       47.63140            \\nREMARK 290   SMTRY3   7  0.000000  0.000000  1.000000       22.66667            \\nREMARK 290   SMTRY1   8 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   8  0.866025 -0.500000  0.000000       47.63140            \\nREMARK 290   SMTRY3   8  0.000000  0.000000  1.000000       22.66667            \\nREMARK 290   SMTRY1   9 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   9 -0.866025 -0.500000  0.000000       47.63140            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       22.66667            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2, 3, 4, 5, 6, 7                                     \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 300 REMARK: THE CRYSTALLOGRAPHIC ASYMMETRIC UNIT OF INSULIN CONSISTS OF  \\nREMARK 300 TWO INSULIN MOLECULES EACH CONSISTING OF TWO CHAINS.  THIS           \\nREMARK 300 ENTRY PRESENTS COORDINATES FOR MOLECULES I (CHAIN                    \\nREMARK 300 INDICATORS *A* AND *B*) AND II (CHAIN INDICATORS *C* AND             \\nREMARK 300 *D*).  THE QUASI-TWO-FOLD AXIS THAT TRANSFORMS MOLECULE I            \\nREMARK 300 INTO MOLECULE II IS GIVEN IN THE *MTRIX* RECORDS BELOW.              \\nREMARK 300 APPLYING THE THREE-FOLD CRYSTALLOGRAPHIC AXIS YIELDS A               \\nREMARK 300 HEXAMER AROUND THE AXIS.  THERE ARE TWO ZINC IONS SITUATED           \\nREMARK 300 ON THIS THREE-FOLD AXIS.  COORDINATES FOR THE ZINC IONS AND          \\nREMARK 300 SOME WATER MOLECULES ARE INCLUDED BELOW WITH A BLANK CHAIN           \\nREMARK 300 INDICATOR.                                                           \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 1680 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 3790 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -15.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 1740 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 3620 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -15.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: C, D                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 3                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DODECAMERIC                \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 20600 ANGSTROM**2                         \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 12080 ANGSTROM**2                       \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -260.0 KCAL/MOL                       \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B, C, D                            \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 4                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: HEXAMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 5730 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 10440 ANGSTROM**2                       \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -95.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: C, D                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 5                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: HEXAMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 5580 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 10930 ANGSTROM**2                       \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -95.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 6                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: TETRAMERIC                 \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 5120 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 5710 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -45.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: C, D                                  \\nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 7                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: TETRAMERIC                 \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 4820 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 6010 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -40.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B, C, D                            \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 375                                                                      \\nREMARK 375 SPECIAL POSITION                                                     \\nREMARK 375 THE FOLLOWING ATOMS ARE FOUND TO BE WITHIN 0.15 ANGSTROMS            \\nREMARK 375 OF A SYMMETRY RELATED ATOM AND ARE ASSUMED TO BE ON SPECIAL          \\nREMARK 375 POSITIONS.                                                           \\nREMARK 375                                                                      \\nREMARK 375 ATOM RES CSSEQI                                                      \\nREMARK 375 ZN    ZN B 101  LIES ON A SPECIAL POSITION.                          \\nREMARK 375 ZN    ZN D 101  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH B 224  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH B 245  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH D 323  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH D 403  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH D 455  LIES ON A SPECIAL POSITION.                          \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              ZN B 101  ZN                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  10   NE2                                                    \\nREMARK 620 2 HIS B  10   NE2  98.9                                              \\nREMARK 620 3 HIS B  10   NE2  98.7  98.8                                        \\nREMARK 620 4 HOH B 213   O    90.2 163.2  93.7                                  \\nREMARK 620 N                    1     2     3                                   \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              ZN D 101  ZN                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS D  10   NE2                                                    \\nREMARK 620 2 HIS D  10   NE2 103.4                                              \\nREMARK 620 3 HIS D  10   NE2 103.4 103.4                                        \\nREMARK 620 N                    1     2                                         \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: D1                                                  \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: DIMER-FORMING RESIDUES IN MOLECULE I               \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: D2                                                  \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: DIMER-FORMING RESIDUES IN MOLECULE II              \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: H1                                                  \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: HEXAMER-FORMING RESIDUES IN MOLECULE I             \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: H2                                                  \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: HEXAMER-FORMING RESIDUES IN MOLECULE II            \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: SI1                                                 \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: SURFACE-INVARIANT RESIDUES IN MOLECULE I NOT       \\nREMARK 800  INVOLVED IN DIMERIZATION                                            \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: SI2                                                 \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: SURFACE-INVARIANT RESIDUES IN MOLECULE II NOT      \\nREMARK 800  INVOLVED IN DIMERIZATION                                            \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE ZN B 31                   \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE ZN D 31                   \\nDBREF  4INS A    1    21  UNP    P01315   INS_PIG         88    108             \\nDBREF  4INS B    1    30  UNP    P01315   INS_PIG         25     54             \\nDBREF  4INS C    1    21  UNP    P01315   INS_PIG         88    108             \\nDBREF  4INS D    1    30  UNP    P01315   INS_PIG         25     54             \\nSEQRES   1 A   21  GLY ILE VAL GLU GLN CYS CYS THR SER ILE CYS SER LEU          \\nSEQRES   2 A   21  TYR GLN LEU GLU ASN TYR CYS ASN                              \\nSEQRES   1 B   30  PHE VAL ASN GLN HIS LEU CYS GLY SER HIS LEU VAL GLU          \\nSEQRES   2 B   30  ALA LEU TYR LEU VAL CYS GLY GLU ARG GLY PHE PHE TYR          \\nSEQRES   3 B   30  THR PRO LYS ALA                                              \\nSEQRES   1 C   21  GLY ILE VAL GLU GLN CYS CYS THR SER ILE CYS SER LEU          \\nSEQRES   2 C   21  TYR GLN LEU GLU ASN TYR CYS ASN                              \\nSEQRES   1 D   30  PHE VAL ASN GLN HIS LEU CYS GLY SER HIS LEU VAL GLU          \\nSEQRES   2 D   30  ALA LEU TYR LEU VAL CYS GLY GLU ARG GLY PHE PHE TYR          \\nSEQRES   3 D   30  THR PRO LYS ALA                                              \\nHET     ZN  B 101       1                                                       \\nHET     ZN  D 101       1                                                       \\nHETNAM      ZN ZINC ION                                                         \\nFORMUL   5   ZN    2(ZN 2+)                                                     \\nFORMUL   7  HOH   *350(H2 O)                                                    \\nHELIX    1 A11 GLY A    1  ILE A   10  1VAL 203 O H-BONDED TO HOH         10    \\nHELIX    2 A12 SER A   12  GLU A   17  5CNTCTS MOSTLY GT 3A,NOT IDEAL      6    \\nHELIX    3 B11 SER B    9  GLY B   20  1CYS 67 GLY 68, 3(10) CONTACTS     12    \\nHELIX    4 A21 GLY C    1  ILE C   10  1NOT IDEAL ALPH,SOME PI CNTCTS     10    \\nHELIX    5 A22 SER C   12  GLU C   17  5CNTCTS MOSTLY GT 3A,NOT IDEAL      6    \\nHELIX    6 B21 SER D    9  GLY D   20  1CYS 67,GLY 68, 3(10) CONTACTS     12    \\nSHEET    1   B 2 PHE B  24  TYR B  26  0                                        \\nSHEET    2   B 2 PHE D  24  TYR D  26 -1  N  PHE B  24   O  TYR D  26           \\nSSBOND   1 CYS A    6    CYS A   11                          1555   1555  2.05  \\nSSBOND   2 CYS A    7    CYS B    7                          1555   1555  1.97  \\nSSBOND   3 CYS A   20    CYS B   19                          1555   1555  2.00  \\nSSBOND   4 CYS C    6    CYS C   11                          1555   1555  2.06  \\nSSBOND   5 CYS C    7    CYS D    7                          1555   1555  2.01  \\nSSBOND   6 CYS C   20    CYS D   19                          1555   1555  2.02  \\nLINK         NE2 HIS B  10                ZN    ZN B 101     1555   1555  2.11  \\nLINK         NE2 HIS B  10                ZN    ZN B 101     2555   1555  2.10  \\nLINK         NE2 HIS B  10                ZN    ZN B 101     3555   1555  2.11  \\nLINK        ZN    ZN B 101                 O   HOH B 213     1555   1555  2.19  \\nLINK         NE2 HIS D  10                ZN    ZN D 101     1555   1555  2.08  \\nLINK         NE2 HIS D  10                ZN    ZN D 101     3555   1555  2.08  \\nLINK         NE2 HIS D  10                ZN    ZN D 101     2555   1555  2.08  \\nSITE     1  D1  5 VAL B  12  TYR B  16  PHE B  24  PHE B  25                    \\nSITE     2  D1  5 TYR B  26                                                     \\nSITE     1  D2  5 VAL D  12  TYR D  16  PHE D  24  PHE D  25                    \\nSITE     2  D2  5 TYR D  26                                                     \\nSITE     1  H1  7 LEU A  13  TYR A  14  PHE B   1  GLU B  13                    \\nSITE     2  H1  7 ALA B  14  LEU B  17  VAL B  18                               \\nSITE     1  H2  7 LEU C  13  TYR C  14  PHE D   1  GLU D  13                    \\nSITE     2  H2  7 ALA D  14  LEU D  17  VAL D  18                               \\nSITE     1 SI1  7 GLY A   1  GLU A   4  GLN A   5  CYS A   7                    \\nSITE     2 SI1  7 TYR A  19  ASN A  21  CYS B   7                               \\nSITE     1 SI2  7 GLY C   1  GLU C   4  GLN C   5  CYS C   7                    \\nSITE     2 SI2  7 TYR C  19  ASN C  21  CYS D   7                               \\nSITE     1 AC1  1 HIS B  10                                                     \\nSITE     1 AC2  1 HIS D  10                                                     \\nCRYST1   82.500   82.500   34.000  90.00  90.00 120.00 H 3          18          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.012121  0.006998  0.000000        0.00000                         \\nSCALE2      0.000000  0.013996  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.029412        0.00000                         \\nMTRIX1   1 -0.878620 -0.476960  0.023050        0.00000    1                    \\nMTRIX2   1 -0.477430  0.878370 -0.022860        0.00000    1                    \\nMTRIX3   1 -0.009350 -0.031090 -0.999470        0.00000    1                    \\nATOM      1  N   GLY A   1      -8.863  16.944  14.289  1.00 21.88           N  \\nATOM      2  CA  GLY A   1      -9.929  17.026  13.244  1.00 22.85           C  \\nATOM      3  C   GLY A   1     -10.051  15.625  12.618  1.00 43.92           C  \\nATOM      4  O   GLY A   1      -9.782  14.728  13.407  1.00 25.22           O  \\nATOM      5  N   ILE A   2     -10.333  15.531  11.332  1.00 26.28           N  \\nATOM      6  CA  ILE A   2     -10.488  14.266  10.600  1.00 20.84           C  \\nATOM      7  C   ILE A   2      -9.367  13.302  10.658  1.00 11.81           C  \\nATOM      8  O   ILE A   2      -9.580  12.092  10.969  1.00 20.31           O  \\nATOM      9  CB  ILE A   2     -10.883  14.493   9.095  1.00 40.00           C  \\nATOM     10  CG1 ILE A   2     -11.579  13.146   8.697  1.00 36.74           C  \\nATOM     11  CG2 ILE A   2      -9.741  14.794   8.140  1.00 23.02           C  \\nATOM     12  CD1 ILE A   2     -12.813  13.031   9.640  1.00 26.69           C  \\nATOM     13  N   VAL A   3      -8.133  13.759  10.483  1.00 16.57           N  \\nATOM     14  CA  VAL A   3      -6.966  12.901  10.576  1.00 15.75           C  \\nATOM     15  C   VAL A   3      -6.892  12.161  11.922  1.00 22.09           C  \\nATOM     16  O   VAL A   3      -6.547  10.990  12.037  1.00 24.52           O  \\nATOM     17  CB  VAL A   3      -5.697  13.708  10.225  1.00 21.34           C  \\nATOM     18  CG1 VAL A   3      -4.382  12.960  10.448  1.00 32.48           C  \\nATOM     19  CG2 VAL A   3      -5.842  14.209   8.777  1.00 26.35           C  \\nATOM     20  N   GLU A   4      -7.043  13.019  12.935  1.00 16.58           N  \\nATOM     21  CA  GLU A   4      -6.889  12.474  14.295  1.00 15.32           C  \\nATOM     22  C   GLU A   4      -8.004  11.558  14.610  1.00 16.88           C  \\nATOM     23  O   GLU A   4      -7.888  10.474  15.128  1.00 23.30           O  \\nATOM     24  CB  GLU A   4      -6.809  13.691  15.266  1.00 17.11           C  \\nATOM     25  CG  GLU A   4      -5.615  14.565  14.951  1.00 21.45           C  \\nATOM     26  CD  GLU A   4      -5.704  15.457  13.735  1.00 21.59           C  \\nATOM     27  OE1 GLU A   4      -6.757  15.959  13.377  1.00 23.43           O  \\nATOM     28  OE2 GLU A   4      -4.568  15.569  13.179  1.00 25.36           O  \\nATOM     29  N   GLN A   5      -9.199  12.048  14.356  1.00 15.69           N  \\nATOM     30  CA  GLN A   5     -10.407  11.299  14.630  1.00 12.38           C  \\nATOM     31  C   GLN A   5     -10.431   9.940  13.980  1.00 19.86           C  \\nATOM     32  O   GLN A   5     -10.815   8.931  14.542  1.00 16.83           O  \\nATOM     33  CB  GLN A   5     -11.594  12.130  14.152  1.00 21.13           C  \\nATOM     34  CG  GLN A   5     -12.860  11.374  14.561  1.00 22.06           C  \\nATOM     35  CD  GLN A   5     -13.946  11.901  13.634  1.00 42.02           C  \\nATOM     36  OE1 GLN A   5     -13.908  13.027  13.169  1.00 55.10           O  \\nATOM     37  NE2 GLN A   5     -14.943  11.030  13.351  1.00 27.27           N  \\nATOM     38  N   CYS A   6     -10.033   9.815  12.695  1.00 13.19           N  \\nATOM     39  CA  CYS A   6     -10.050   8.518  12.065  1.00 12.63           C  \\nATOM     40  C   CYS A   6      -9.105   7.520  12.667  1.00  9.95           C  \\nATOM     41  O   CYS A   6      -9.395   6.288  12.666  1.00 14.22           O  \\nATOM     42  CB  CYS A   6      -9.660   8.673  10.559  1.00 12.54           C  \\nATOM     43  SG  CYS A   6     -10.925   9.459   9.579  1.00 13.00           S  \\nATOM     44  N   CYS A   7      -8.018   7.992  13.171  1.00 10.84           N  \\nATOM     45  CA  CYS A   7      -6.964   7.186  13.808  1.00 17.02           C  \\nATOM     46  C   CYS A   7      -7.236   6.948  15.358  1.00 13.71           C  \\nATOM     47  O   CYS A   7      -7.061   5.782  15.768  1.00 19.28           O  \\nATOM     48  CB  CYS A   7      -5.578   7.826  13.656  1.00 20.24           C  \\nATOM     49  SG  CYS A   7      -4.181   6.819  14.134  1.00 13.80           S  \\nATOM     50  N   THR A   8      -7.655   7.937  16.058  1.00 12.57           N  \\nATOM     51  CA  THR A   8      -7.862   7.732  17.520  1.00 19.99           C  \\nATOM     52  C   THR A   8      -9.143   6.997  17.870  1.00 26.34           C  \\nATOM     53  O   THR A   8      -9.189   6.157  18.795  1.00 25.43           O  \\nATOM     54  CB  THR A   8      -7.728   9.055  18.386  1.00 20.77           C  \\nATOM     55  OG1 THR A   8      -8.889   9.918  18.117  1.00 26.76           O  \\nATOM     56  CG2 THR A   8      -6.334   9.700  18.196  1.00 26.50           C  \\nATOM     57  N   SER A   9     -10.170   7.350  17.058  1.00 20.01           N  \\nATOM     58  CA  SER A   9     -11.509   6.803  17.121  1.00 16.88           C  \\nATOM     59  C   SER A   9     -11.796   5.981  15.856  1.00 12.70           C  \\nATOM     60  O   SER A   9     -11.139   5.010  15.473  1.00 17.60           O  \\nATOM     61  CB  SER A   9     -12.331   8.067  17.439  1.00 19.52           C  \\nATOM     62  OG  SER A   9     -13.674   7.774  17.650  1.00 32.34           O  \\nATOM     63  N   ILE A  10     -12.883   6.382  15.159  1.00 15.34           N  \\nATOM     64  CA  ILE A  10     -13.350   5.723  13.932  1.00 20.23           C  \\nATOM     65  C   ILE A  10     -13.969   6.902  13.106  1.00 17.50           C  \\nATOM     66  O   ILE A  10     -14.355   7.922  13.623  1.00 16.60           O  \\nATOM     67  CB  ILE A  10     -14.366   4.524  14.047  1.00 19.39           C  \\nATOM     68  CG1 ILE A  10     -15.702   4.874  14.742  1.00 22.05           C  \\nATOM     69  CG2 ILE A  10     -13.711   3.300  14.723  1.00 23.30           C  \\nATOM     70  CD1 ILE A  10     -16.702   3.722  15.005  1.00 42.11           C  \\nATOM     71  N   CYS A  11     -14.080   6.685  11.767  1.00 12.14           N  \\nATOM     72  CA  CYS A  11     -14.665   7.679  10.880  1.00 11.24           C  \\nATOM     73  C   CYS A  11     -15.301   6.881   9.766  1.00 12.17           C  \\nATOM     74  O   CYS A  11     -14.962   5.692   9.528  1.00 21.14           O  \\nATOM     75  CB  CYS A  11     -13.695   8.702  10.417  1.00 13.03           C  \\nATOM     76  SG  CYS A  11     -12.375   8.019   9.385  1.00 13.60           S  \\nATOM     77  N   SER A  12     -16.233   7.557   9.095  1.00 11.37           N  \\nATOM     78  CA  SER A  12     -16.999   6.978   8.005  1.00  9.91           C  \\nATOM     79  C   SER A  12     -16.563   7.644   6.726  1.00  7.40           C  \\nATOM     80  O   SER A  12     -15.967   8.753   6.711  1.00  9.67           O  \\nATOM     81  CB  SER A  12     -18.516   7.183   8.084  1.00 16.64           C  \\nATOM     82  OG  SER A  12     -18.869   8.543   7.881  1.00 17.14           O  \\nATOM     83  N   LEU A  13     -16.852   6.914   5.612  1.00 11.35           N  \\nATOM     84  CA  LEU A  13     -16.530   7.444   4.259  1.00 11.35           C  \\nATOM     85  C   LEU A  13     -17.317   8.715   4.030  1.00 12.55           C  \\nATOM     86  O   LEU A  13     -16.835   9.521   3.226  1.00 11.78           O  \\nATOM     87  CB  LEU A  13     -16.774   6.348   3.232  1.00 11.66           C  \\nATOM     88  CG  LEU A  13     -15.940   5.046   3.316  1.00 18.12           C  \\nATOM     89  CD1 LEU A  13     -16.050   4.197   2.018  1.00 18.76           C  \\nATOM     90  CD2 LEU A  13     -14.471   5.320   3.537  1.00 17.26           C  \\nATOM     91  N   TYR A  14     -18.491   8.790   4.629  1.00 10.84           N  \\nATOM     92  CA  TYR A  14     -19.282  10.035   4.368  1.00 10.75           C  \\nATOM     93  C   TYR A  14     -18.639  11.228   4.963  1.00 12.81           C  \\nATOM     94  O   TYR A  14     -18.706  12.298   4.341  1.00 15.11           O  \\nATOM     95  CB  TYR A  14     -20.746   9.900   4.799  1.00 12.90           C  \\nATOM     96  CG  TYR A  14     -21.463   8.764   4.079  1.00 18.23           C  \\nATOM     97  CD1 TYR A  14     -22.110   9.123   2.891  1.00 18.95           C  \\nATOM     98  CD2 TYR A  14     -21.461   7.440   4.475  1.00 15.42           C  \\nATOM     99  CE1 TYR A  14     -22.767   8.167   2.086  1.00 18.15           C  \\nATOM    100  CE2 TYR A  14     -22.118   6.436   3.676  1.00 14.31           C  \\nATOM    101  CZ  TYR A  14     -22.738   6.856   2.556  1.00 15.47           C  \\nATOM    102  OH  TYR A  14     -23.436   5.926   1.716  1.00 24.86           O  \\nATOM    103  N   GLN A  15     -17.945  11.100   6.091  1.00  9.63           N  \\nATOM    104  CA  GLN A  15     -17.178  12.138   6.774  1.00  9.40           C  \\nATOM    105  C   GLN A  15     -16.012  12.543   5.900  1.00 10.52           C  \\nATOM    106  O   GLN A  15     -15.611  13.717   5.722  1.00 14.25           O  \\nATOM    107  CB  GLN A  15     -16.774  11.841   8.205  1.00 13.89           C  \\nATOM    108  CG  GLN A  15     -17.894  11.668   9.206  1.00 17.53           C  \\nATOM    109  CD  GLN A  15     -17.524  11.056  10.515  1.00 28.21           C  \\nATOM    110  OE1 GLN A  15     -16.865  10.027  10.598  1.00 20.14           O  \\nATOM    111  NE2 GLN A  15     -17.994  11.650  11.624  1.00 30.25           N  \\nATOM    112  N   LEU A  16     -15.352  11.525   5.325  1.00 12.99           N  \\nATOM    113  CA  LEU A  16     -14.185  11.826   4.470  1.00 11.19           C  \\nATOM    114  C   LEU A  16     -14.605  12.634   3.249  1.00 15.54           C  \\nATOM    115  O   LEU A  16     -13.767  13.398   2.745  1.00 16.01           O  \\nATOM    116  CB  LEU A  16     -13.588  10.521   4.060  1.00 12.67           C  \\nATOM    117  CG  LEU A  16     -12.954   9.717   5.182  1.00 13.07           C  \\nATOM    118  CD1 LEU A  16     -12.115   8.571   4.602  1.00 16.61           C  \\nATOM    119  CD2 LEU A  16     -12.041  10.559   6.028  1.00 16.50           C  \\nATOM    120  N   GLU A  17     -15.779  12.420   2.759  1.00 17.50           N  \\nATOM    121  CA  GLU A  17     -16.223  13.179   1.589  1.00 17.72           C  \\nATOM    122  C   GLU A  17     -16.171  14.693   1.811  1.00 19.21           C  \\nATOM    123  O   GLU A  17     -16.118  15.466   0.803  1.00 18.48           O  \\nATOM    124  CB  GLU A  17     -17.645  12.862   1.215  1.00 17.38           C  \\nATOM    125  CG  GLU A  17     -17.885  11.629   0.360  1.00 27.97           C  \\nATOM    126  CD  GLU A  17     -19.225  11.667  -0.391  1.00 26.70           C  \\nATOM    127  OE1 GLU A  17     -20.201  11.466   0.276  1.00 29.93           O  \\nATOM    128  OE2 GLU A  17     -19.127  11.873  -1.643  1.00 34.66           O  \\nATOM    129  N   ASN A  18     -16.094  15.074   3.104  1.00 15.10           N  \\nATOM    130  CA  ASN A  18     -16.029  16.534   3.332  1.00 18.85           C  \\nATOM    131  C   ASN A  18     -14.703  17.131   2.954  1.00 18.46           C  \\nATOM    132  O   ASN A  18     -14.545  18.377   2.834  1.00 19.68           O  \\nATOM    133  CB  ASN A  18     -16.489  16.934   4.738  1.00 20.66           C  \\nATOM    134  CG  ASN A  18     -17.868  16.338   5.142  1.00 29.79           C  \\nATOM    135  OD1 ASN A  18     -18.813  16.053   4.382  1.00 34.48           O  \\nATOM    136  ND2 ASN A  18     -17.991  16.168   6.452  1.00 36.00           N  \\nATOM    137  N   TYR A  19     -13.697  16.327   2.738  1.00 15.68           N  \\nATOM    138  CA  TYR A  19     -12.358  16.724   2.380  1.00 14.19           C  \\nATOM    139  C   TYR A  19     -12.154  16.695   0.899  1.00 13.20           C  \\nATOM    140  O   TYR A  19     -11.010  17.038   0.480  1.00 16.12           O  \\nATOM    141  CB  TYR A  19     -11.364  15.840   3.178  1.00 14.35           C  \\nATOM    142  CG  TYR A  19     -11.586  16.223   4.634  1.00 21.24           C  \\nATOM    143  CD1 TYR A  19     -10.853  17.300   5.129  1.00 26.61           C  \\nATOM    144  CD2 TYR A  19     -12.562  15.703   5.445  1.00 19.21           C  \\nATOM    145  CE1 TYR A  19     -11.084  17.801   6.393  1.00 27.80           C  \\nATOM    146  CE2 TYR A  19     -12.833  16.207   6.714  1.00 23.98           C  \\nATOM    147  CZ  TYR A  19     -12.081  17.267   7.187  1.00 34.08           C  \\nATOM    148  OH  TYR A  19     -12.227  17.849   8.400  1.00 37.96           O  \\nATOM    149  N   CYS A  20     -13.057  16.313   0.077  1.00 13.05           N  \\nATOM    150  CA  CYS A  20     -12.838  16.309  -1.389  1.00 18.69           C  \\nATOM    151  C   CYS A  20     -12.984  17.799  -1.802  1.00 19.09           C  \\nATOM    152  O   CYS A  20     -13.588  18.579  -1.084  1.00 19.31           O  \\nATOM    153  CB  CYS A  20     -13.850  15.490  -2.157  1.00 15.99           C  \\nATOM    154  SG  CYS A  20     -13.923  13.761  -1.584  1.00 12.90           S  \\nATOM    155  N   ASN A  21     -12.380  18.063  -2.909  1.00 17.63           N  \\nATOM    156  CA  ASN A  21     -12.404  19.399  -3.608  1.00 25.23           C  \\nATOM    157  C   ASN A  21     -13.642  19.696  -4.447  1.00 34.82           C  \\nATOM    158  O   ASN A  21     -14.146  18.703  -4.956  1.00 31.24           O  \\nATOM    159  CB  ASN A  21     -11.228  19.392  -4.521  1.00 19.06           C  \\nATOM    160  CG  ASN A  21     -10.020  20.283  -4.456  1.00 40.71           C  \\nATOM    161  OD1 ASN A  21     -10.067  21.380  -5.083  1.00 68.22           O  \\nATOM    162  ND2 ASN A  21      -9.004  19.667  -3.808  1.00 39.69           N  \\nATOM    163  OXT ASN A  21     -13.881  20.890  -4.604  1.00 41.83           O  \\nTER     164      ASN A  21                                                      \\nATOM    165  N   PHE B   1     -21.768   1.132   3.577  1.00 25.87           N  \\nATOM    166  CA  PHE B   1     -20.374   1.368   4.053  1.00 24.30           C  \\nATOM    167  C   PHE B   1     -20.341   1.145   5.585  1.00 39.74           C  \\nATOM    168  O   PHE B   1     -21.423   1.141   6.173  1.00 38.10           O  \\nATOM    169  CB  PHE B   1     -19.806   2.718   3.624  1.00 22.51           C  \\nATOM    170  CG  PHE B   1     -19.924   2.916   2.131  1.00 16.52           C  \\nATOM    171  CD1 PHE B   1     -20.067   4.204   1.618  1.00 35.58           C  \\nATOM    172  CD2 PHE B   1     -19.709   1.873   1.262  1.00 20.86           C  \\nATOM    173  CE1 PHE B   1     -20.093   4.444   0.243  1.00 52.66           C  \\nATOM    174  CE2 PHE B   1     -19.824   2.067  -0.123  1.00 51.46           C  \\nATOM    175  CZ  PHE B   1     -20.011   3.332  -0.631  1.00 42.63           C  \\nATOM    176  N   VAL B   2     -19.104   0.899   6.027  1.00 21.12           N  \\nATOM    177  CA  VAL B   2     -18.754   0.598   7.406  1.00 36.74           C  \\nATOM    178  C   VAL B   2     -17.780   1.656   7.965  1.00 23.52           C  \\nATOM    179  O   VAL B   2     -17.104   2.328   7.197  1.00 19.56           O  \\nATOM    180  CB  VAL B   2     -18.048  -0.765   7.638  1.00 30.58           C  \\nATOM    181  CG1 VAL B   2     -18.993  -1.953   7.609  1.00 25.73           C  \\nATOM    182  CG2 VAL B   2     -16.776  -0.916   6.799  1.00 22.31           C  \\nATOM    183  N   ASN B   3     -17.741   1.753   9.278  1.00 13.38           N  \\nATOM    184  CA  ASN B   3     -16.872   2.691   9.950  1.00 13.94           C  \\nATOM    185  C   ASN B   3     -15.457   2.100   9.881  1.00 15.03           C  \\nATOM    186  O   ASN B   3     -15.312   0.857   9.926  1.00 24.85           O  \\nATOM    187  CB  ASN B   3     -17.272   3.010  11.382  1.00 25.01           C  \\nATOM    188  CG  ASN B   3     -18.513   3.844  11.511  1.00 49.04           C  \\nATOM    189  OD1 ASN B   3     -18.658   4.774  10.724  1.00 34.50           O  \\nATOM    190  ND2 ASN B   3     -19.333   3.415  12.473  1.00 35.00           N  \\nATOM    191  N   GLN B   4     -14.509   3.031   9.767  1.00 12.52           N  \\nATOM    192  CA  GLN B   4     -13.137   2.542   9.571  1.00 22.69           C  \\nATOM    193  C   GLN B   4     -12.213   3.224  10.580  1.00 13.29           C  \\nATOM    194  O   GLN B   4     -12.347   4.333  11.118  1.00 20.53           O  \\nATOM    195  CB  GLN B   4     -12.666   2.760   8.116  1.00 39.18           C  \\nATOM    196  CG AGLN B   4     -13.007   1.731   7.035  0.60 11.45           C  \\nATOM    197  CG BGLN B   4     -12.978   1.763   6.996  0.40 37.30           C  \\nATOM    198  CD AGLN B   4     -12.270   0.520   6.830  0.60 12.42           C  \\nATOM    199  CD BGLN B   4     -14.070   2.781   6.746  0.40 32.97           C  \\nATOM    200  OE1AGLN B   4     -12.812  -0.612   6.494  0.60 17.67           O  \\nATOM    201  OE1BGLN B   4     -14.059   3.957   7.112  0.40 40.00           O  \\nATOM    202  NE2AGLN B   4     -10.898   0.624   6.949  0.60 28.94           N  \\nATOM    203  NE2BGLN B   4     -15.108   2.179   6.165  0.40 35.67           N  \\nATOM    204  N   HIS B   5     -11.158   2.442  10.837  1.00 13.02           N  \\nATOM    205  CA  HIS B   5     -10.083   3.000  11.779  1.00 17.05           C  \\nATOM    206  C   HIS B   5      -8.855   3.149  10.899  1.00 10.95           C  \\nATOM    207  O   HIS B   5      -8.284   2.166  10.380  1.00 17.14           O  \\nATOM    208  CB  HIS B   5      -9.982   1.956  12.877  1.00 22.24           C  \\nATOM    209  CG  HIS B   5      -8.934   2.400  13.860  1.00 25.74           C  \\nATOM    210  ND1 HIS B   5      -8.072   1.535  14.436  1.00 35.32           N  \\nATOM    211  CD2 HIS B   5      -8.637   3.596  14.329  1.00 28.02           C  \\nATOM    212  CE1 HIS B   5      -7.275   2.240  15.211  1.00 28.73           C  \\nATOM    213  NE2 HIS B   5      -7.571   3.509  15.150  1.00 30.21           N  \\nATOM    214  N   LEU B   6      -8.529   4.400  10.604  1.00 11.30           N  \\nATOM    215  CA  LEU B   6      -7.468   4.709   9.611  1.00 11.13           C  \\nATOM    216  C   LEU B   6      -6.399   5.604  10.158  1.00 11.03           C  \\nATOM    217  O   LEU B   6      -6.695   6.779  10.484  1.00 13.66           O  \\nATOM    218  CB  LEU B   6      -8.231   5.398   8.411  1.00 14.13           C  \\nATOM    219  CG  LEU B   6      -9.251   4.634   7.563  1.00 13.39           C  \\nATOM    220  CD1 LEU B   6     -10.017   5.598   6.671  1.00 14.70           C  \\nATOM    221  CD2 LEU B   6      -8.620   3.517   6.767  1.00 18.25           C  \\nATOM    222  N   CYS B   7      -5.180   5.069  10.115  1.00 10.06           N  \\nATOM    223  CA  CYS B   7      -4.058   5.835  10.569  1.00 10.70           C  \\nATOM    224  C   CYS B   7      -3.033   5.982   9.484  1.00 13.26           C  \\nATOM    225  O   CYS B   7      -2.955   5.198   8.573  1.00 19.10           O  \\nATOM    226  CB  CYS B   7      -3.434   5.105  11.762  1.00 15.88           C  \\nATOM    227  SG  CYS B   7      -4.523   5.099  13.246  1.00 16.40           S  \\nATOM    228  N   GLY B   8      -2.181   6.993   9.540  1.00 12.37           N  \\nATOM    229  CA  GLY B   8      -1.070   7.261   8.632  1.00 12.72           C  \\nATOM    230  C   GLY B   8      -1.465   7.317   7.204  1.00 13.24           C  \\nATOM    231  O   GLY B   8      -2.470   7.884   6.744  1.00 11.92           O  \\nATOM    232  N   SER B   9      -0.609   6.582   6.429  1.00 11.74           N  \\nATOM    233  CA  SER B   9      -0.863   6.544   4.980  1.00 15.89           C  \\nATOM    234  C   SER B   9      -2.183   5.870   4.578  1.00  9.73           C  \\nATOM    235  O   SER B   9      -2.649   6.111   3.528  1.00 10.43           O  \\nATOM    236  CB  SER B   9       0.309   5.921   4.206  1.00 17.74           C  \\nATOM    237  OG  SER B   9       0.534   4.626   4.735  1.00 17.37           O  \\nATOM    238  N   HIS B  10      -2.721   5.100   5.451  1.00 10.19           N  \\nATOM    239  CA  HIS B  10      -3.940   4.379   5.188  1.00  7.66           C  \\nATOM    240  C   HIS B  10      -5.081   5.431   5.075  1.00 10.17           C  \\nATOM    241  O   HIS B  10      -6.021   5.163   4.291  1.00 10.92           O  \\nATOM    242  CB  HIS B  10      -4.234   3.316   6.228  1.00  9.55           C  \\nATOM    243  CG  HIS B  10      -3.192   2.269   6.364  1.00  9.55           C  \\nATOM    244  ND1 HIS B  10      -3.043   1.310   5.423  1.00 15.86           N  \\nATOM    245  CD2 HIS B  10      -2.289   1.991   7.311  1.00  8.47           C  \\nATOM    246  CE1 HIS B  10      -2.078   0.573   5.774  1.00 10.65           C  \\nATOM    247  NE2 HIS B  10      -1.589   0.939   6.878  1.00  9.41           N  \\nATOM    248  N   LEU B  11      -5.016   6.497   5.810  1.00  8.93           N  \\nATOM    249  CA  LEU B  11      -6.071   7.518   5.617  1.00  9.64           C  \\nATOM    250  C   LEU B  11      -5.967   8.182   4.279  1.00  7.89           C  \\nATOM    251  O   LEU B  11      -6.969   8.462   3.666  1.00  9.74           O  \\nATOM    252  CB  LEU B  11      -5.860   8.541   6.740  1.00  6.93           C  \\nATOM    253  CG  LEU B  11      -6.949   9.607   6.783  1.00 14.50           C  \\nATOM    254  CD1 LEU B  11      -8.376   9.229   6.627  1.00 18.34           C  \\nATOM    255  CD2 LEU B  11      -6.742  10.309   8.115  1.00 20.70           C  \\nATOM    256  N   VAL B  12      -4.751   8.449   3.799  1.00 10.12           N  \\nATOM    257  CA  VAL B  12      -4.579   9.057   2.495  1.00  8.05           C  \\nATOM    258  C   VAL B  12      -5.050   8.131   1.372  1.00  8.14           C  \\nATOM    259  O   VAL B  12      -5.595   8.653   0.398  1.00 11.63           O  \\nATOM    260  CB  VAL B  12      -3.153   9.538   2.230  1.00 11.54           C  \\nATOM    261  CG1AVAL B  12      -2.822   9.786   0.799  0.50  4.68           C  \\nATOM    262  CG1BVAL B  12      -2.809  10.670   3.148  0.50 17.75           C  \\nATOM    263  CG2AVAL B  12      -2.809  10.670   3.148  0.50 17.75           C  \\nATOM    264  CG2BVAL B  12      -1.963   8.655   2.123  0.50 10.87           C  \\nATOM    265  N   GLU B  13      -4.906   6.880   1.502  1.00  6.12           N  \\nATOM    266  CA  GLU B  13      -5.432   5.946   0.542  1.00  8.88           C  \\nATOM    267  C   GLU B  13      -6.966   6.014   0.472  1.00 12.22           C  \\nATOM    268  O   GLU B  13      -7.578   6.035  -0.614  1.00 11.15           O  \\nATOM    269  CB  GLU B  13      -4.996   4.506   0.854  1.00 12.65           C  \\nATOM    270  CG  GLU B  13      -3.497   4.444   0.582  1.00 15.60           C  \\nATOM    271  CD  GLU B  13      -3.246   3.857  -0.794  1.00 53.85           C  \\nATOM    272  OE1 GLU B  13      -4.238   3.643  -1.500  1.00 33.68           O  \\nATOM    273  OE2 GLU B  13      -2.114   3.611  -1.126  1.00 47.24           O  \\nATOM    274  N   ALA B  14      -7.659   6.004   1.637  1.00  7.15           N  \\nATOM    275  CA  ALA B  14      -9.061   6.164   1.719  1.00  7.29           C  \\nATOM    276  C   ALA B  14      -9.563   7.482   1.051  1.00  6.80           C  \\nATOM    277  O   ALA B  14     -10.595   7.468   0.346  1.00 11.10           O  \\nATOM    278  CB  ALA B  14      -9.604   6.039   3.106  1.00 12.06           C  \\nATOM    279  N   LEU B  15      -8.876   8.580   1.321  1.00  6.72           N  \\nATOM    280  CA  LEU B  15      -9.224   9.854   0.717  1.00 13.51           C  \\nATOM    281  C   LEU B  15      -9.111   9.815  -0.829  1.00 14.62           C  \\nATOM    282  O   LEU B  15      -9.956  10.390  -1.496  1.00 12.32           O  \\nATOM    283  CB  LEU B  15      -8.317  10.981   1.327  1.00  9.71           C  \\nATOM    284  CG  LEU B  15      -8.755  11.581   2.649  1.00  8.92           C  \\nATOM    285  CD1 LEU B  15      -7.682  12.475   3.236  1.00 14.49           C  \\nATOM    286  CD2 LEU B  15     -10.096  12.235   2.460  1.00 12.03           C  \\nATOM    287  N   TYR B  16      -8.050   9.147  -1.297  1.00  8.65           N  \\nATOM    288  CA  TYR B  16      -7.838   8.961  -2.686  1.00  8.75           C  \\nATOM    289  C   TYR B  16      -8.999   8.175  -3.284  1.00 11.14           C  \\nATOM    290  O   TYR B  16      -9.508   8.504  -4.371  1.00 14.34           O  \\nATOM    291  CB  TYR B  16      -6.494   8.247  -3.047  1.00  7.72           C  \\nATOM    292  CG  TYR B  16      -6.271   8.027  -4.522  1.00 10.81           C  \\nATOM    293  CD1 TYR B  16      -6.450   6.784  -5.047  1.00 17.09           C  \\nATOM    294  CD2 TYR B  16      -6.009   9.104  -5.338  1.00 12.64           C  \\nATOM    295  CE1 TYR B  16      -6.354   6.581  -6.467  1.00 17.76           C  \\nATOM    296  CE2 TYR B  16      -5.898   8.958  -6.741  1.00 13.94           C  \\nATOM    297  CZ  TYR B  16      -6.110   7.692  -7.259  1.00 17.34           C  \\nATOM    298  OH  TYR B  16      -5.925   7.520  -8.594  1.00 25.34           O  \\nATOM    299  N   LEU B  17      -9.428   7.109  -2.664  1.00  8.68           N  \\nATOM    300  CA  LEU B  17     -10.566   6.290  -3.167  1.00  8.83           C  \\nATOM    301  C   LEU B  17     -11.861   7.087  -3.142  1.00 10.95           C  \\nATOM    302  O   LEU B  17     -12.650   7.046  -4.073  1.00 15.67           O  \\nATOM    303  CB  LEU B  17     -10.665   5.052  -2.327  1.00 10.82           C  \\nATOM    304  CG  LEU B  17      -9.594   4.104  -2.924  1.00 28.76           C  \\nATOM    305  CD1 LEU B  17      -9.136   3.067  -1.933  1.00 30.52           C  \\nATOM    306  CD2 LEU B  17     -10.280   3.540  -4.157  1.00 34.91           C  \\nATOM    307  N   VAL B  18     -12.123   7.786  -2.036  1.00  8.85           N  \\nATOM    308  CA  VAL B  18     -13.351   8.545  -1.933  1.00  8.77           C  \\nATOM    309  C   VAL B  18     -13.433   9.713  -2.873  1.00  9.77           C  \\nATOM    310  O   VAL B  18     -14.472   9.937  -3.457  1.00 16.86           O  \\nATOM    311  CB  VAL B  18     -13.604   8.974  -0.463  1.00 14.39           C  \\nATOM    312  CG1 VAL B  18     -14.784   9.899  -0.282  1.00 11.72           C  \\nATOM    313  CG2 VAL B  18     -13.862   7.763   0.393  1.00 12.58           C  \\nATOM    314  N   CYS B  19     -12.422  10.518  -2.958  1.00  9.03           N  \\nATOM    315  CA  CYS B  19     -12.433  11.758  -3.756  1.00  8.88           C  \\nATOM    316  C   CYS B  19     -11.994  11.555  -5.212  1.00 14.69           C  \\nATOM    317  O   CYS B  19     -12.410  12.303  -6.126  1.00 16.46           O  \\nATOM    318  CB  CYS B  19     -11.558  12.719  -3.005  1.00 11.19           C  \\nATOM    319  SG  CYS B  19     -12.040  13.127  -1.344  1.00 10.10           S  \\nATOM    320  N   GLY B  20     -11.149  10.609  -5.463  1.00 17.12           N  \\nATOM    321  CA  GLY B  20     -10.685  10.359  -6.851  1.00 21.59           C  \\nATOM    322  C   GLY B  20     -10.275  11.650  -7.524  1.00 21.38           C  \\nATOM    323  O   GLY B  20      -9.494  12.483  -7.053  1.00 20.41           O  \\nATOM    324  N   GLU B  21     -10.784  11.844  -8.710  1.00 29.76           N  \\nATOM    325  CA  GLU B  21     -10.398  13.043  -9.501  1.00 24.44           C  \\nATOM    326  C   GLU B  21     -10.898  14.356  -9.065  1.00 19.21           C  \\nATOM    327  O   GLU B  21     -10.430  15.331  -9.665  1.00 19.00           O  \\nATOM    328  CB  GLU B  21     -10.776  12.724 -10.968  1.00 28.66           C  \\nATOM    329  CG AGLU B  21     -12.310  12.519 -11.045  0.50 56.11           C  \\nATOM    330  CG BGLU B  21      -9.804  13.415 -11.966  0.50 75.35           C  \\nATOM    331  CD AGLU B  21     -12.707  11.349 -11.901  0.50 62.47           C  \\nATOM    332  CD BGLU B  21      -9.689  13.292 -13.466  0.50 52.68           C  \\nATOM    333  OE1AGLU B  21     -12.515  10.193 -11.564  0.50 48.34           O  \\nATOM    334  OE1BGLU B  21     -10.540  12.768 -14.159  0.50 50.28           O  \\nATOM    335  OE2AGLU B  21     -13.225  11.772 -12.958  0.50 49.92           O  \\nATOM    336  OE2BGLU B  21      -8.505  13.537 -13.781  0.50 27.33           O  \\nATOM    337  N   ARG B  22     -11.703  14.491  -8.034  1.00 15.49           N  \\nATOM    338  CA  ARG B  22     -12.089  15.732  -7.456  1.00 16.44           C  \\nATOM    339  C   ARG B  22     -10.797  16.222  -6.745  1.00 17.70           C  \\nATOM    340  O   ARG B  22     -10.636  17.458  -6.608  1.00 21.36           O  \\nATOM    341  CB  ARG B  22     -13.234  15.678  -6.464  1.00 21.99           C  \\nATOM    342  CG  ARG B  22     -14.645  15.427  -7.037  1.00 57.89           C  \\nATOM    343  CD  ARG B  22     -15.675  15.167  -5.960  1.00 31.23           C  \\nATOM    344  NE AARG B  22     -15.739  16.404  -5.124  0.50 16.46           N  \\nATOM    345  NE BARG B  22     -15.629  13.808  -5.271  0.50 17.69           N  \\nATOM    346  CZ AARG B  22     -16.608  16.581  -4.143  0.50 39.57           C  \\nATOM    347  CZ BARG B  22     -16.379  13.225  -4.283  0.50 33.09           C  \\nATOM    348  NH1AARG B  22     -16.743  17.672  -3.416  0.50 20.14           N  \\nATOM    349  NH1BARG B  22     -16.987  14.046  -3.392  0.50 51.50           N  \\nATOM    350  NH2AARG B  22     -17.405  15.551  -3.871  0.50 35.40           N  \\nATOM    351  NH2BARG B  22     -16.705  11.943  -4.184  0.50 19.59           N  \\nATOM    352  N   GLY B  23     -10.007  15.246  -6.287  1.00 19.18           N  \\nATOM    353  CA  GLY B  23      -8.844  15.673  -5.491  1.00 11.89           C  \\nATOM    354  C   GLY B  23      -9.339  15.932  -4.075  1.00 12.83           C  \\nATOM    355  O   GLY B  23     -10.524  15.922  -3.626  1.00 14.47           O  \\nATOM    356  N   PHE B  24      -8.343  16.165  -3.187  1.00 12.54           N  \\nATOM    357  CA  PHE B  24      -8.584  16.432  -1.765  1.00 10.08           C  \\nATOM    358  C   PHE B  24      -7.488  17.220  -1.123  1.00 10.77           C  \\nATOM    359  O   PHE B  24      -6.411  17.409  -1.657  1.00 10.93           O  \\nATOM    360  CB  PHE B  24      -8.754  15.111  -1.032  1.00  3.80           C  \\nATOM    361  CG  PHE B  24      -7.638  14.114  -1.034  1.00  5.98           C  \\nATOM    362  CD1 PHE B  24      -7.488  13.202  -2.069  1.00  5.61           C  \\nATOM    363  CD2 PHE B  24      -6.667  14.205  -0.036  1.00  8.93           C  \\nATOM    364  CE1 PHE B  24      -6.375  12.338  -2.106  1.00 14.64           C  \\nATOM    365  CE2 PHE B  24      -5.573  13.387   0.027  1.00 11.74           C  \\nATOM    366  CZ  PHE B  24      -5.457  12.470  -1.008  1.00  9.78           C  \\nATOM    367  N   PHE B  25      -7.717  17.612   0.116  1.00 14.20           N  \\nATOM    368  CA  PHE B  25      -6.813  18.302   1.052  1.00 12.03           C  \\nATOM    369  C   PHE B  25      -6.569  17.356   2.221  1.00 12.69           C  \\nATOM    370  O   PHE B  25      -7.485  16.788   2.757  1.00 15.22           O  \\nATOM    371  CB  PHE B  25      -7.387  19.633   1.684  1.00 17.25           C  \\nATOM    372  CG  PHE B  25      -7.105  20.689   0.637  1.00 30.38           C  \\nATOM    373  CD1 PHE B  25      -7.842  20.802  -0.543  1.00 61.20           C  \\nATOM    374  CD2 PHE B  25      -6.003  21.541   0.896  1.00 56.90           C  \\nATOM    375  CE1 PHE B  25      -7.445  21.790  -1.461  1.00 29.52           C  \\nATOM    376  CE2 PHE B  25      -5.648  22.564  -0.027  1.00 40.31           C  \\nATOM    377  CZ  PHE B  25      -6.382  22.681  -1.235  1.00 30.38           C  \\nATOM    378  N   TYR B  26      -5.345  17.202   2.583  1.00 11.25           N  \\nATOM    379  CA  TYR B  26      -4.996  16.333   3.717  1.00 10.42           C  \\nATOM    380  C   TYR B  26      -4.445  17.350   4.714  1.00 15.08           C  \\nATOM    381  O   TYR B  26      -3.350  17.906   4.518  1.00 14.52           O  \\nATOM    382  CB  TYR B  26      -3.949  15.288   3.319  1.00  9.51           C  \\nATOM    383  CG  TYR B  26      -3.404  14.530   4.474  1.00 12.61           C  \\nATOM    384  CD1 TYR B  26      -4.243  13.688   5.178  1.00 20.50           C  \\nATOM    385  CD2 TYR B  26      -2.105  14.676   4.857  1.00 13.88           C  \\nATOM    386  CE1 TYR B  26      -3.652  12.967   6.246  1.00 16.38           C  \\nATOM    387  CE2 TYR B  26      -1.577  14.010   5.941  1.00 10.97           C  \\nATOM    388  CZ  TYR B  26      -2.347  13.149   6.642  1.00 11.71           C  \\nATOM    389  OH  TYR B  26      -1.853  12.447   7.734  1.00 16.39           O  \\nATOM    390  N   THR B  27      -5.287  17.537   5.752  1.00 15.47           N  \\nATOM    391  CA  THR B  27      -4.902  18.555   6.772  1.00 20.05           C  \\nATOM    392  C   THR B  27      -4.834  18.053   8.250  1.00 15.09           C  \\nATOM    393  O   THR B  27      -5.825  18.282   8.943  1.00 22.56           O  \\nATOM    394  CB  THR B  27      -5.856  19.825   6.753  1.00 26.34           C  \\nATOM    395  OG1ATHR B  27      -7.228  19.328   6.558  0.50 39.91           O  \\nATOM    396  OG1BTHR B  27      -5.691  20.478   5.511  0.50 29.43           O  \\nATOM    397  CG2ATHR B  27      -5.505  20.781   5.606  0.50 34.53           C  \\nATOM    398  CG2BTHR B  27      -5.413  20.850   7.858  0.50 34.13           C  \\nATOM    399  N   PRO B  28      -3.702  17.548   8.603  1.00 18.26           N  \\nATOM    400  CA  PRO B  28      -3.494  17.055   9.954  1.00 21.56           C  \\nATOM    401  C   PRO B  28      -3.306  18.220  10.892  1.00 22.68           C  \\nATOM    402  O   PRO B  28      -3.072  19.330  10.484  1.00 21.93           O  \\nATOM    403  CB  PRO B  28      -2.249  16.209   9.808  1.00 21.60           C  \\nATOM    404  CG  PRO B  28      -1.544  16.617   8.595  1.00 21.39           C  \\nATOM    405  CD  PRO B  28      -2.526  17.320   7.778  1.00 14.32           C  \\nATOM    406  N   LYS B  29      -3.452  17.975  12.175  1.00 26.27           N  \\nATOM    407  CA  LYS B  29      -3.227  18.941  13.307  1.00 23.17           C  \\nATOM    408  C   LYS B  29      -1.707  18.995  13.459  1.00 52.81           C  \\nATOM    409  O   LYS B  29      -1.026  17.919  13.406  1.00 39.37           O  \\nATOM    410  CB  LYS B  29      -3.764  18.417  14.615  1.00 22.26           C  \\nATOM    411  CG  LYS B  29      -3.990  19.385  15.801  1.00 48.01           C  \\nATOM    412  CD  LYS B  29      -5.153  18.811  16.622  1.00 37.36           C  \\nATOM    413  CE  LYS B  29      -5.067  18.493  18.087  1.00 53.09           C  \\nATOM    414  NZ  LYS B  29      -4.208  19.418  18.841  1.00 61.16           N  \\nATOM    415  N   ALA B  30      -1.166  20.052  13.779  1.00 53.30           N  \\nATOM    416  CA  ALA B  30       0.148  20.539  13.902  1.00 45.30           C  \\nATOM    417  C   ALA B  30       0.991  20.467  15.167  1.00 50.30           C  \\nATOM    418  O   ALA B  30       0.427  20.710  16.268  1.00 62.63           O  \\nATOM    419  CB  ALA B  30       0.033  22.113  13.690  1.00 53.30           C  \\nATOM    420  OXT ALA B  30       2.226  20.205  15.000  1.00 76.30           O  \\nTER     421      ALA B  30                                                      \\nATOM    422  N   GLY C   1      -0.643  19.956 -14.073  1.00 26.16           N  \\nATOM    423  CA  GLY C   1      -0.389  20.033 -12.615  1.00 30.96           C  \\nATOM    424  C   GLY C   1       0.447  18.825 -12.180  1.00 33.76           C  \\nATOM    425  O   GLY C   1       1.216  18.311 -13.006  1.00 21.35           O  \\nATOM    426  N   ILE C   2       0.244  18.434 -10.942  1.00 23.96           N  \\nATOM    427  CA  ILE C   2       1.003  17.290 -10.393  1.00 15.36           C  \\nATOM    428  C   ILE C   2       0.946  16.025 -11.185  1.00 13.59           C  \\nATOM    429  O   ILE C   2       1.971  15.359 -11.278  1.00 17.19           O  \\nATOM    430  CB  ILE C   2       0.491  17.013  -8.931  1.00 16.47           C  \\nATOM    431  CG1 ILE C   2       1.539  16.143  -8.164  1.00 15.91           C  \\nATOM    432  CG2 ILE C   2      -0.969  16.533  -8.863  1.00 17.06           C  \\nATOM    433  CD1 ILE C   2       1.081  15.828  -6.720  1.00 19.56           C  \\nATOM    434  N   VAL C   3      -0.179  15.655 -11.786  1.00 13.55           N  \\nATOM    435  CA  VAL C   3      -0.278  14.455 -12.591  1.00 17.58           C  \\nATOM    436  C   VAL C   3       0.590  14.454 -13.881  1.00 21.18           C  \\nATOM    437  O   VAL C   3       1.245  13.451 -14.197  1.00 19.26           O  \\nATOM    438  CB  VAL C   3      -1.709  14.136 -12.915  1.00 26.49           C  \\nATOM    439  CG1 VAL C   3      -1.808  12.868 -13.745  1.00 30.71           C  \\nATOM    440  CG2 VAL C   3      -2.567  13.989 -11.662  1.00 16.85           C  \\nATOM    441  N   GLU C   4       0.621  15.590 -14.497  1.00 24.23           N  \\nATOM    442  CA  GLU C   4       1.481  15.743 -15.686  1.00 23.06           C  \\nATOM    443  C   GLU C   4       2.966  15.744 -15.343  1.00 19.78           C  \\nATOM    444  O   GLU C   4       3.805  15.198 -16.061  1.00 29.30           O  \\nATOM    445  CB  GLU C   4       1.063  17.059 -16.361  1.00 24.43           C  \\nATOM    446  CG  GLU C   4      -0.349  16.900 -16.964  1.00 19.87           C  \\nATOM    447  CD  GLU C   4      -1.541  17.024 -16.030  1.00 27.93           C  \\nATOM    448  OE1 GLU C   4      -2.429  16.169 -16.211  1.00 37.08           O  \\nATOM    449  OE2 GLU C   4      -1.527  17.967 -15.181  1.00 27.92           O  \\nATOM    450  N   GLN C   5       3.289  16.386 -14.236  1.00 17.90           N  \\nATOM    451  CA  GLN C   5       4.695  16.445 -13.800  1.00 17.90           C  \\nATOM    452  C   GLN C   5       5.206  15.104 -13.307  1.00 23.62           C  \\nATOM    453  O   GLN C   5       6.331  14.684 -13.578  1.00 26.25           O  \\nATOM    454  CB  GLN C   5       4.820  17.553 -12.780  1.00 22.64           C  \\nATOM    455  CG  GLN C   5       4.373  18.969 -13.199  1.00 32.56           C  \\nATOM    456  CD  GLN C   5       5.405  19.755 -12.404  1.00 59.85           C  \\nATOM    457  OE1 GLN C   5       6.478  19.979 -12.947  1.00 50.22           O  \\nATOM    458  NE2 GLN C   5       4.976  19.940 -11.148  1.00 56.46           N  \\nATOM    459  N   CYS C   6       4.386  14.408 -12.514  1.00 20.45           N  \\nATOM    460  CA  CYS C   6       4.949  13.166 -11.921  1.00 20.95           C  \\nATOM    461  C   CYS C   6       4.472  11.875 -12.436  1.00 18.67           C  \\nATOM    462  O   CYS C   6       5.127  10.851 -12.293  1.00 17.98           O  \\nATOM    463  CB  CYS C   6       4.732  13.264 -10.354  1.00 16.76           C  \\nATOM    464  SG  CYS C   6       5.416  14.719  -9.509  1.00 20.00           S  \\nATOM    465  N   CYS C   7       3.347  11.802 -13.063  1.00 15.93           N  \\nATOM    466  CA  CYS C   7       2.858  10.515 -13.612  1.00 10.31           C  \\nATOM    467  C   CYS C   7       3.090  10.422 -15.112  1.00 27.34           C  \\nATOM    468  O   CYS C   7       3.601   9.455 -15.603  1.00 24.77           O  \\nATOM    469  CB  CYS C   7       1.348  10.332 -13.329  1.00 17.60           C  \\nATOM    470  SG  CYS C   7       0.602   8.899 -13.976  1.00 19.30           S  \\nATOM    471  N   THR C   8       2.691  11.395 -15.898  1.00 22.31           N  \\nATOM    472  CA  THR C   8       2.912  11.356 -17.382  1.00 20.22           C  \\nATOM    473  C   THR C   8       4.408  11.508 -17.641  1.00 29.24           C  \\nATOM    474  O   THR C   8       5.051  10.716 -18.357  1.00 25.27           O  \\nATOM    475  CB  THR C   8       1.993  12.487 -17.997  1.00 28.76           C  \\nATOM    476  OG1 THR C   8       0.590  12.071 -17.868  1.00 41.93           O  \\nATOM    477  CG2 THR C   8       2.536  12.795 -19.370  1.00 39.10           C  \\nATOM    478  N   SER C   9       4.957  12.549 -16.969  1.00 23.04           N  \\nATOM    479  CA  SER C   9       6.438  12.783 -17.031  1.00 23.81           C  \\nATOM    480  C   SER C   9       7.038  12.104 -15.796  1.00 22.97           C  \\nATOM    481  O   SER C   9       6.505  11.080 -15.351  1.00 28.36           O  \\nATOM    482  CB  SER C   9       6.838  14.249 -17.258  1.00 22.96           C  \\nATOM    483  OG  SER C   9       8.266  14.523 -17.319  1.00 30.96           O  \\nATOM    484  N   ILE C  10       8.157  12.532 -15.274  1.00 18.48           N  \\nATOM    485  CA  ILE C  10       8.820  12.068 -14.116  1.00 14.18           C  \\nATOM    486  C   ILE C  10       9.162  13.268 -13.217  1.00 22.06           C  \\nATOM    487  O   ILE C  10       9.398  14.336 -13.807  1.00 21.37           O  \\nATOM    488  CB  ILE C  10      10.082  11.219 -14.296  1.00 14.27           C  \\nATOM    489  CG1 ILE C  10      11.168  12.058 -14.941  1.00 17.05           C  \\nATOM    490  CG2 ILE C  10       9.771   9.901 -15.075  1.00 23.12           C  \\nATOM    491  CD1 ILE C  10      12.447  11.250 -15.181  1.00 28.86           C  \\nATOM    492  N   CYS C  11       9.158  13.001 -11.897  1.00 17.73           N  \\nATOM    493  CA  CYS C  11       9.488  14.178 -11.063  1.00 13.63           C  \\nATOM    494  C   CYS C  11      10.301  13.733  -9.869  1.00 24.12           C  \\nATOM    495  O   CYS C  11      10.238  12.518  -9.595  1.00 27.63           O  \\nATOM    496  CB  CYS C  11       8.284  14.934 -10.501  1.00 24.47           C  \\nATOM    497  SG  CYS C  11       7.324  13.971  -9.301  1.00 22.60           S  \\nATOM    498  N   SER C  12      10.941  14.682  -9.241  1.00 20.64           N  \\nATOM    499  CA  SER C  12      11.739  14.434  -8.095  1.00 19.89           C  \\nATOM    500  C   SER C  12      10.978  14.612  -6.784  1.00 26.49           C  \\nATOM    501  O   SER C  12       9.925  15.241  -6.843  1.00 17.94           O  \\nATOM    502  CB  SER C  12      12.896  15.478  -7.911  1.00 22.13           C  \\nATOM    503  OG  SER C  12      12.280  16.737  -7.889  1.00 25.55           O  \\nATOM    504  N   LEU C  13      11.522  14.165  -5.654  1.00 20.92           N  \\nATOM    505  CA  LEU C  13      10.827  14.392  -4.422  1.00 18.59           C  \\nATOM    506  C   LEU C  13      10.811  15.860  -4.099  1.00 17.62           C  \\nATOM    507  O   LEU C  13       9.865  16.311  -3.427  1.00 17.56           O  \\nATOM    508  CB  LEU C  13      11.451  13.616  -3.261  1.00 24.54           C  \\nATOM    509  CG  LEU C  13      11.620  12.137  -3.475  1.00 37.60           C  \\nATOM    510  CD1 LEU C  13      12.386  11.586  -2.273  1.00 42.94           C  \\nATOM    511  CD2 LEU C  13      10.209  11.562  -3.628  1.00 33.13           C  \\nATOM    512  N   TYR C  14      11.756  16.642  -4.538  1.00 19.39           N  \\nATOM    513  CA  TYR C  14      11.756  18.053  -4.253  1.00 15.81           C  \\nATOM    514  C   TYR C  14      10.632  18.665  -4.977  1.00 16.49           C  \\nATOM    515  O   TYR C  14      10.098  19.673  -4.517  1.00 24.31           O  \\nATOM    516  CB  TYR C  14      13.061  18.792  -4.698  1.00 23.42           C  \\nATOM    517  CG  TYR C  14      14.244  18.334  -3.884  1.00 22.68           C  \\nATOM    518  CD1 TYR C  14      14.406  18.926  -2.643  1.00 37.43           C  \\nATOM    519  CD2 TYR C  14      15.065  17.294  -4.321  1.00 44.52           C  \\nATOM    520  CE1 TYR C  14      15.469  18.455  -1.806  1.00 35.18           C  \\nATOM    521  CE2 TYR C  14      16.138  16.837  -3.558  1.00 39.13           C  \\nATOM    522  CZ  TYR C  14      16.305  17.440  -2.304  1.00 43.24           C  \\nATOM    523  OH  TYR C  14      17.325  17.067  -1.472  1.00 40.09           O  \\nATOM    524  N   GLN C  15      10.286  18.189  -6.156  1.00 16.44           N  \\nATOM    525  CA  GLN C  15       9.175  18.706  -6.908  1.00 13.87           C  \\nATOM    526  C   GLN C  15       7.940  18.402  -6.108  1.00 16.45           C  \\nATOM    527  O   GLN C  15       7.042  19.228  -6.047  1.00 17.74           O  \\nATOM    528  CB  GLN C  15       9.021  18.202  -8.378  1.00 19.83           C  \\nATOM    529  CG  GLN C  15      10.045  18.695  -9.368  1.00 27.50           C  \\nATOM    530  CD  GLN C  15      10.002  18.165 -10.771  1.00 46.46           C  \\nATOM    531  OE1 GLN C  15      10.459  17.057 -11.040  1.00 43.32           O  \\nATOM    532  NE2 GLN C  15       9.456  18.889 -11.761  1.00 35.38           N  \\nATOM    533  N   LEU C  16       7.841  17.245  -5.491  1.00 18.47           N  \\nATOM    534  CA  LEU C  16       6.580  16.984  -4.783  1.00 17.18           C  \\nATOM    535  C   LEU C  16       6.311  17.904  -3.587  1.00 21.36           C  \\nATOM    536  O   LEU C  16       5.195  18.069  -3.079  1.00 17.34           O  \\nATOM    537  CB  LEU C  16       6.542  15.527  -4.227  1.00 17.15           C  \\nATOM    538  CG  LEU C  16       6.500  14.520  -5.341  1.00 17.53           C  \\nATOM    539  CD1 LEU C  16       6.477  13.227  -4.593  1.00 22.82           C  \\nATOM    540  CD2 LEU C  16       5.117  14.493  -5.991  1.00 18.33           C  \\nATOM    541  N   GLU C  17       7.389  18.427  -3.040  1.00 16.47           N  \\nATOM    542  CA  GLU C  17       7.353  19.319  -1.959  1.00 10.93           C  \\nATOM    543  C   GLU C  17       6.614  20.601  -2.282  1.00 11.20           C  \\nATOM    544  O   GLU C  17       6.183  21.310  -1.386  1.00 13.69           O  \\nATOM    545  CB  GLU C  17       8.751  19.800  -1.584  1.00 16.99           C  \\nATOM    546  CG  GLU C  17       9.178  18.980  -0.408  1.00 22.69           C  \\nATOM    547  CD  GLU C  17      10.397  19.524   0.269  1.00 26.38           C  \\nATOM    548  OE1 GLU C  17      11.270  20.013  -0.342  1.00 26.42           O  \\nATOM    549  OE2 GLU C  17      10.358  19.291   1.470  1.00 30.68           O  \\nATOM    550  N   ASN C  18       6.466  20.844  -3.575  1.00 13.42           N  \\nATOM    551  CA  ASN C  18       5.707  22.010  -4.006  1.00 12.91           C  \\nATOM    552  C   ASN C  18       4.238  21.879  -3.683  1.00 14.96           C  \\nATOM    553  O   ASN C  18       3.506  22.887  -3.698  1.00 17.72           O  \\nATOM    554  CB  ASN C  18       5.894  22.279  -5.506  1.00 16.62           C  \\nATOM    555  CG  ASN C  18       7.286  22.755  -5.884  1.00 20.64           C  \\nATOM    556  OD1 ASN C  18       7.761  22.375  -6.927  1.00 28.41           O  \\nATOM    557  ND2 ASN C  18       7.836  23.540  -4.999  1.00 26.53           N  \\nATOM    558  N   TYR C  19       3.683  20.717  -3.420  1.00 10.42           N  \\nATOM    559  CA  TYR C  19       2.315  20.489  -3.110  1.00 11.44           C  \\nATOM    560  C   TYR C  19       2.035  20.284  -1.641  1.00 19.22           C  \\nATOM    561  O   TYR C  19       0.884  20.008  -1.330  1.00 14.52           O  \\nATOM    562  CB  TYR C  19       1.745  19.342  -3.938  1.00 13.89           C  \\nATOM    563  CG  TYR C  19       1.995  19.579  -5.394  1.00 13.15           C  \\nATOM    564  CD1 TYR C  19       1.176  20.360  -6.185  1.00 15.85           C  \\nATOM    565  CD2 TYR C  19       3.101  19.022  -6.004  1.00 20.18           C  \\nATOM    566  CE1 TYR C  19       1.482  20.590  -7.526  1.00 15.80           C  \\nATOM    567  CE2 TYR C  19       3.427  19.257  -7.342  1.00 20.94           C  \\nATOM    568  CZ  TYR C  19       2.600  20.077  -8.111  1.00 16.07           C  \\nATOM    569  OH  TYR C  19       2.930  20.198  -9.438  1.00 28.20           O  \\nATOM    570  N   CYS C  20       2.997  20.523  -0.779  1.00 15.85           N  \\nATOM    571  CA  CYS C  20       2.816  20.515   0.660  1.00 12.75           C  \\nATOM    572  C   CYS C  20       2.167  21.848   1.080  1.00 18.29           C  \\nATOM    573  O   CYS C  20       2.424  22.842   0.449  1.00 26.63           O  \\nATOM    574  CB  CYS C  20       4.116  20.329   1.465  1.00 12.02           C  \\nATOM    575  SG  CYS C  20       5.109  18.925   1.079  1.00 13.70           S  \\nATOM    576  N   ASN C  21       1.396  21.899   2.168  1.00 17.14           N  \\nATOM    577  CA  ASN C  21       0.814  23.152   2.677  1.00 22.18           C  \\nATOM    578  C   ASN C  21       1.879  23.650   3.681  1.00 31.46           C  \\nATOM    579  O   ASN C  21       2.937  22.991   3.836  1.00 44.77           O  \\nATOM    580  CB  ASN C  21      -0.433  22.757   3.387  1.00 21.67           C  \\nATOM    581  CG  ASN C  21      -1.636  22.166   2.742  1.00 29.17           C  \\nATOM    582  OD1 ASN C  21      -2.062  22.468   1.647  1.00 33.63           O  \\nATOM    583  ND2 ASN C  21      -2.242  21.277   3.591  1.00 33.21           N  \\nATOM    584  OXT ASN C  21       1.575  24.585   4.398  1.00 52.38           O  \\nTER     585      ASN C  21                                                      \\nATOM    586  N   PHE D   1      18.330  11.816  -3.893  1.00 61.50           N  \\nATOM    587  CA  PHE D   1      17.047  11.272  -4.371  1.00 28.75           C  \\nATOM    588  C   PHE D   1      17.165  10.885  -5.854  1.00 45.77           C  \\nATOM    589  O   PHE D   1      18.169  11.137  -6.550  1.00 49.85           O  \\nATOM    590  CB  PHE D   1      15.801  12.158  -4.041  1.00 30.01           C  \\nATOM    591  CG  PHE D   1      16.035  12.424  -2.571  1.00 31.74           C  \\nATOM    592  CD1 PHE D   1      15.980  11.294  -1.745  1.00 39.79           C  \\nATOM    593  CD2 PHE D   1      16.425  13.629  -2.070  1.00 31.26           C  \\nATOM    594  CE1 PHE D   1      16.219  11.375  -0.405  1.00 31.66           C  \\nATOM    595  CE2 PHE D   1      16.706  13.719  -0.715  1.00 53.74           C  \\nATOM    596  CZ  PHE D   1      16.609  12.612   0.115  1.00 37.15           C  \\nATOM    597  N   VAL D   2      16.117  10.168  -6.274  1.00 33.71           N  \\nATOM    598  CA  VAL D   2      16.006   9.666  -7.682  1.00 24.16           C  \\nATOM    599  C   VAL D   2      14.648  10.218  -8.239  1.00 33.79           C  \\nATOM    600  O   VAL D   2      13.733  10.588  -7.495  1.00 22.26           O  \\nATOM    601  CB  VAL D   2      16.128   8.144  -7.911  1.00 27.80           C  \\nATOM    602  CG1 VAL D   2      17.529   7.562  -7.875  1.00 85.81           C  \\nATOM    603  CG2 VAL D   2      15.239   7.326  -6.976  1.00 36.43           C  \\nATOM    604  N   ASN D   3      14.564  10.315  -9.558  1.00 19.91           N  \\nATOM    605  CA  ASN D   3      13.383  10.694 -10.332  1.00 33.51           C  \\nATOM    606  C   ASN D   3      12.449   9.525 -10.618  1.00 27.19           C  \\nATOM    607  O   ASN D   3      12.946   8.479 -11.058  1.00 35.28           O  \\nATOM    608  CB  ASN D   3      13.966  11.427 -11.571  1.00 28.03           C  \\nATOM    609  CG  ASN D   3      14.214  12.869 -11.102  1.00 36.72           C  \\nATOM    610  OD1 ASN D   3      14.425  13.813 -11.829  1.00 71.83           O  \\nATOM    611  ND2 ASN D   3      14.114  13.118  -9.791  1.00 69.95           N  \\nATOM    612  N   GLN D   4      11.128   9.652 -10.488  1.00 38.71           N  \\nATOM    613  CA  GLN D   4      10.224   8.523 -10.782  1.00 30.37           C  \\nATOM    614  C   GLN D   4       8.833   8.864 -11.345  1.00 13.92           C  \\nATOM    615  O   GLN D   4       8.361  10.028 -11.197  1.00 25.56           O  \\nATOM    616  CB  GLN D   4       9.983   7.921  -9.378  1.00 38.84           C  \\nATOM    617  CG  GLN D   4      10.350   6.429  -9.194  1.00 40.71           C  \\nATOM    618  CD  GLN D   4      10.515   6.248  -7.678  1.00 31.29           C  \\nATOM    619  OE1 GLN D   4      10.490   5.138  -7.268  1.00 47.62           O  \\nATOM    620  NE2 GLN D   4      10.697   7.383  -6.997  1.00 30.17           N  \\nATOM    621  N   HIS D   5       8.291   7.803 -12.000  1.00 13.87           N  \\nATOM    622  CA  HIS D   5       6.903   7.919 -12.482  1.00 16.22           C  \\nATOM    623  C   HIS D   5       6.029   7.528 -11.252  1.00 16.61           C  \\nATOM    624  O   HIS D   5       6.115   6.355 -10.822  1.00 21.77           O  \\nATOM    625  CB  HIS D   5       6.427   7.086 -13.663  1.00 24.93           C  \\nATOM    626  CG  HIS D   5       6.855   7.450 -15.054  1.00 32.91           C  \\nATOM    627  ND1 HIS D   5       6.285   8.338 -15.876  1.00 31.86           N  \\nATOM    628  CD2 HIS D   5       7.867   6.966 -15.822  1.00 31.55           C  \\nATOM    629  CE1 HIS D   5       6.866   8.458 -17.068  1.00 28.50           C  \\nATOM    630  NE2 HIS D   5       7.838   7.587 -17.015  1.00 18.29           N  \\nATOM    631  N   LEU D   6       5.259   8.452 -10.792  1.00 11.23           N  \\nATOM    632  CA  LEU D   6       4.302   8.232  -9.639  1.00 22.97           C  \\nATOM    633  C   LEU D   6       2.854   8.566 -10.048  1.00 11.62           C  \\nATOM    634  O   LEU D   6       2.505   9.705 -10.335  1.00 14.32           O  \\nATOM    635  CB  LEU D   6       4.815   9.069  -8.489  1.00 16.96           C  \\nATOM    636  CG  LEU D   6       6.253   8.828  -7.985  1.00 18.92           C  \\nATOM    637  CD1 LEU D   6       6.828  10.000  -7.271  1.00 17.11           C  \\nATOM    638  CD2 LEU D   6       6.091   7.594  -7.091  1.00 18.40           C  \\nATOM    639  N   CYS D   7       2.119   7.495 -10.108  1.00 13.70           N  \\nATOM    640  CA  CYS D   7       0.745   7.570 -10.582  1.00 18.64           C  \\nATOM    641  C   CYS D   7      -0.288   7.058  -9.564  1.00 17.90           C  \\nATOM    642  O   CYS D   7       0.016   6.138  -8.853  1.00 18.63           O  \\nATOM    643  CB  CYS D   7       0.607   6.723 -11.869  1.00 31.44           C  \\nATOM    644  SG  CYS D   7       1.635   7.350 -13.232  1.00 20.00           S  \\nATOM    645  N   GLY D   8      -1.440   7.618  -9.644  1.00 19.94           N  \\nATOM    646  CA  GLY D   8      -2.580   7.155  -8.776  1.00 18.12           C  \\nATOM    647  C   GLY D   8      -2.305   7.237  -7.329  1.00  8.63           C  \\nATOM    648  O   GLY D   8      -1.914   8.246  -6.763  1.00 13.74           O  \\nATOM    649  N   SER D   9      -2.656   6.101  -6.694  1.00 11.65           N  \\nATOM    650  CA  SER D   9      -2.483   5.997  -5.227  1.00 12.39           C  \\nATOM    651  C   SER D   9      -1.032   6.120  -4.867  1.00  8.86           C  \\nATOM    652  O   SER D   9      -0.760   6.584  -3.762  1.00  9.84           O  \\nATOM    653  CB  SER D   9      -3.264   4.870  -4.568  1.00 14.54           C  \\nATOM    654  OG  SER D   9      -2.756   3.670  -5.015  1.00 21.40           O  \\nATOM    655  N   HIS D  10      -0.166   5.698  -5.800  1.00  8.91           N  \\nATOM    656  CA  HIS D  10       1.270   5.805  -5.452  1.00  9.09           C  \\nATOM    657  C   HIS D  10       1.755   7.227  -5.317  1.00  7.69           C  \\nATOM    658  O   HIS D  10       2.658   7.481  -4.533  1.00 11.42           O  \\nATOM    659  CB  HIS D  10       2.092   5.048  -6.482  1.00  8.95           C  \\nATOM    660  CG  HIS D  10       1.707   3.599  -6.615  1.00  8.78           C  \\nATOM    661  ND1 HIS D  10       1.951   2.701  -5.608  1.00 13.99           N  \\nATOM    662  CD2 HIS D  10       1.059   2.922  -7.561  1.00 11.16           C  \\nATOM    663  CE1 HIS D  10       1.536   1.530  -6.025  1.00  8.79           C  \\nATOM    664  NE2 HIS D  10       0.948   1.628  -7.159  1.00 11.60           N  \\nATOM    665  N   LEU D  11       1.142   8.109  -6.070  1.00  9.45           N  \\nATOM    666  CA  LEU D  11       1.434   9.541  -6.053  1.00  9.98           C  \\nATOM    667  C   LEU D  11       1.027  10.155  -4.729  1.00  5.91           C  \\nATOM    668  O   LEU D  11       1.818  10.853  -4.084  1.00  8.83           O  \\nATOM    669  CB  LEU D  11       0.815  10.196  -7.249  1.00 10.51           C  \\nATOM    670  CG  LEU D  11       1.105  11.705  -7.294  1.00 13.37           C  \\nATOM    671  CD1 LEU D  11       2.526  12.118  -7.270  1.00 11.72           C  \\nATOM    672  CD2 LEU D  11       0.404  12.225  -8.564  1.00 16.22           C  \\nATOM    673  N   VAL D  12      -0.188   9.816  -4.302  1.00  6.52           N  \\nATOM    674  CA  VAL D  12      -0.534  10.395  -2.938  1.00  6.95           C  \\nATOM    675  C   VAL D  12       0.207   9.758  -1.787  1.00  5.55           C  \\nATOM    676  O   VAL D  12       0.467  10.475  -0.761  1.00  8.60           O  \\nATOM    677  CB  VAL D  12      -2.065  10.355  -2.790  1.00 14.07           C  \\nATOM    678  CG1 VAL D  12      -2.687  11.271  -3.899  1.00 16.64           C  \\nATOM    679  CG2 VAL D  12      -2.636   9.010  -2.737  1.00 14.03           C  \\nATOM    680  N   GLU D  13       0.681   8.537  -1.890  1.00  8.61           N  \\nATOM    681  CA  GLU D  13       1.509   7.903  -0.866  1.00  7.25           C  \\nATOM    682  C   GLU D  13       2.815   8.631  -0.804  1.00  7.02           C  \\nATOM    683  O   GLU D  13       3.403   8.915   0.274  1.00 11.89           O  \\nATOM    684  CB  GLU D  13       1.770   6.435  -1.051  1.00 11.84           C  \\nATOM    685  CG  GLU D  13       0.458   5.634  -0.816  1.00 27.71           C  \\nATOM    686  CD  GLU D  13       0.536   4.937   0.492  1.00 62.15           C  \\nATOM    687  OE1 GLU D  13       1.499   5.094   1.172  1.00 28.12           O  \\nATOM    688  OE2 GLU D  13      -0.429   4.258   0.726  1.00 32.97           O  \\nATOM    689  N   ALA D  14       3.390   9.046  -1.888  1.00  9.58           N  \\nATOM    690  CA  ALA D  14       4.643   9.836  -1.985  1.00 10.00           C  \\nATOM    691  C   ALA D  14       4.495  11.218  -1.377  1.00 10.02           C  \\nATOM    692  O   ALA D  14       5.373  11.718  -0.675  1.00 12.34           O  \\nATOM    693  CB  ALA D  14       5.151   9.884  -3.366  1.00  9.23           C  \\nATOM    694  N   LEU D  15       3.340  11.836  -1.609  1.00  8.93           N  \\nATOM    695  CA  LEU D  15       3.014  13.121  -1.053  1.00  9.97           C  \\nATOM    696  C   LEU D  15       2.968  13.020   0.450  1.00 16.23           C  \\nATOM    697  O   LEU D  15       3.462  13.914   1.142  1.00 11.63           O  \\nATOM    698  CB  LEU D  15       1.719  13.654  -1.641  1.00  9.29           C  \\nATOM    699  CG  LEU D  15       1.771  14.340  -2.979  1.00 10.49           C  \\nATOM    700  CD1 LEU D  15       0.398  14.483  -3.556  1.00 12.99           C  \\nATOM    701  CD2 LEU D  15       2.487  15.663  -2.804  1.00 15.87           C  \\nATOM    702  N   TYR D  16       2.313  11.974   0.938  1.00 11.90           N  \\nATOM    703  CA  TYR D  16       2.254  11.791   2.374  1.00  8.84           C  \\nATOM    704  C   TYR D  16       3.662  11.710   3.013  1.00 11.08           C  \\nATOM    705  O   TYR D  16       3.925  12.338   4.014  1.00 10.74           O  \\nATOM    706  CB  TYR D  16       1.409  10.545   2.657  1.00  8.68           C  \\nATOM    707  CG  TYR D  16       1.408  10.201   4.150  1.00  9.93           C  \\nATOM    708  CD1 TYR D  16       0.611  10.879   5.091  1.00 13.27           C  \\nATOM    709  CD2 TYR D  16       2.232   9.163   4.535  1.00 10.34           C  \\nATOM    710  CE1 TYR D  16       0.657  10.554   6.429  1.00 11.61           C  \\nATOM    711  CE2 TYR D  16       2.284   8.846   5.894  1.00 17.86           C  \\nATOM    712  CZ  TYR D  16       1.518   9.554   6.843  1.00 13.23           C  \\nATOM    713  OH  TYR D  16       1.664   9.081   8.177  1.00 17.80           O  \\nATOM    714  N   LEU D  17       4.478  10.910   2.349  1.00  9.51           N  \\nATOM    715  CA  LEU D  17       5.832  10.719   2.819  1.00  8.94           C  \\nATOM    716  C   LEU D  17       6.672  11.993   2.879  1.00 12.51           C  \\nATOM    717  O   LEU D  17       7.395  12.298   3.853  1.00 13.68           O  \\nATOM    718  CB  LEU D  17       6.573   9.683   1.932  1.00 12.01           C  \\nATOM    719  CG  LEU D  17       7.784   9.141   2.606  1.00 26.75           C  \\nATOM    720  CD1 LEU D  17       7.274   8.084   3.613  1.00 27.77           C  \\nATOM    721  CD2 LEU D  17       8.533   8.579   1.435  1.00 36.25           C  \\nATOM    722  N   VAL D  18       6.585  12.773   1.784  1.00 10.21           N  \\nATOM    723  CA  VAL D  18       7.348  14.029   1.661  1.00  8.25           C  \\nATOM    724  C   VAL D  18       6.789  15.099   2.514  1.00  8.01           C  \\nATOM    725  O   VAL D  18       7.560  15.914   3.063  1.00 14.17           O  \\nATOM    726  CB  VAL D  18       7.357  14.409   0.176  1.00  8.24           C  \\nATOM    727  CG1 VAL D  18       7.683  15.903   0.006  1.00 18.17           C  \\nATOM    728  CG2 VAL D  18       8.321  13.582  -0.638  1.00 16.22           C  \\nATOM    729  N   CYS D  19       5.469  15.224   2.565  1.00  8.80           N  \\nATOM    730  CA  CYS D  19       4.790  16.327   3.282  1.00 10.89           C  \\nATOM    731  C   CYS D  19       4.550  16.076   4.760  1.00 20.41           C  \\nATOM    732  O   CYS D  19       4.771  17.057   5.513  1.00 17.13           O  \\nATOM    733  CB  CYS D  19       3.540  16.778   2.546  1.00 11.71           C  \\nATOM    734  SG  CYS D  19       3.846  17.363   0.909  1.00 11.30           S  \\nATOM    735  N   GLY D  20       4.108  14.860   5.076  1.00 15.31           N  \\nATOM    736  CA  GLY D  20       3.841  14.553   6.475  1.00 18.10           C  \\nATOM    737  C   GLY D  20       2.815  15.479   7.177  1.00 19.19           C  \\nATOM    738  O   GLY D  20       1.669  15.721   6.770  1.00 18.72           O  \\nATOM    739  N   GLU D  21       3.387  15.976   8.278  1.00 20.01           N  \\nATOM    740  CA  GLU D  21       2.659  16.847   9.244  1.00 23.69           C  \\nATOM    741  C   GLU D  21       2.237  18.138   8.636  1.00 19.09           C  \\nATOM    742  O   GLU D  21       1.246  18.727   9.109  1.00 24.03           O  \\nATOM    743  CB  GLU D  21       3.575  17.120  10.476  1.00 22.76           C  \\nATOM    744  CG  GLU D  21       5.075  17.192  10.046  1.00 53.97           C  \\nATOM    745  CD  GLU D  21       6.099  17.832  10.938  1.00 78.86           C  \\nATOM    746  OE1 GLU D  21       5.775  18.563  11.863  1.00 73.07           O  \\nATOM    747  OE2 GLU D  21       7.300  17.568  10.645  1.00 72.74           O  \\nATOM    748  N   ARG D  22       2.952  18.555   7.566  1.00 15.75           N  \\nATOM    749  CA  ARG D  22       2.552  19.835   6.996  1.00 21.02           C  \\nATOM    750  C   ARG D  22       1.231  19.779   6.228  1.00 30.79           C  \\nATOM    751  O   ARG D  22       0.558  20.740   5.955  1.00 22.02           O  \\nATOM    752  CB  ARG D  22       3.613  20.247   5.973  1.00 29.63           C  \\nATOM    753  CG  ARG D  22       5.024  20.325   6.450  1.00 16.81           C  \\nATOM    754  CD  ARG D  22       5.952  20.706   5.309  1.00 32.57           C  \\nATOM    755  NE AARG D  22       5.584  21.912   4.554  0.50 23.88           N  \\nATOM    756  NE BARG D  22       6.663  19.518   4.827  0.50 24.77           N  \\nATOM    757  CZ AARG D  22       6.163  22.431   3.476  0.50 34.85           C  \\nATOM    758  CZ BARG D  22       7.482  19.684   3.777  0.50 11.98           C  \\nATOM    759  NH1AARG D  22       5.819  23.547   2.845  0.50 18.26           N  \\nATOM    760  NH1BARG D  22       7.547  20.848   3.124  0.50 20.33           N  \\nATOM    761  NH2AARG D  22       7.190  21.770   2.930  0.50 20.80           N  \\nATOM    762  NH2BARG D  22       8.196  18.646   3.318  0.50 20.38           N  \\nATOM    763  N   GLY D  23       0.914  18.555   5.836  1.00 17.93           N  \\nATOM    764  CA  GLY D  23      -0.237  18.302   4.993  1.00 12.82           C  \\nATOM    765  C   GLY D  23       0.087  18.615   3.514  1.00 11.10           C  \\nATOM    766  O   GLY D  23       1.137  19.102   3.198  1.00 12.49           O  \\nATOM    767  N   PHE D  24      -0.903  18.290   2.672  1.00 11.76           N  \\nATOM    768  CA  PHE D  24      -0.774  18.497   1.209  1.00 10.21           C  \\nATOM    769  C   PHE D  24      -2.127  18.688   0.607  1.00  9.65           C  \\nATOM    770  O   PHE D  24      -3.142  18.470   1.204  1.00 12.03           O  \\nATOM    771  CB  PHE D  24       0.000  17.400   0.578  1.00  9.41           C  \\nATOM    772  CG  PHE D  24      -0.602  16.046   0.597  1.00  9.89           C  \\nATOM    773  CD1 PHE D  24      -0.280  15.182   1.648  1.00 12.55           C  \\nATOM    774  CD2 PHE D  24      -1.497  15.657  -0.387  1.00 12.14           C  \\nATOM    775  CE1 PHE D  24      -0.863  13.901   1.714  1.00 19.03           C  \\nATOM    776  CE2 PHE D  24      -2.049  14.388  -0.368  1.00 12.53           C  \\nATOM    777  CZ  PHE D  24      -1.737  13.486   0.692  1.00 14.23           C  \\nATOM    778  N   PHE D  25      -2.070  19.069  -0.693  1.00  8.89           N  \\nATOM    779  CA  PHE D  25      -3.285  19.185  -1.469  1.00 10.03           C  \\nATOM    780  C   PHE D  25      -3.021  18.326  -2.724  1.00  9.84           C  \\nATOM    781  O   PHE D  25      -1.918  18.362  -3.216  1.00 12.46           O  \\nATOM    782  CB  PHE D  25      -3.686  20.613  -1.925  1.00 10.32           C  \\nATOM    783  CG  PHE D  25      -2.567  21.455  -2.479  1.00 14.41           C  \\nATOM    784  CD1 PHE D  25      -2.435  21.546  -3.850  1.00 20.47           C  \\nATOM    785  CD2 PHE D  25      -1.647  22.070  -1.651  1.00 14.83           C  \\nATOM    786  CE1 PHE D  25      -1.470  22.321  -4.476  1.00 13.89           C  \\nATOM    787  CE2 PHE D  25      -0.608  22.775  -2.271  1.00 20.12           C  \\nATOM    788  CZ  PHE D  25      -0.558  22.880  -3.660  1.00 13.72           C  \\nATOM    789  N   TYR D  26      -3.962  17.546  -3.192  1.00 10.77           N  \\nATOM    790  CA  TYR D  26      -3.915  16.723  -4.366  1.00  8.24           C  \\nATOM    791  C   TYR D  26      -5.013  17.222  -5.272  1.00 11.68           C  \\nATOM    792  O   TYR D  26      -6.209  17.040  -5.011  1.00 12.49           O  \\nATOM    793  CB  TYR D  26      -4.090  15.233  -4.010  1.00 12.91           C  \\nATOM    794  CG  TYR D  26      -4.159  14.291  -5.196  1.00 13.37           C  \\nATOM    795  CD1 TYR D  26      -5.379  13.599  -5.455  1.00 12.94           C  \\nATOM    796  CD2 TYR D  26      -3.138  14.227  -6.110  1.00 12.82           C  \\nATOM    797  CE1 TYR D  26      -5.491  12.770  -6.582  1.00 18.45           C  \\nATOM    798  CE2 TYR D  26      -3.289  13.401  -7.216  1.00 14.97           C  \\nATOM    799  CZ  TYR D  26      -4.390  12.679  -7.390  1.00 16.87           C  \\nATOM    800  OH  TYR D  26      -4.411  11.849  -8.508  1.00 19.99           O  \\nATOM    801  N   THR D  27      -4.639  17.933  -6.342  1.00 10.07           N  \\nATOM    802  CA  THR D  27      -5.563  18.555  -7.264  1.00 15.24           C  \\nATOM    803  C   THR D  27      -5.254  18.120  -8.704  1.00 16.10           C  \\nATOM    804  O   THR D  27      -4.545  18.821  -9.404  1.00 18.07           O  \\nATOM    805  CB  THR D  27      -5.536  20.114  -7.048  1.00 16.40           C  \\nATOM    806  OG1 THR D  27      -4.178  20.585  -6.974  1.00 19.93           O  \\nATOM    807  CG2 THR D  27      -6.181  20.408  -5.709  1.00 16.74           C  \\nATOM    808  N   PRO D  28      -5.766  17.035  -9.149  1.00 11.98           N  \\nATOM    809  CA  PRO D  28      -5.571  16.537 -10.477  1.00 13.55           C  \\nATOM    810  C   PRO D  28      -6.105  17.452 -11.585  1.00 21.64           C  \\nATOM    811  O   PRO D  28      -5.471  17.453 -12.635  1.00 24.41           O  \\nATOM    812  CB  PRO D  28      -6.192  15.163 -10.479  1.00 21.77           C  \\nATOM    813  CG  PRO D  28      -6.813  14.874  -9.135  1.00 18.60           C  \\nATOM    814  CD  PRO D  28      -6.687  16.109  -8.343  1.00 11.54           C  \\nATOM    815  N   LYS D  29      -7.103  18.249 -11.346  1.00 18.85           N  \\nATOM    816  CA  LYS D  29      -7.647  19.148 -12.397  1.00 21.86           C  \\nATOM    817  C   LYS D  29      -6.782  20.313 -12.719  1.00 25.32           C  \\nATOM    818  O   LYS D  29      -6.944  20.988 -13.739  1.00 28.67           O  \\nATOM    819  CB  LYS D  29      -9.020  19.689 -11.971  1.00 37.73           C  \\nATOM    820  CG ALYS D  29     -10.118  18.678 -12.365  0.50 26.24           C  \\nATOM    821  CG BLYS D  29      -9.869  20.535 -12.933  0.50 51.60           C  \\nATOM    822  CD ALYS D  29     -11.261  18.659 -11.369  0.50 25.69           C  \\nATOM    823  CD BLYS D  29     -11.094  21.211 -12.318  0.50 26.73           C  \\nATOM    824  CE ALYS D  29     -12.315  17.615 -11.670  0.50 22.42           C  \\nATOM    825  CE BLYS D  29     -12.076  21.723 -13.345  0.50 41.26           C  \\nATOM    826  NZ ALYS D  29     -13.648  18.108 -11.182  0.50 39.07           N  \\nATOM    827  NZ BLYS D  29     -13.280  22.348 -12.731  0.50 38.89           N  \\nATOM    828  N   ALA D  30      -5.874  20.552 -11.809  1.00 17.54           N  \\nATOM    829  CA  ALA D  30      -5.022  21.709 -11.876  1.00 15.58           C  \\nATOM    830  C   ALA D  30      -3.942  21.534 -12.910  1.00 35.26           C  \\nATOM    831  O   ALA D  30      -3.347  20.451 -13.025  1.00 37.09           O  \\nATOM    832  CB  ALA D  30      -4.392  21.978 -10.527  1.00 32.86           C  \\nATOM    833  OXT ALA D  30      -3.822  22.597 -13.538  1.00 43.22           O  \\nTER     834      ALA D  30                                                      \\nHETATM  835 ZN    ZN B 101      -0.002  -0.004   7.891  0.33 10.40          ZN  \\nHETATM  836 ZN    ZN D 101       0.000   0.000  -8.039  0.33 11.00          ZN  \\nHETATM  837  O   HOH A 101      -9.600  22.800   0.340  1.00 86.30           O  \\nHETATM  838  O   HOH A 102     -11.312  20.399   0.771  1.00 76.27           O  \\nHETATM  839  O   HOH A 103     -12.203  23.328   2.558  0.50 53.25           O  \\nHETATM  840  O   HOH A 104     -10.414  20.888   3.177  1.00 46.17           O  \\nHETATM  841  O   HOH A 105      -9.814  18.858  10.172  0.50 36.40           O  \\nHETATM  842  O   HOH A 106      -5.367   9.301  10.390  1.00 29.29           O  \\nHETATM  843  O   HOH A 107      -7.077  17.022  11.114  1.00 24.63           O  \\nHETATM  844  O   HOH A 108      -1.805  14.585  13.563  1.00 36.72           O  \\nHETATM  845  O   HOH A 109      -7.400  20.900  13.300  1.00 82.49           O  \\nHETATM  846  O   HOH A 110     -10.436  21.800  14.703  1.00 82.89           O  \\nHETATM  847  O   HOH A 111      -8.811  19.800  14.800  1.00 84.33           O  \\nHETATM  848  O   HOH A 112      -1.885  13.663  16.148  1.00 61.43           O  \\nHETATM  849  O   HOH A 113     -11.760  21.100  16.700  0.50 40.14           O  \\nHETATM  850  O   HOH A 114      -8.270  20.622  17.537  0.50 29.13           O  \\nHETATM  851  O   HOH A 115      -7.800  16.300  17.668  1.00 75.77           O  \\nHETATM  852  O   HOH A 116      -8.894  18.338  17.943  1.00 51.71           O  \\nHETATM  853  O   HOH A 117     -11.769  21.611  10.650  0.50 56.71           O  \\nHETATM  854  O   HOH A 118      -8.631  22.492  11.840  0.50 43.99           O  \\nHETATM  855  O   HOH A 119      -7.614  21.771  17.400  0.50 39.65           O  \\nHETATM  856  O   HOH A 120      -4.837  15.671  19.009  1.00 33.67           O  \\nHETATM  857  O   HOH A 121      -5.300  12.855  19.148  1.00 51.00           O  \\nHETATM  858  O   HOH A 122      -9.409  23.116  19.633  1.00 66.30           O  \\nHETATM  859  O   HOH A 123      -5.648  12.110  21.371  1.00 36.62           O  \\nHETATM  860  O   HOH A 124      -6.414  11.027  24.091  1.00 36.40           O  \\nHETATM  861  O   HOH A 125     -16.809  20.696   1.745  1.00 71.89           O  \\nHETATM  862  O   HOH A 126     -20.717  13.454   2.846  1.00 29.91           O  \\nHETATM  863  O   HOH A 127     -10.096  20.400  -7.963  1.00 73.01           O  \\nHETATM  864  O   HOH A 128     -16.384  20.545   4.220  1.00 62.82           O  \\nHETATM  865  O   HOH A 129     -13.362  23.086   0.118  1.00 64.26           O  \\nHETATM  866  O   HOH A 130      -8.031  12.739  18.649  0.50 27.91           O  \\nHETATM  867  O   HOH A 131     -16.226  18.618  -0.171  1.00 38.87           O  \\nHETATM  868  O   HOH B 201      -5.500  23.300   3.200  1.00 76.30           O  \\nHETATM  869  O   HOH B 202      -8.655  24.051   2.000  1.00 56.40           O  \\nHETATM  870  O   HOH B 203      -0.463   3.085   2.990  1.00 35.04           O  \\nHETATM  871  O   HOH B 204      -9.400  22.600   4.900  0.50 52.19           O  \\nHETATM  872  O   HOH B 205      -7.624  16.101   5.527  1.00 25.99           O  \\nHETATM  873  O   HOH B 206      -9.100  20.600   6.100  0.50 60.00           O  \\nHETATM  874  O   HOH B 207      -8.768  19.973   5.675  0.50 51.16           O  \\nHETATM  875  O   HOH B 208      -7.991  22.590   6.582  0.50 57.20           O  \\nHETATM  876  O   HOH B 209     -12.272  21.574   6.100  1.00 55.48           O  \\nHETATM  877  O   HOH B 210      -9.484  23.456   7.067  0.50 41.28           O  \\nHETATM  878  O   HOH B 211      -3.193  10.008   8.356  1.00 20.33           O  \\nHETATM  879  O   HOH B 212      -9.443  20.240   8.418  0.50 46.63           O  \\nHETATM  880  O   HOH B 213      -0.074   1.535   9.452  1.00 25.97           O  \\nHETATM  881  O   HOH B 214       0.475  13.399   9.000  1.00 61.33           O  \\nHETATM  882  O   HOH B 215      -8.473  18.398   9.204  0.50 34.10           O  \\nHETATM  883  O   HOH B 216       4.832  22.586  10.697  0.50 41.13           O  \\nHETATM  884  O   HOH B 217      -9.503  22.800  10.020  0.50 40.98           O  \\nHETATM  885  O   HOH B 218       3.402  22.242  10.235  0.50 38.69           O  \\nHETATM  886  O   HOH B 219      -9.990  20.281  10.186  0.50 56.19           O  \\nHETATM  887  O   HOH B 220      -6.059  22.694  11.492  1.00 61.96           O  \\nHETATM  888  O   HOH B 221       2.400   0.700  11.139  1.00 63.37           O  \\nHETATM  889  O   HOH B 222       0.589  13.731  11.296  0.50 60.20           O  \\nHETATM  890  O   HOH B 223      -0.216  11.635  11.240  0.50 52.12           O  \\nHETATM  891  O   HOH B 224      -0.023  -0.033  11.206  0.33 21.05           O  \\nHETATM  892  O   HOH B 225      -5.435  20.514  11.630  1.00 51.70           O  \\nHETATM  893  O   HOH B 226      -1.821   8.684  12.044  1.00 36.04           O  \\nHETATM  894  O   HOH B 227       4.207  21.400  12.867  0.50 42.63           O  \\nHETATM  895  O   HOH B 228      -3.059  10.929  12.998  0.50 54.91           O  \\nHETATM  896  O   HOH B 229      -2.700  22.200  12.300  1.00 69.61           O  \\nHETATM  897  O   HOH B 230       1.000  16.700  13.200  1.00 45.00           O  \\nHETATM  898  O   HOH B 231       3.947  18.704  13.926  0.50 87.22           O  \\nHETATM  899  O   HOH B 232       0.063   7.371  13.477  0.50 44.43           O  \\nHETATM  900  O   HOH B 233       4.331  21.351  15.200  1.00 48.37           O  \\nHETATM  901  O   HOH B 234      -3.119  22.865  15.776  1.00 59.24           O  \\nHETATM  902  O   HOH B 235      -5.600  22.740  16.755  0.50 60.98           O  \\nHETATM  903  O   HOH B 236      -1.700  21.227  17.206  1.00 50.32           O  \\nHETATM  904  O   HOH B 237       4.677  20.377  17.882  1.00 49.25           O  \\nHETATM  905  O   HOH B 238      -1.199  22.838  19.138  1.00 36.10           O  \\nHETATM  906  O   HOH B 239       1.686  21.194  18.890  1.00 36.62           O  \\nHETATM  907  O   HOH B 240       3.848  22.050  20.436  1.00 73.68           O  \\nHETATM  908  O   HOH B 241      -2.438  17.554  21.488  1.00 27.54           O  \\nHETATM  909  O   HOH B 242       1.801  22.097  22.791  1.00 33.66           O  \\nHETATM  910  O   HOH B 243      -8.670  18.367  -8.735  1.00 27.41           O  \\nHETATM  911  O   HOH B 244       1.208   0.917  -0.239  1.00 44.11           O  \\nHETATM  912  O   HOH B 245       0.000   0.000   2.721  0.33 78.41           O  \\nHETATM  913  O   HOH B 246       0.397   4.490  11.763  0.50 23.04           O  \\nHETATM  914  O   HOH B 247       2.938   2.622  12.785  1.00 45.76           O  \\nHETATM  915  O   HOH B 248      -8.497  11.582  -5.037  1.00 65.08           O  \\nHETATM  916  O   HOH B 249     -16.177  12.121  -3.599  0.50 17.96           O  \\nHETATM  917  O   HOH C 101      13.443  19.181   0.629  1.00 37.18           O  \\nHETATM  918  O   HOH C 102      14.910  22.149   0.107  1.00 64.34           O  \\nHETATM  919  O   HOH C 103      15.228  18.988   2.773  1.00 47.95           O  \\nHETATM  920  O   HOH C 104      -2.652  21.385   6.534  1.00 64.51           O  \\nHETATM  921  O   HOH C 105      -2.600  23.100   8.100  0.50 37.93           O  \\nHETATM  922  O   HOH C 106       6.358  23.611  -0.018  1.00 94.84           O  \\nHETATM  923  O   HOH C 107       2.469  24.709  -2.126  1.00 29.22           O  \\nHETATM  924  O   HOH D 201       1.228   4.480   7.603  1.00 34.42           O  \\nHETATM  925  O   HOH D 202       5.794  10.580   7.551  0.50 18.82           O  \\nHETATM  926  O   HOH D 203       3.540   5.186   7.514  1.00 50.94           O  \\nHETATM  927  O   HOH D 204       3.162  12.350   8.534  1.00 60.02           O  \\nHETATM  928  O   HOH D 205       5.402   7.037   8.018  1.00 76.30           O  \\nHETATM  929  O   HOH D 206       4.499   3.274   9.801  1.00 21.36           O  \\nHETATM  930  O   HOH D 207       3.575  13.150  11.256  0.50 53.19           O  \\nHETATM  931  O   HOH D 208       6.250  11.753  12.889  1.00 50.22           O  \\nHETATM  932  O   HOH D 209       4.989  14.660  13.137  1.00 52.11           O  \\nHETATM  933  O   HOH D 210       7.840  13.593  13.240  1.00 36.77           O  \\nHETATM  934  O   HOH D 211      -1.841   2.626  14.104  1.00 47.47           O  \\nHETATM  935  O   HOH D 212      -0.629   5.214  14.504  1.00 52.63           O  \\nHETATM  936  O   HOH D 213      -3.496  10.462  15.323  1.00 41.57           O  \\nHETATM  937  O   HOH D 214      -3.814  12.169  17.184  1.00 40.38           O  \\nHETATM  938  O   HOH D 215      -0.459  10.100  16.800  1.00 96.03           O  \\nHETATM  939  O   HOH D 216      -2.265   8.874  17.970  1.00 48.81           O  \\nHETATM  940  O   HOH D 217       4.304  23.913   8.300  1.00 56.36           O  \\nHETATM  941  O   HOH D 218      -3.499   9.530  21.143  1.00 58.40           O  \\nHETATM  942  O   HOH D 219      -2.158  10.172  22.900  1.00 34.73           O  \\nHETATM  943  O   HOH D 220     -12.014  22.141  -9.704  1.00 66.63           O  \\nHETATM  944  O   HOH D 221      -1.763  20.077  -8.993  1.00 21.36           O  \\nHETATM  945  O   HOH D 222      -2.718  10.300  25.150  1.00 73.39           O  \\nHETATM  946  O   HOH D 223      -1.739  18.453  -6.295  1.00 12.21           O  \\nHETATM  947  O   HOH D 224      27.252   2.224   9.433  1.00 83.01           O  \\nHETATM  948  O   HOH D 225      14.487  19.051   4.963  0.50 47.26           O  \\nHETATM  949  O   HOH D 226      15.038  21.545   5.268  1.00 55.92           O  \\nHETATM  950  O   HOH D 227       8.897  22.662   0.833  0.50 72.59           O  \\nHETATM  951  O   HOH D 228       5.430   4.632   0.353  0.50 68.27           O  \\nHETATM  952  O   HOH D 229      10.779  23.846   1.400  0.50 44.66           O  \\nHETATM  953  O   HOH D 230       3.208   6.718   2.188  1.00 44.14           O  \\nHETATM  954  O   HOH D 231       1.948   1.415   2.000  1.00 83.31           O  \\nHETATM  955  O   HOH D 232       3.289   3.437   2.898  1.00 29.40           O  \\nHETATM  956  O   HOH D 233      10.311  22.834   3.121  0.50 53.93           O  \\nHETATM  957  O   HOH D 234      10.183  24.198   3.002  0.50 47.80           O  \\nHETATM  958  O   HOH D 235      23.650   0.900   3.000  1.00 86.30           O  \\nHETATM  959  O   HOH D 236       8.256  18.783   3.330  0.50 14.13           O  \\nHETATM  960  O   HOH D 237       5.786   4.688   3.296  1.00 79.24           O  \\nHETATM  961  O   HOH D 238      12.041  23.097   3.478  0.50 31.09           O  \\nHETATM  962  O   HOH D 239       9.781  13.999   4.347  1.00 38.00           O  \\nHETATM  963  O   HOH D 240      24.800   2.000   4.680  1.00 50.76           O  \\nHETATM  964  O   HOH D 241       1.700  14.430   4.490  1.00 60.70           O  \\nHETATM  965  O   HOH D 242       4.071   5.888   5.145  1.00 60.78           O  \\nHETATM  966  O   HOH D 243      12.744  23.426   5.604  0.50 40.09           O  \\nHETATM  967  O   HOH D 244      10.354  21.084   4.829  0.50 33.95           O  \\nHETATM  968  O   HOH D 245      12.369  21.475   5.162  0.50 61.18           O  \\nHETATM  969  O   HOH D 246      13.100  13.724   6.029  1.00 21.23           O  \\nHETATM  970  O   HOH D 247      25.212   3.031   6.890  1.00 49.58           O  \\nHETATM  971  O   HOH D 248      10.600  10.100   6.800  0.50 50.00           O  \\nHETATM  972  O   HOH D 249      11.191  23.508   6.621  0.50 37.26           O  \\nHETATM  973  O   HOH D 250      14.528  16.192   7.070  1.00 51.29           O  \\nHETATM  974  O   HOH D 251       1.492  23.231   6.892  0.50 27.00           O  \\nHETATM  975  O   HOH D 252      14.502  18.882   6.980  0.50 59.76           O  \\nHETATM  976  O   HOH D 253       0.172  22.959   7.609  0.50 29.31           O  \\nHETATM  977  O   HOH D 254      10.079  20.626   8.067  0.52 33.28           O  \\nHETATM  978  O   HOH D 255       8.276  22.353   6.634  0.50 71.16           O  \\nHETATM  979  O   HOH D 256      11.200  20.200   6.480  0.50 43.34           O  \\nHETATM  980  O   HOH D 257       9.467  21.709   6.550  0.50 60.52           O  \\nHETATM  981  O   HOH D 258      12.911  19.857   7.882  0.50 41.83           O  \\nHETATM  982  O   HOH D 259      24.906   0.400   8.400  1.00 59.59           O  \\nHETATM  983  O   HOH D 260      22.962   4.308   7.971  1.00 75.46           O  \\nHETATM  984  O   HOH D 261       6.986  23.533   8.608  0.50 42.86           O  \\nHETATM  985  O   HOH D 262       7.747  22.514   8.736  0.50 51.31           O  \\nHETATM  986  O   HOH D 263      11.350  23.300   9.170  1.00 50.61           O  \\nHETATM  987  O   HOH D 264      13.640  22.500   8.400  1.00 61.09           O  \\nHETATM  988  O   HOH D 265      21.200   5.800   9.000  1.00 46.66           O  \\nHETATM  989  O   HOH D 266      -0.559  20.337   9.227  0.50 31.04           O  \\nHETATM  990  O   HOH D 267       2.956   7.356   9.255  1.00 37.48           O  \\nHETATM  991  O   HOH D 268      11.907  19.705   9.066  0.50 35.59           O  \\nHETATM  992  O   HOH D 269      16.094  16.073   9.932  1.00 48.31           O  \\nHETATM  993  O   HOH D 270      13.482  17.840   9.054  1.00 66.41           O  \\nHETATM  994  O   HOH D 271       9.900  19.400  10.176  1.00 57.45           O  \\nHETATM  995  O   HOH D 272       8.300  21.900   9.770  0.50 62.82           O  \\nHETATM  996  O   HOH D 273       5.531  14.046   9.251  1.00 53.12           O  \\nHETATM  997  O   HOH D 274       0.500  21.500   9.739  0.50 40.73           O  \\nHETATM  998  O   HOH D 275      24.500   1.839  10.261  0.50 44.08           O  \\nHETATM  999  O   HOH D 276      14.403  20.407   9.881  0.25 62.98           O  \\nHETATM 1000  O   HOH D 277       5.099  10.014  10.625  1.00 39.93           O  \\nHETATM 1001  O   HOH D 278       0.786  10.457  10.424  0.50 27.11           O  \\nHETATM 1002  O   HOH D 279       7.099  20.641  10.117  0.50 45.52           O  \\nHETATM 1003  O   HOH D 280       1.327   5.583  11.052  0.50 56.24           O  \\nHETATM 1004  O   HOH D 281      10.114  17.143  11.048  1.00 38.47           O  \\nHETATM 1005  O   HOH D 282      13.122  21.921  10.724  0.50 29.51           O  \\nHETATM 1006  O   HOH D 283      24.000   4.400  11.200  1.00 63.61           O  \\nHETATM 1007  O   HOH D 284      20.200   6.301  11.100  1.00 62.49           O  \\nHETATM 1008  O   HOH D 285       1.431   8.001  10.900  1.00 61.63           O  \\nHETATM 1009  O   HOH D 286       3.725   4.937  12.004  1.00 60.07           O  \\nHETATM 1010  O   HOH D 287       9.246  23.287  11.503  1.00 45.58           O  \\nHETATM 1011  O   HOH D 288      13.713  17.818  11.963  1.00 66.30           O  \\nHETATM 1012  O   HOH D 289      16.125  12.139  12.240  1.00 50.03           O  \\nHETATM 1013  O   HOH D 290      12.904  20.420  11.491  0.50 30.82           O  \\nHETATM 1014  O   HOH D 291      22.345   2.177  11.300  1.00 53.69           O  \\nHETATM 1015  O   HOH D 292       3.200   8.800  12.200  1.00 58.95           O  \\nHETATM 1016  O   HOH D 293      12.623  23.254  12.208  0.50 36.01           O  \\nHETATM 1017  O   HOH D 294       6.083  21.437  12.952  0.50 45.78           O  \\nHETATM 1018  O   HOH D 295      15.629  16.552  13.704  1.00 54.02           O  \\nHETATM 1019  O   HOH D 296       1.400  10.000  13.205  1.00 62.38           O  \\nHETATM 1020  O   HOH D 297      10.681  15.953  13.460  1.00 37.62           O  \\nHETATM 1021  O   HOH D 298       3.526  17.510  13.393  0.50 58.59           O  \\nHETATM 1022  O   HOH D 299      17.589  14.886  12.994  0.50 46.92           O  \\nHETATM 1023  O   HOH D 300       8.436  16.103  13.124  0.50 66.40           O  \\nHETATM 1024  O   HOH D 301      20.767   2.758  14.127  0.50 49.50           O  \\nHETATM 1025  O   HOH D 302      19.461   3.175  13.236  0.50 43.30           O  \\nHETATM 1026  O   HOH D 303      10.600  21.200  12.800  1.00 65.41           O  \\nHETATM 1027  O   HOH D 304      17.091  12.037  14.424  1.00 66.30           O  \\nHETATM 1028  O   HOH D 305      26.230   1.151  12.982  0.50 42.97           O  \\nHETATM 1029  O   HOH D 306       8.200  19.886  13.338  1.00 59.62           O  \\nHETATM 1030  O   HOH D 307       3.000   8.250  14.500  1.00 39.38           O  \\nHETATM 1031  O   HOH D 308      23.669   1.157  13.223  1.00 49.56           O  \\nHETATM 1032  O   HOH D 309       6.706  17.656  14.200  1.00 62.44           O  \\nHETATM 1033  O   HOH D 310      21.660   8.100  14.400  1.00 78.85           O  \\nHETATM 1034  O   HOH D 311      16.002  14.165  14.204  0.50 36.21           O  \\nHETATM 1035  O   HOH D 312      18.624   9.971  14.472  1.00 68.09           O  \\nHETATM 1036  O   HOH D 313      12.345  18.363  14.320  1.00 61.52           O  \\nHETATM 1037  O   HOH D 314      15.375  13.331  15.234  0.50 38.87           O  \\nHETATM 1038  O   HOH D 315       8.900  16.074  15.111  0.50 56.02           O  \\nHETATM 1039  O   HOH D 316       9.800  20.500  15.000  0.50 37.74           O  \\nHETATM 1040  O   HOH D 317      20.471   6.314  15.326  0.50 42.47           O  \\nHETATM 1041  O   HOH D 318      15.956  10.347  15.577  1.00 73.09           O  \\nHETATM 1042  O   HOH D 319       4.166  16.122  15.117  1.00 64.59           O  \\nHETATM 1043  O   HOH D 320      22.179   1.588  16.322  1.00 60.14           O  \\nHETATM 1044  O   HOH D 321       7.227  21.575  16.034  1.00 79.51           O  \\nHETATM 1045  O   HOH D 322      24.987   3.024  15.000  0.50 77.50           O  \\nHETATM 1046  O   HOH D 323       0.000   0.000  15.940  0.33 48.19           O  \\nHETATM 1047  O   HOH D 324       2.058   1.124  15.886  0.50 57.48           O  \\nHETATM 1048  O   HOH D 325      19.200   4.000  16.256  1.00 52.18           O  \\nHETATM 1049  O   HOH D 326      14.044  21.773  15.730  0.50 35.08           O  \\nHETATM 1050  O   HOH D 327       1.900   3.400  16.729  1.00 42.63           O  \\nHETATM 1051  O   HOH D 328      14.400  17.800  15.886  0.50 52.96           O  \\nHETATM 1052  O   HOH D 329      13.700  15.600  15.000  1.00 65.01           O  \\nHETATM 1053  O   HOH D 330       2.100   6.200  15.800  1.00 73.63           O  \\nHETATM 1054  O   HOH D 331      13.105  18.675  16.692  0.50 57.75           O  \\nHETATM 1055  O   HOH D 332       5.384   1.947  16.328  1.00 52.83           O  \\nHETATM 1056  O   HOH D 333      19.387   1.649  16.538  1.00 38.74           O  \\nHETATM 1057  O   HOH D 334       6.570  18.429  16.505  1.00 50.65           O  \\nHETATM 1058  O   HOH D 335      18.269   7.536  16.565  1.00 60.44           O  \\nHETATM 1059  O   HOH D 336      16.961  12.601  17.169  1.00 81.43           O  \\nHETATM 1060  O   HOH D 337      19.266  10.576  17.790  1.00 63.02           O  \\nHETATM 1061  O   HOH D 338      -2.564   5.638  17.137  1.00 48.33           O  \\nHETATM 1062  O   HOH D 339      21.336   8.539  16.849  1.00 46.39           O  \\nHETATM 1063  O   HOH D 340      11.361  23.312  16.935  1.00 56.56           O  \\nHETATM 1064  O   HOH D 341      11.482  20.115  16.180  0.50 36.42           O  \\nHETATM 1065  O   HOH D 342      10.431  19.608  16.704  0.50 61.59           O  \\nHETATM 1066  O   HOH D 343      22.701   4.882  16.533  0.50 59.04           O  \\nHETATM 1067  O   HOH D 344      -0.559   7.125  16.712  1.00 55.39           O  \\nHETATM 1068  O   HOH D 345      22.349   3.895  16.628  0.50 44.55           O  \\nHETATM 1069  O   HOH D 346      24.254   2.529  17.221  0.50 64.98           O  \\nHETATM 1070  O   HOH D 347      17.313  15.074  17.581  1.00 53.72           O  \\nHETATM 1071  O   HOH D 348      16.509   9.562  17.812  1.00 54.16           O  \\nHETATM 1072  O   HOH D 349      17.535   3.747  17.792  1.00 51.88           O  \\nHETATM 1073  O   HOH D 350       0.500   3.000  18.600  0.50 35.00           O  \\nHETATM 1074  O   HOH D 351       6.000   4.100  17.800  1.00 50.00           O  \\nHETATM 1075  O   HOH D 352       2.700   3.900  19.000  0.50 40.00           O  \\nHETATM 1076  O   HOH D 353       2.992   5.257  18.400  0.50 40.00           O  \\nHETATM 1077  O   HOH D 354      22.320   6.616  18.376  1.00 76.30           O  \\nHETATM 1078  O   HOH D 355      14.300  15.147  17.400  0.50 41.41           O  \\nHETATM 1079  O   HOH D 356      12.379  14.000  18.485  1.00 69.21           O  \\nHETATM 1080  O   HOH D 357      26.585   1.034  18.302  1.00 72.39           O  \\nHETATM 1081  O   HOH D 358      24.664   3.165  18.640  0.50 55.98           O  \\nHETATM 1082  O   HOH D 359      16.000   6.400  18.500  1.00 75.03           O  \\nHETATM 1083  O   HOH D 360       7.587  19.898  18.900  1.00 74.10           O  \\nHETATM 1084  O   HOH D 361      23.636   0.400  18.433  0.50 38.58           O  \\nHETATM 1085  O   HOH D 362       7.200   0.000  19.364  1.00 80.00           O  \\nHETATM 1086  O   HOH D 363      12.325  20.187  18.584  1.00 76.30           O  \\nHETATM 1087  O   HOH D 364       8.868  21.918  18.717  1.00 88.44           O  \\nHETATM 1088  O   HOH D 365       4.000   0.889  19.161  1.00 56.19           O  \\nHETATM 1089  O   HOH D 366      11.500  16.200  18.500  1.00 54.59           O  \\nHETATM 1090  O   HOH D 367      15.033  14.600  19.135  0.50 40.58           O  \\nHETATM 1091  O   HOH D 368      16.700  11.300  19.500  0.50 53.18           O  \\nHETATM 1092  O   HOH D 369      17.595   8.307  19.856  1.00 43.95           O  \\nHETATM 1093  O   HOH D 370       8.460  17.052  19.897  1.00 58.87           O  \\nHETATM 1094  O   HOH D 371      11.600  18.400  20.000  1.00 60.00           O  \\nHETATM 1095  O   HOH D 372       7.200   3.000  19.668  1.00 79.22           O  \\nHETATM 1096  O   HOH D 373      21.000   4.000  19.956  1.00 59.41           O  \\nHETATM 1097  O   HOH D 374      -0.800   4.800  19.700  1.00 50.00           O  \\nHETATM 1098  O   HOH D 375      13.592   4.017  20.054  1.00 43.20           O  \\nHETATM 1099  O   HOH D 376      -0.400   2.600  20.000  0.50 40.00           O  \\nHETATM 1100  O   HOH D 377      19.480   6.217  19.797  1.00 62.99           O  \\nHETATM 1101  O   HOH D 378      20.876   1.574  20.116  0.50 41.62           O  \\nHETATM 1102  O   HOH D 379      18.870   3.203  20.295  1.00 65.25           O  \\nHETATM 1103  O   HOH D 380      22.043   8.448  20.602  0.50 38.02           O  \\nHETATM 1104  O   HOH D 381      15.420   7.740  20.813  1.00 61.77           O  \\nHETATM 1105  O   HOH D 382      14.400  19.900  20.800  1.00 46.45           O  \\nHETATM 1106  O   HOH D 383       8.900   4.300  20.900  1.00 66.62           O  \\nHETATM 1107  O   HOH D 384      12.900   7.700  20.600  1.00 50.81           O  \\nHETATM 1108  O   HOH D 385      23.373   3.907  20.784  0.50 37.81           O  \\nHETATM 1109  O   HOH D 386      22.600   5.200  21.000  0.50 50.00           O  \\nHETATM 1110  O   HOH D 387      11.000  20.300  21.100  0.50 50.00           O  \\nHETATM 1111  O   HOH D 388      15.900  13.400  20.500  0.50 37.02           O  \\nHETATM 1112  O   HOH D 389      15.000  15.900  20.409  0.50 51.72           O  \\nHETATM 1113  O   HOH D 390      11.455   5.672  21.400  0.50 45.30           O  \\nHETATM 1114  O   HOH D 391      19.300  11.200  21.251  1.00 93.71           O  \\nHETATM 1115  O   HOH D 392      12.867  15.311  21.383  1.00104.59           O  \\nHETATM 1116  O   HOH D 393      16.200  18.500  20.516  0.50 57.42           O  \\nHETATM 1117  O   HOH D 394      17.900  14.200  21.400  0.50 54.11           O  \\nHETATM 1118  O   HOH D 395       2.426   2.789  21.667  0.50 37.21           O  \\nHETATM 1119  O   HOH D 396       5.591   3.514  22.644  1.00 66.58           O  \\nHETATM 1120  O   HOH D 397      21.490   9.784  21.789  0.50 37.24           O  \\nHETATM 1121  O   HOH D 398      22.459   7.099  21.906  0.50 54.11           O  \\nHETATM 1122  O   HOH D 399      23.362   5.246  22.116  0.50 53.14           O  \\nHETATM 1123  O   HOH D 400      10.026   5.995  22.154  0.50 36.85           O  \\nHETATM 1124  O   HOH D 401      16.230  16.756  22.900  1.00 51.60           O  \\nHETATM 1125  O   HOH D 402       7.950   1.500  22.400  1.00 74.32           O  \\nHETATM 1126  O   HOH D 403       0.000   0.000  22.203  0.33 26.53           O  \\nHETATM 1127  O   HOH D 404       6.000   0.331  22.809  1.00 81.99           O  \\nHETATM 1128  O   HOH D 405       2.400   1.900  22.950  0.50 53.75           O  \\nHETATM 1129  O   HOH D 406      16.826   9.360  22.915  1.00 64.50           O  \\nHETATM 1130  O   HOH D 407      10.028   0.600  22.667  1.00 50.88           O  \\nHETATM 1131  O   HOH D 408       8.566  22.210  23.275  1.00 52.52           O  \\nHETATM 1132  O   HOH D 409      19.041   7.317  22.965  1.00 45.49           O  \\nHETATM 1133  O   HOH D 410      13.154  17.896  23.121  0.50 58.99           O  \\nHETATM 1134  O   HOH D 411      25.120   2.792  23.670  1.00 49.60           O  \\nHETATM 1135  O   HOH D 412       0.848   2.630  23.300  0.50 51.96           O  \\nHETATM 1136  O   HOH D 413      12.799   4.914  23.300  1.00 45.19           O  \\nHETATM 1137  O   HOH D 414      14.100   1.511  23.000  1.00 54.91           O  \\nHETATM 1138  O   HOH D 415      18.000   5.200  23.400  0.50 40.00           O  \\nHETATM 1139  O   HOH D 416      20.460   5.246  23.195  0.50 42.74           O  \\nHETATM 1140  O   HOH D 417      18.500  13.238  23.926  0.50 60.34           O  \\nHETATM 1141  O   HOH D 418       9.469   4.007  23.263  0.50 65.39           O  \\nHETATM 1142  O   HOH D 419       3.330   4.805  23.757  1.00 24.03           O  \\nHETATM 1143  O   HOH D 420      -1.798   3.543  24.064  1.00 80.64           O  \\nHETATM 1144  O   HOH D 421       1.468   0.225  24.184  1.00 35.46           O  \\nHETATM 1145  O   HOH D 422      13.052  19.509  23.128  0.50 62.84           O  \\nHETATM 1146  O   HOH D 423       9.900  22.890  25.259  1.00 65.26           O  \\nHETATM 1147  O   HOH D 424      16.495  17.126  26.000  1.00 40.00           O  \\nHETATM 1148  O   HOH D 425      21.700   5.400  24.514  0.50 50.22           O  \\nHETATM 1149  O   HOH D 426       6.642  20.679  24.898  1.00 58.56           O  \\nHETATM 1150  O   HOH D 427      20.692   8.311  24.457  0.50 48.03           O  \\nHETATM 1151  O   HOH D 428       7.700   4.223  25.055  1.00 33.74           O  \\nHETATM 1152  O   HOH D 429      19.581  11.070  25.093  1.00 65.09           O  \\nHETATM 1153  O   HOH D 430      22.450   8.700  25.159  0.50 48.15           O  \\nHETATM 1154  O   HOH D 431      12.210  21.699  25.848  0.50 44.27           O  \\nHETATM 1155  O   HOH D 432      11.500  21.600  24.157  0.50 70.69           O  \\nHETATM 1156  O   HOH D 433      25.803   0.095  26.600  1.00 56.59           O  \\nHETATM 1157  O   HOH D 434      23.392   1.295  25.732  1.00 48.54           O  \\nHETATM 1158  O   HOH D 435      12.194   3.114  25.000  1.00 76.30           O  \\nHETATM 1159  O   HOH D 436       4.708   1.340  26.039  1.00 30.59           O  \\nHETATM 1160  O   HOH D 437      14.388  19.737  25.636  1.00 64.76           O  \\nHETATM 1161  O   HOH D 438       5.419   3.667  26.434  1.00 31.12           O  \\nHETATM 1162  O   HOH D 439      23.512   4.500  26.629  1.00 66.30           O  \\nHETATM 1163  O   HOH D 440      16.389  14.064  27.171  1.00 74.53           O  \\nHETATM 1164  O   HOH D 441      18.700  14.000  26.532  1.00 79.38           O  \\nHETATM 1165  O   HOH D 442      10.926  10.684  -7.060  1.00 44.80           O  \\nHETATM 1166  O   HOH D 443      12.500  21.800  27.254  0.50 47.63           O  \\nHETATM 1167  O   HOH D 444      27.000   1.000  28.300  1.00 70.92           O  \\nHETATM 1168  O   HOH D 445      20.879   8.934  27.397  0.50 49.09           O  \\nHETATM 1169  O   HOH D 446       5.301   3.962  28.700  0.60 31.54           O  \\nHETATM 1170  O   HOH D 447      13.794  13.317  28.059  1.00 74.41           O  \\nHETATM 1171  O   HOH D 448       6.578   4.370  28.779  0.40 28.91           O  \\nHETATM 1172  O   HOH D 449      19.254  14.024  28.832  1.00 55.11           O  \\nHETATM 1173  O   HOH D 450      14.080   9.849  -4.521  1.00 27.58           O  \\nHETATM 1174  O   HOH D 451      23.137   6.636  29.293  0.50 55.90           O  \\nHETATM 1175  O   HOH D 452      10.475  22.332  30.034  1.00 41.35           O  \\nHETATM 1176  O   HOH D 453      14.200  22.300  30.301  1.00 67.11           O  \\nHETATM 1177  O   HOH D 454       4.564   5.544  30.131  1.00 37.24           O  \\nHETATM 1178  O   HOH D 455       0.000   0.000  30.941  0.33 24.09           O  \\nHETATM 1179  O   HOH D 456       3.559   3.107  30.844  1.00 34.61           O  \\nHETATM 1180  O   HOH D 457      -1.317   2.461  30.833  1.00 27.74           O  \\nHETATM 1181  O   HOH D 458       6.900   3.468  31.640  1.00 38.60           O  \\nHETATM 1182  O   HOH D 459       1.484   2.379  31.639  1.00 81.34           O  \\nHETATM 1183  O   HOH D 460      12.200  21.561  31.604  1.00 55.36           O  \\nHETATM 1184  O   HOH D 461     -11.275  25.048 -12.333  1.00 77.73           O  \\nHETATM 1185  O   HOH D 462      11.090  22.753  33.200  1.00 96.30           O  \\nHETATM 1186  O   HOH D 463       3.296   3.651  33.200  1.00 86.30           O  \\nCONECT   43   76                                                                \\nCONECT   49  227                                                                \\nCONECT   76   43                                                                \\nCONECT  154  319                                                                \\nCONECT  227   49                                                                \\nCONECT  247  835                                                                \\nCONECT  319  154                                                                \\nCONECT  464  497                                                                \\nCONECT  470  644                                                                \\nCONECT  497  464                                                                \\nCONECT  575  734                                                                \\nCONECT  644  470                                                                \\nCONECT  664  836                                                                \\nCONECT  734  575                                                                \\nCONECT  835  247  880                                                           \\nCONECT  836  664                                                                \\nCONECT  880  835                                                                \\nMASTER      477    0    2    6    2    0   14    9 1158    4   17   10          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_17539308553244367.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_17539308553244367.zoomTo();\nviewer_17539308553244367.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_17539308553244367\"  style=\"position: relative; width: 1000px; height: 300px;\">\n",
       "        <p id=\"3dmolwarning_17539308553244367\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.1/build/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_17539308553244367 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_17539308553244367\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_17539308553244367 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17539308553244367\"),{backgroundColor:\"white\"});\n",
       "viewer_17539308553244367.zoomTo();\n",
       "\tviewer_17539308553244367.addModel(\"HEADER    HORMONE                                 10-JUL-89   4INS              \\nTITLE     THE STRUCTURE OF 2ZN PIG INSULIN CRYSTALS AT 1.5 ANGSTROMS RESOLUTION \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: INSULIN (CHAIN A);                                         \\nCOMPND   3 CHAIN: A, C;                                                         \\nCOMPND   4 ENGINEERED: YES;                                                     \\nCOMPND   5 MOL_ID: 2;                                                           \\nCOMPND   6 MOLECULE: INSULIN (CHAIN B);                                         \\nCOMPND   7 CHAIN: B, D;                                                         \\nCOMPND   8 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: SUS SCROFA;                                     \\nSOURCE   3 ORGANISM_COMMON: PIG;                                                \\nSOURCE   4 ORGANISM_TAXID: 9823;                                                \\nSOURCE   5 MOL_ID: 2;                                                           \\nSOURCE   6 ORGANISM_SCIENTIFIC: SUS SCROFA;                                     \\nSOURCE   7 ORGANISM_COMMON: PIG;                                                \\nSOURCE   8 ORGANISM_TAXID: 9823                                                 \\nKEYWDS    HORMONE                                                               \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    G.G.DODSON,E.J.DODSON,D.C.HODGKIN,N.W.ISAACS,M.VIJAYAN                \\nREVDAT   8   23-OCT-24 4INS    1       LINK                                     \\nREVDAT   7   29-NOV-17 4INS    1       HELIX                                    \\nREVDAT   6   29-FEB-12 4INS    1       JRNL   VERSN                             \\nREVDAT   5   24-FEB-09 4INS    1       VERSN                                    \\nREVDAT   4   01-APR-03 4INS    1       JRNL                                     \\nREVDAT   3   31-JUL-94 4INS    3       HETATM                                   \\nREVDAT   2   15-JUL-93 4INS    1       HEADER                                   \\nREVDAT   1   15-APR-90 4INS    0                                                \\nSPRSDE     15-APR-90 4INS      1INS                                             \\nJRNL        AUTH   E.N.BAKER,T.L.BLUNDELL,J.F.CUTFIELD,S.M.CUTFIELD,E.J.DODSON, \\nJRNL        AUTH 2 G.G.DODSON,D.M.HODGKIN,R.E.HUBBARD,N.W.ISAACS,C.D.REYNOLDS,  \\nJRNL        AUTH 3 K.SAKABE,N.SAKABE,N.M.VIJAYAN                                \\nJRNL        TITL   THE STRUCTURE OF 2ZN PIG INSULIN CRYSTALS AT 1.5 A           \\nJRNL        TITL 2 RESOLUTION.                                                  \\nJRNL        REF    PHILOS.TRANS.R.SOC.LONDON,    V. 319   369 1988              \\nJRNL        REF  2 SER.B                                                        \\nJRNL        REFN                   ISSN 0080-4622                               \\nJRNL        PMID   2905485                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   J.BORDAS,G.G.DODSON,H.GREWE,M.H.J.KOCH,B.KREBS,J.RANDALL     \\nREMARK   1  TITL   A COMPARATIVE ASSESSMENT OF THE ZINC-PROTEIN COORDINATION IN \\nREMARK   1  TITL 2 2ZN-INSULIN AS DETERMINED BY X-RAY ABSORPTION FINE STRUCTURE \\nREMARK   1  TITL 3 (EXAFS) AND X-RAY CRYSTALLOGRAPHY                            \\nREMARK   1  REF    PROC.R.SOC.LONDON,SER.B       V. 219    21 1983              \\nREMARK   1  REFN                   ISSN 0080-4649                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   E.J.DODSON,G.G.DODSON,D.C.HODGKIN,C.D.REYNOLDS               \\nREMARK   1  TITL   STRUCTURAL RELATIONSHIPS IN THE TWO-ZINC INSULIN HEXAMER     \\nREMARK   1  REF    CAN.J.BIOCHEM.                V.  57   469 1979              \\nREMARK   1  REFN                   ISSN 0008-4018                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   N.W.ISAACS,R.C.AGARWAL                                       \\nREMARK   1  TITL   EXPERIENCE WITH FAST FOURIER LEAST SQUARES IN THE REFINEMENT \\nREMARK   1  TITL 2 OF THE CRYSTAL STRUCTURE OF RHOMBOHEDRAL 2-ZINC INSULIN AT   \\nREMARK   1  TITL 3 1.5 ANGSTROMS RESOLUTION                                     \\nREMARK   1  REF    ACTA CRYSTALLOGR.,SECT.A      V.  34   782 1978              \\nREMARK   1  REFN                   ISSN 0108-7673                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   G.BENTLEY,G.DODSON,A.LEWITOVA                                \\nREMARK   1  TITL   RHOMBOHEDRAL INSULIN CRYSTAL TRANSFORMATION                  \\nREMARK   1  REF    J.MOL.BIOL.                   V. 126   871 1978              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  AUTH   E.J.DODSON,N.W.ISAACS,J.S.ROLLETT                            \\nREMARK   1  TITL   A METHOD FOR FITTING SATISFACTORY MODELS TO SETS OF ATOMIC   \\nREMARK   1  TITL 2 POSITIONS IN PROTEIN STRUCTURE REFINEMENTS                   \\nREMARK   1  REF    ACTA CRYSTALLOGR.,SECT.A      V.  32   311 1976              \\nREMARK   1  REFN                   ISSN 0108-7673                               \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  AUTH   D.C.HODGKIN                                                  \\nREMARK   1  TITL   VARIETIES OF INSULIN                                         \\nREMARK   1  REF    J.ENDOCRINOL.                 V.  63     1 1974              \\nREMARK   1  REFN                   ISSN 0022-0795                               \\nREMARK   1 REFERENCE 7                                                          \\nREMARK   1  AUTH   D.C.HODGKIN                                                  \\nREMARK   1  TITL   THE STRUCTURE OF INSULIN                                     \\nREMARK   1  REF    DAN.TIDSSKR.FARM.             V.  46     1 1972              \\nREMARK   1  REFN                   ISSN 0011-6513                               \\nREMARK   1 REFERENCE 8                                                          \\nREMARK   1  AUTH   T.BLUNDELL,G.DODSON,D.HODGKIN,D.MERCOLA                      \\nREMARK   1  TITL   INSULIN. THE STRUCTURE IN THE CRYSTAL AND ITS REFLECTION IN  \\nREMARK   1  TITL 2 CHEMISTRY AND BIOLOGY                                        \\nREMARK   1  REF    ADV.PROTEIN CHEM.             V.  26   279 1972              \\nREMARK   1  REFN                   ISSN 0065-3233                               \\nREMARK   1 REFERENCE 9                                                          \\nREMARK   1  AUTH   T.L.BLUNDELL,J.F.CUTFIELD,E.J.DODSON,G.G.DODSON,D.C.HODGKIN, \\nREMARK   1  AUTH 2 D.A.MERCOLA                                                  \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF RHOMBOHEDRAL 2 ZINC INSULIN         \\nREMARK   1  REF    COLD SPRING HARBOR            V.  36   233 1972              \\nREMARK   1  REF  2 SYMP.QUANT.BIOL.                                             \\nREMARK   1  REFN                   ISSN 0091-7451                               \\nREMARK   1 REFERENCE 10                                                         \\nREMARK   1  AUTH   T.L.BLUNDELL,J.F.CUTFIELD,S.M.CUTFIELD,E.J.DODSON,           \\nREMARK   1  AUTH 2 G.G.DODSON,D.C.HODGKIN,D.A.MERCOLA,M.VIJAYAN                 \\nREMARK   1  TITL   ATOMIC POSITIONS IN RHOMBOHEDRAL 2-ZINC INSULIN CRYSTALS     \\nREMARK   1  REF    NATURE                        V. 231   506 1971              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 11                                                         \\nREMARK   1  AUTH   T.L.BLUNDELL,G.G.DODSON,E.DODSON,D.C.HODGKIN,M.VIJAYAN       \\nREMARK   1  TITL   X-RAY ANALYSIS AND THE STRUCTURE OF INSULIN                  \\nREMARK   1  REF    RECENT PROG.HORM.RES.         V.  27     1 1971              \\nREMARK   1  REFN                   ISSN 0079-9963                               \\nREMARK   1 REFERENCE 12                                                         \\nREMARK   1  AUTH   E.N.BAKER,G.DODSON                                           \\nREMARK   1  TITL   X-RAY DIFFRACTION DATA ON SOME CRYSTALLINE VARIETIES OF      \\nREMARK   1  TITL 2 INSULIN                                                      \\nREMARK   1  REF    J.MOL.BIOL.                   V.  54   605 1970              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 13                                                         \\nREMARK   1  AUTH   M.J.ADAMS,T.L.BLUNDELL,E.J.DODSON,G.G.DODSON,M.VIJAYAN,      \\nREMARK   1  AUTH 2 E.N.BAKER,M.M.HARDING,D.C.HODGKIN,B.RIMMER,S.SHEAT           \\nREMARK   1  TITL   STRUCTURE OF RHOMBOHEDRAL 2 ZINC INSULIN CRYSTALS            \\nREMARK   1  REF    NATURE                        V. 224   491 1969              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 14                                                         \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5   187 1972              \\nREMARK   1  REF  2 AND STRUCTURE (DATA SECTION)                                 \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                                                                \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.50 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PROLSQ                                               \\nREMARK   3   AUTHORS     : KONNERT,HENDRICKSON                                  \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.50                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.153                           \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT/AGREEMENT OF MODEL WITH ALL DATA.                               \\nREMARK   3   R VALUE   (WORKING + TEST SET, NO CUTOFF) : NULL                   \\nREMARK   3   R VALUE          (WORKING SET, NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE                  (NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE TEST SET SIZE (%, NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE TEST SET COUNT   (NO CUTOFF) : NULL                   \\nREMARK   3   TOTAL NUMBER OF REFLECTIONS   (NO CUTOFF) : NULL                   \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 806                                     \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 2                                       \\nREMARK   3   SOLVENT ATOMS            : 350                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   DISTANCE RESTRAINTS.                    RMS    SIGMA               \\nREMARK   3    BOND LENGTH                     (A) : 0.005 ; NULL                \\nREMARK   3    ANGLE DISTANCE                  (A) : NULL  ; NULL                \\nREMARK   3    INTRAPLANAR 1-4 DISTANCE        (A) : NULL  ; NULL                \\nREMARK   3    H-BOND OR METAL COORDINATION    (A) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   PLANE RESTRAINT                  (A) : NULL  ; NULL                \\nREMARK   3   CHIRAL-CENTER RESTRAINT       (A**3) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   NON-BONDED CONTACT RESTRAINTS.                                     \\nREMARK   3    SINGLE TORSION                  (A) : NULL  ; NULL                \\nREMARK   3    MULTIPLE TORSION                (A) : NULL  ; NULL                \\nREMARK   3    H-BOND (X...Y)                  (A) : NULL  ; NULL                \\nREMARK   3    H-BOND (X-H...Y)                (A) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   CONFORMATIONAL TORSION ANGLE RESTRAINTS.                           \\nREMARK   3    SPECIFIED                 (DEGREES) : NULL  ; NULL                \\nREMARK   3    PLANAR                    (DEGREES) : NULL  ; NULL                \\nREMARK   3    STAGGERED                 (DEGREES) : NULL  ; NULL                \\nREMARK   3    TRANSVERSE                (DEGREES) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND               (A**2) : NULL  ; NULL                \\nREMARK   3   MAIN-CHAIN ANGLE              (A**2) : NULL  ; NULL                \\nREMARK   3   SIDE-CHAIN BOND               (A**2) : NULL  ; NULL                \\nREMARK   3   SIDE-CHAIN ANGLE              (A**2) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  SOME RESIDUES ARE APPARENTLY DISORDERED BUT DIFFICULT TO            \\nREMARK   3  DESCRIBE IN TERMS OF ATOMIC POSITIONS.  ALA B 30 IS ONE OF          \\nREMARK   3  THESE RESIDUES.                                                     \\nREMARK   3                                                                      \\nREMARK   3  THE FOLLOWING RESIDUES ARE DISORDERED - GLN B 4, VAL B 12,          \\nREMARK   3  GLU B 21, ARG B 22, ARG D 22, LYS D 29.                             \\nREMARK   4                                                                      \\nREMARK   4 4INS COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179350.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 36.05                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 1.92                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: H 3                              \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z                                                \\nREMARK 290       3555   -X+Y,-X,Z                                               \\nREMARK 290       4555   X+2/3,Y+1/3,Z+1/3                                       \\nREMARK 290       5555   -Y+2/3,X-Y+1/3,Z+1/3                                    \\nREMARK 290       6555   -X+Y+2/3,-X+1/3,Z+1/3                                   \\nREMARK 290       7555   X+1/3,Y+2/3,Z+2/3                                       \\nREMARK 290       8555   -Y+1/3,X-Y+2/3,Z+2/3                                    \\nREMARK 290       9555   -X+Y+1/3,-X+2/3,Z+2/3                                   \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       41.25000            \\nREMARK 290   SMTRY2   4  0.000000  1.000000  0.000000       23.81570            \\nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000       11.33333            \\nREMARK 290   SMTRY1   5 -0.500000 -0.866025  0.000000       41.25000            \\nREMARK 290   SMTRY2   5  0.866025 -0.500000  0.000000       23.81570            \\nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000       11.33333            \\nREMARK 290   SMTRY1   6 -0.500000  0.866025  0.000000       41.25000            \\nREMARK 290   SMTRY2   6 -0.866025 -0.500000  0.000000       23.81570            \\nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       11.33333            \\nREMARK 290   SMTRY1   7  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000       47.63140            \\nREMARK 290   SMTRY3   7  0.000000  0.000000  1.000000       22.66667            \\nREMARK 290   SMTRY1   8 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   8  0.866025 -0.500000  0.000000       47.63140            \\nREMARK 290   SMTRY3   8  0.000000  0.000000  1.000000       22.66667            \\nREMARK 290   SMTRY1   9 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   9 -0.866025 -0.500000  0.000000       47.63140            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       22.66667            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2, 3, 4, 5, 6, 7                                     \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 300 REMARK: THE CRYSTALLOGRAPHIC ASYMMETRIC UNIT OF INSULIN CONSISTS OF  \\nREMARK 300 TWO INSULIN MOLECULES EACH CONSISTING OF TWO CHAINS.  THIS           \\nREMARK 300 ENTRY PRESENTS COORDINATES FOR MOLECULES I (CHAIN                    \\nREMARK 300 INDICATORS *A* AND *B*) AND II (CHAIN INDICATORS *C* AND             \\nREMARK 300 *D*).  THE QUASI-TWO-FOLD AXIS THAT TRANSFORMS MOLECULE I            \\nREMARK 300 INTO MOLECULE II IS GIVEN IN THE *MTRIX* RECORDS BELOW.              \\nREMARK 300 APPLYING THE THREE-FOLD CRYSTALLOGRAPHIC AXIS YIELDS A               \\nREMARK 300 HEXAMER AROUND THE AXIS.  THERE ARE TWO ZINC IONS SITUATED           \\nREMARK 300 ON THIS THREE-FOLD AXIS.  COORDINATES FOR THE ZINC IONS AND          \\nREMARK 300 SOME WATER MOLECULES ARE INCLUDED BELOW WITH A BLANK CHAIN           \\nREMARK 300 INDICATOR.                                                           \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 1680 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 3790 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -15.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 1740 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 3620 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -15.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: C, D                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 3                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DODECAMERIC                \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 20600 ANGSTROM**2                         \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 12080 ANGSTROM**2                       \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -260.0 KCAL/MOL                       \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B, C, D                            \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 4                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: HEXAMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 5730 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 10440 ANGSTROM**2                       \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -95.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: C, D                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 5                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: HEXAMERIC                  \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 5580 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 10930 ANGSTROM**2                       \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -95.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 6                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: TETRAMERIC                 \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 5120 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 5710 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -45.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: C, D                                  \\nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 7                                                       \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: TETRAMERIC                 \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 4820 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 6010 ANGSTROM**2                        \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -40.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B, C, D                            \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 375                                                                      \\nREMARK 375 SPECIAL POSITION                                                     \\nREMARK 375 THE FOLLOWING ATOMS ARE FOUND TO BE WITHIN 0.15 ANGSTROMS            \\nREMARK 375 OF A SYMMETRY RELATED ATOM AND ARE ASSUMED TO BE ON SPECIAL          \\nREMARK 375 POSITIONS.                                                           \\nREMARK 375                                                                      \\nREMARK 375 ATOM RES CSSEQI                                                      \\nREMARK 375 ZN    ZN B 101  LIES ON A SPECIAL POSITION.                          \\nREMARK 375 ZN    ZN D 101  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH B 224  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH B 245  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH D 323  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH D 403  LIES ON A SPECIAL POSITION.                          \\nREMARK 375      HOH D 455  LIES ON A SPECIAL POSITION.                          \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              ZN B 101  ZN                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  10   NE2                                                    \\nREMARK 620 2 HIS B  10   NE2  98.9                                              \\nREMARK 620 3 HIS B  10   NE2  98.7  98.8                                        \\nREMARK 620 4 HOH B 213   O    90.2 163.2  93.7                                  \\nREMARK 620 N                    1     2     3                                   \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              ZN D 101  ZN                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS D  10   NE2                                                    \\nREMARK 620 2 HIS D  10   NE2 103.4                                              \\nREMARK 620 3 HIS D  10   NE2 103.4 103.4                                        \\nREMARK 620 N                    1     2                                         \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: D1                                                  \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: DIMER-FORMING RESIDUES IN MOLECULE I               \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: D2                                                  \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: DIMER-FORMING RESIDUES IN MOLECULE II              \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: H1                                                  \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: HEXAMER-FORMING RESIDUES IN MOLECULE I             \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: H2                                                  \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: HEXAMER-FORMING RESIDUES IN MOLECULE II            \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: SI1                                                 \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: SURFACE-INVARIANT RESIDUES IN MOLECULE I NOT       \\nREMARK 800  INVOLVED IN DIMERIZATION                                            \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: SI2                                                 \\nREMARK 800 EVIDENCE_CODE: AUTHOR                                                \\nREMARK 800 SITE_DESCRIPTION: SURFACE-INVARIANT RESIDUES IN MOLECULE II NOT      \\nREMARK 800  INVOLVED IN DIMERIZATION                                            \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE ZN B 31                   \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE ZN D 31                   \\nDBREF  4INS A    1    21  UNP    P01315   INS_PIG         88    108             \\nDBREF  4INS B    1    30  UNP    P01315   INS_PIG         25     54             \\nDBREF  4INS C    1    21  UNP    P01315   INS_PIG         88    108             \\nDBREF  4INS D    1    30  UNP    P01315   INS_PIG         25     54             \\nSEQRES   1 A   21  GLY ILE VAL GLU GLN CYS CYS THR SER ILE CYS SER LEU          \\nSEQRES   2 A   21  TYR GLN LEU GLU ASN TYR CYS ASN                              \\nSEQRES   1 B   30  PHE VAL ASN GLN HIS LEU CYS GLY SER HIS LEU VAL GLU          \\nSEQRES   2 B   30  ALA LEU TYR LEU VAL CYS GLY GLU ARG GLY PHE PHE TYR          \\nSEQRES   3 B   30  THR PRO LYS ALA                                              \\nSEQRES   1 C   21  GLY ILE VAL GLU GLN CYS CYS THR SER ILE CYS SER LEU          \\nSEQRES   2 C   21  TYR GLN LEU GLU ASN TYR CYS ASN                              \\nSEQRES   1 D   30  PHE VAL ASN GLN HIS LEU CYS GLY SER HIS LEU VAL GLU          \\nSEQRES   2 D   30  ALA LEU TYR LEU VAL CYS GLY GLU ARG GLY PHE PHE TYR          \\nSEQRES   3 D   30  THR PRO LYS ALA                                              \\nHET     ZN  B 101       1                                                       \\nHET     ZN  D 101       1                                                       \\nHETNAM      ZN ZINC ION                                                         \\nFORMUL   5   ZN    2(ZN 2+)                                                     \\nFORMUL   7  HOH   *350(H2 O)                                                    \\nHELIX    1 A11 GLY A    1  ILE A   10  1VAL 203 O H-BONDED TO HOH         10    \\nHELIX    2 A12 SER A   12  GLU A   17  5CNTCTS MOSTLY GT 3A,NOT IDEAL      6    \\nHELIX    3 B11 SER B    9  GLY B   20  1CYS 67 GLY 68, 3(10) CONTACTS     12    \\nHELIX    4 A21 GLY C    1  ILE C   10  1NOT IDEAL ALPH,SOME PI CNTCTS     10    \\nHELIX    5 A22 SER C   12  GLU C   17  5CNTCTS MOSTLY GT 3A,NOT IDEAL      6    \\nHELIX    6 B21 SER D    9  GLY D   20  1CYS 67,GLY 68, 3(10) CONTACTS     12    \\nSHEET    1   B 2 PHE B  24  TYR B  26  0                                        \\nSHEET    2   B 2 PHE D  24  TYR D  26 -1  N  PHE B  24   O  TYR D  26           \\nSSBOND   1 CYS A    6    CYS A   11                          1555   1555  2.05  \\nSSBOND   2 CYS A    7    CYS B    7                          1555   1555  1.97  \\nSSBOND   3 CYS A   20    CYS B   19                          1555   1555  2.00  \\nSSBOND   4 CYS C    6    CYS C   11                          1555   1555  2.06  \\nSSBOND   5 CYS C    7    CYS D    7                          1555   1555  2.01  \\nSSBOND   6 CYS C   20    CYS D   19                          1555   1555  2.02  \\nLINK         NE2 HIS B  10                ZN    ZN B 101     1555   1555  2.11  \\nLINK         NE2 HIS B  10                ZN    ZN B 101     2555   1555  2.10  \\nLINK         NE2 HIS B  10                ZN    ZN B 101     3555   1555  2.11  \\nLINK        ZN    ZN B 101                 O   HOH B 213     1555   1555  2.19  \\nLINK         NE2 HIS D  10                ZN    ZN D 101     1555   1555  2.08  \\nLINK         NE2 HIS D  10                ZN    ZN D 101     3555   1555  2.08  \\nLINK         NE2 HIS D  10                ZN    ZN D 101     2555   1555  2.08  \\nSITE     1  D1  5 VAL B  12  TYR B  16  PHE B  24  PHE B  25                    \\nSITE     2  D1  5 TYR B  26                                                     \\nSITE     1  D2  5 VAL D  12  TYR D  16  PHE D  24  PHE D  25                    \\nSITE     2  D2  5 TYR D  26                                                     \\nSITE     1  H1  7 LEU A  13  TYR A  14  PHE B   1  GLU B  13                    \\nSITE     2  H1  7 ALA B  14  LEU B  17  VAL B  18                               \\nSITE     1  H2  7 LEU C  13  TYR C  14  PHE D   1  GLU D  13                    \\nSITE     2  H2  7 ALA D  14  LEU D  17  VAL D  18                               \\nSITE     1 SI1  7 GLY A   1  GLU A   4  GLN A   5  CYS A   7                    \\nSITE     2 SI1  7 TYR A  19  ASN A  21  CYS B   7                               \\nSITE     1 SI2  7 GLY C   1  GLU C   4  GLN C   5  CYS C   7                    \\nSITE     2 SI2  7 TYR C  19  ASN C  21  CYS D   7                               \\nSITE     1 AC1  1 HIS B  10                                                     \\nSITE     1 AC2  1 HIS D  10                                                     \\nCRYST1   82.500   82.500   34.000  90.00  90.00 120.00 H 3          18          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.012121  0.006998  0.000000        0.00000                         \\nSCALE2      0.000000  0.013996  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.029412        0.00000                         \\nMTRIX1   1 -0.878620 -0.476960  0.023050        0.00000    1                    \\nMTRIX2   1 -0.477430  0.878370 -0.022860        0.00000    1                    \\nMTRIX3   1 -0.009350 -0.031090 -0.999470        0.00000    1                    \\nATOM      1  N   GLY A   1      -8.863  16.944  14.289  1.00 21.88           N  \\nATOM      2  CA  GLY A   1      -9.929  17.026  13.244  1.00 22.85           C  \\nATOM      3  C   GLY A   1     -10.051  15.625  12.618  1.00 43.92           C  \\nATOM      4  O   GLY A   1      -9.782  14.728  13.407  1.00 25.22           O  \\nATOM      5  N   ILE A   2     -10.333  15.531  11.332  1.00 26.28           N  \\nATOM      6  CA  ILE A   2     -10.488  14.266  10.600  1.00 20.84           C  \\nATOM      7  C   ILE A   2      -9.367  13.302  10.658  1.00 11.81           C  \\nATOM      8  O   ILE A   2      -9.580  12.092  10.969  1.00 20.31           O  \\nATOM      9  CB  ILE A   2     -10.883  14.493   9.095  1.00 40.00           C  \\nATOM     10  CG1 ILE A   2     -11.579  13.146   8.697  1.00 36.74           C  \\nATOM     11  CG2 ILE A   2      -9.741  14.794   8.140  1.00 23.02           C  \\nATOM     12  CD1 ILE A   2     -12.813  13.031   9.640  1.00 26.69           C  \\nATOM     13  N   VAL A   3      -8.133  13.759  10.483  1.00 16.57           N  \\nATOM     14  CA  VAL A   3      -6.966  12.901  10.576  1.00 15.75           C  \\nATOM     15  C   VAL A   3      -6.892  12.161  11.922  1.00 22.09           C  \\nATOM     16  O   VAL A   3      -6.547  10.990  12.037  1.00 24.52           O  \\nATOM     17  CB  VAL A   3      -5.697  13.708  10.225  1.00 21.34           C  \\nATOM     18  CG1 VAL A   3      -4.382  12.960  10.448  1.00 32.48           C  \\nATOM     19  CG2 VAL A   3      -5.842  14.209   8.777  1.00 26.35           C  \\nATOM     20  N   GLU A   4      -7.043  13.019  12.935  1.00 16.58           N  \\nATOM     21  CA  GLU A   4      -6.889  12.474  14.295  1.00 15.32           C  \\nATOM     22  C   GLU A   4      -8.004  11.558  14.610  1.00 16.88           C  \\nATOM     23  O   GLU A   4      -7.888  10.474  15.128  1.00 23.30           O  \\nATOM     24  CB  GLU A   4      -6.809  13.691  15.266  1.00 17.11           C  \\nATOM     25  CG  GLU A   4      -5.615  14.565  14.951  1.00 21.45           C  \\nATOM     26  CD  GLU A   4      -5.704  15.457  13.735  1.00 21.59           C  \\nATOM     27  OE1 GLU A   4      -6.757  15.959  13.377  1.00 23.43           O  \\nATOM     28  OE2 GLU A   4      -4.568  15.569  13.179  1.00 25.36           O  \\nATOM     29  N   GLN A   5      -9.199  12.048  14.356  1.00 15.69           N  \\nATOM     30  CA  GLN A   5     -10.407  11.299  14.630  1.00 12.38           C  \\nATOM     31  C   GLN A   5     -10.431   9.940  13.980  1.00 19.86           C  \\nATOM     32  O   GLN A   5     -10.815   8.931  14.542  1.00 16.83           O  \\nATOM     33  CB  GLN A   5     -11.594  12.130  14.152  1.00 21.13           C  \\nATOM     34  CG  GLN A   5     -12.860  11.374  14.561  1.00 22.06           C  \\nATOM     35  CD  GLN A   5     -13.946  11.901  13.634  1.00 42.02           C  \\nATOM     36  OE1 GLN A   5     -13.908  13.027  13.169  1.00 55.10           O  \\nATOM     37  NE2 GLN A   5     -14.943  11.030  13.351  1.00 27.27           N  \\nATOM     38  N   CYS A   6     -10.033   9.815  12.695  1.00 13.19           N  \\nATOM     39  CA  CYS A   6     -10.050   8.518  12.065  1.00 12.63           C  \\nATOM     40  C   CYS A   6      -9.105   7.520  12.667  1.00  9.95           C  \\nATOM     41  O   CYS A   6      -9.395   6.288  12.666  1.00 14.22           O  \\nATOM     42  CB  CYS A   6      -9.660   8.673  10.559  1.00 12.54           C  \\nATOM     43  SG  CYS A   6     -10.925   9.459   9.579  1.00 13.00           S  \\nATOM     44  N   CYS A   7      -8.018   7.992  13.171  1.00 10.84           N  \\nATOM     45  CA  CYS A   7      -6.964   7.186  13.808  1.00 17.02           C  \\nATOM     46  C   CYS A   7      -7.236   6.948  15.358  1.00 13.71           C  \\nATOM     47  O   CYS A   7      -7.061   5.782  15.768  1.00 19.28           O  \\nATOM     48  CB  CYS A   7      -5.578   7.826  13.656  1.00 20.24           C  \\nATOM     49  SG  CYS A   7      -4.181   6.819  14.134  1.00 13.80           S  \\nATOM     50  N   THR A   8      -7.655   7.937  16.058  1.00 12.57           N  \\nATOM     51  CA  THR A   8      -7.862   7.732  17.520  1.00 19.99           C  \\nATOM     52  C   THR A   8      -9.143   6.997  17.870  1.00 26.34           C  \\nATOM     53  O   THR A   8      -9.189   6.157  18.795  1.00 25.43           O  \\nATOM     54  CB  THR A   8      -7.728   9.055  18.386  1.00 20.77           C  \\nATOM     55  OG1 THR A   8      -8.889   9.918  18.117  1.00 26.76           O  \\nATOM     56  CG2 THR A   8      -6.334   9.700  18.196  1.00 26.50           C  \\nATOM     57  N   SER A   9     -10.170   7.350  17.058  1.00 20.01           N  \\nATOM     58  CA  SER A   9     -11.509   6.803  17.121  1.00 16.88           C  \\nATOM     59  C   SER A   9     -11.796   5.981  15.856  1.00 12.70           C  \\nATOM     60  O   SER A   9     -11.139   5.010  15.473  1.00 17.60           O  \\nATOM     61  CB  SER A   9     -12.331   8.067  17.439  1.00 19.52           C  \\nATOM     62  OG  SER A   9     -13.674   7.774  17.650  1.00 32.34           O  \\nATOM     63  N   ILE A  10     -12.883   6.382  15.159  1.00 15.34           N  \\nATOM     64  CA  ILE A  10     -13.350   5.723  13.932  1.00 20.23           C  \\nATOM     65  C   ILE A  10     -13.969   6.902  13.106  1.00 17.50           C  \\nATOM     66  O   ILE A  10     -14.355   7.922  13.623  1.00 16.60           O  \\nATOM     67  CB  ILE A  10     -14.366   4.524  14.047  1.00 19.39           C  \\nATOM     68  CG1 ILE A  10     -15.702   4.874  14.742  1.00 22.05           C  \\nATOM     69  CG2 ILE A  10     -13.711   3.300  14.723  1.00 23.30           C  \\nATOM     70  CD1 ILE A  10     -16.702   3.722  15.005  1.00 42.11           C  \\nATOM     71  N   CYS A  11     -14.080   6.685  11.767  1.00 12.14           N  \\nATOM     72  CA  CYS A  11     -14.665   7.679  10.880  1.00 11.24           C  \\nATOM     73  C   CYS A  11     -15.301   6.881   9.766  1.00 12.17           C  \\nATOM     74  O   CYS A  11     -14.962   5.692   9.528  1.00 21.14           O  \\nATOM     75  CB  CYS A  11     -13.695   8.702  10.417  1.00 13.03           C  \\nATOM     76  SG  CYS A  11     -12.375   8.019   9.385  1.00 13.60           S  \\nATOM     77  N   SER A  12     -16.233   7.557   9.095  1.00 11.37           N  \\nATOM     78  CA  SER A  12     -16.999   6.978   8.005  1.00  9.91           C  \\nATOM     79  C   SER A  12     -16.563   7.644   6.726  1.00  7.40           C  \\nATOM     80  O   SER A  12     -15.967   8.753   6.711  1.00  9.67           O  \\nATOM     81  CB  SER A  12     -18.516   7.183   8.084  1.00 16.64           C  \\nATOM     82  OG  SER A  12     -18.869   8.543   7.881  1.00 17.14           O  \\nATOM     83  N   LEU A  13     -16.852   6.914   5.612  1.00 11.35           N  \\nATOM     84  CA  LEU A  13     -16.530   7.444   4.259  1.00 11.35           C  \\nATOM     85  C   LEU A  13     -17.317   8.715   4.030  1.00 12.55           C  \\nATOM     86  O   LEU A  13     -16.835   9.521   3.226  1.00 11.78           O  \\nATOM     87  CB  LEU A  13     -16.774   6.348   3.232  1.00 11.66           C  \\nATOM     88  CG  LEU A  13     -15.940   5.046   3.316  1.00 18.12           C  \\nATOM     89  CD1 LEU A  13     -16.050   4.197   2.018  1.00 18.76           C  \\nATOM     90  CD2 LEU A  13     -14.471   5.320   3.537  1.00 17.26           C  \\nATOM     91  N   TYR A  14     -18.491   8.790   4.629  1.00 10.84           N  \\nATOM     92  CA  TYR A  14     -19.282  10.035   4.368  1.00 10.75           C  \\nATOM     93  C   TYR A  14     -18.639  11.228   4.963  1.00 12.81           C  \\nATOM     94  O   TYR A  14     -18.706  12.298   4.341  1.00 15.11           O  \\nATOM     95  CB  TYR A  14     -20.746   9.900   4.799  1.00 12.90           C  \\nATOM     96  CG  TYR A  14     -21.463   8.764   4.079  1.00 18.23           C  \\nATOM     97  CD1 TYR A  14     -22.110   9.123   2.891  1.00 18.95           C  \\nATOM     98  CD2 TYR A  14     -21.461   7.440   4.475  1.00 15.42           C  \\nATOM     99  CE1 TYR A  14     -22.767   8.167   2.086  1.00 18.15           C  \\nATOM    100  CE2 TYR A  14     -22.118   6.436   3.676  1.00 14.31           C  \\nATOM    101  CZ  TYR A  14     -22.738   6.856   2.556  1.00 15.47           C  \\nATOM    102  OH  TYR A  14     -23.436   5.926   1.716  1.00 24.86           O  \\nATOM    103  N   GLN A  15     -17.945  11.100   6.091  1.00  9.63           N  \\nATOM    104  CA  GLN A  15     -17.178  12.138   6.774  1.00  9.40           C  \\nATOM    105  C   GLN A  15     -16.012  12.543   5.900  1.00 10.52           C  \\nATOM    106  O   GLN A  15     -15.611  13.717   5.722  1.00 14.25           O  \\nATOM    107  CB  GLN A  15     -16.774  11.841   8.205  1.00 13.89           C  \\nATOM    108  CG  GLN A  15     -17.894  11.668   9.206  1.00 17.53           C  \\nATOM    109  CD  GLN A  15     -17.524  11.056  10.515  1.00 28.21           C  \\nATOM    110  OE1 GLN A  15     -16.865  10.027  10.598  1.00 20.14           O  \\nATOM    111  NE2 GLN A  15     -17.994  11.650  11.624  1.00 30.25           N  \\nATOM    112  N   LEU A  16     -15.352  11.525   5.325  1.00 12.99           N  \\nATOM    113  CA  LEU A  16     -14.185  11.826   4.470  1.00 11.19           C  \\nATOM    114  C   LEU A  16     -14.605  12.634   3.249  1.00 15.54           C  \\nATOM    115  O   LEU A  16     -13.767  13.398   2.745  1.00 16.01           O  \\nATOM    116  CB  LEU A  16     -13.588  10.521   4.060  1.00 12.67           C  \\nATOM    117  CG  LEU A  16     -12.954   9.717   5.182  1.00 13.07           C  \\nATOM    118  CD1 LEU A  16     -12.115   8.571   4.602  1.00 16.61           C  \\nATOM    119  CD2 LEU A  16     -12.041  10.559   6.028  1.00 16.50           C  \\nATOM    120  N   GLU A  17     -15.779  12.420   2.759  1.00 17.50           N  \\nATOM    121  CA  GLU A  17     -16.223  13.179   1.589  1.00 17.72           C  \\nATOM    122  C   GLU A  17     -16.171  14.693   1.811  1.00 19.21           C  \\nATOM    123  O   GLU A  17     -16.118  15.466   0.803  1.00 18.48           O  \\nATOM    124  CB  GLU A  17     -17.645  12.862   1.215  1.00 17.38           C  \\nATOM    125  CG  GLU A  17     -17.885  11.629   0.360  1.00 27.97           C  \\nATOM    126  CD  GLU A  17     -19.225  11.667  -0.391  1.00 26.70           C  \\nATOM    127  OE1 GLU A  17     -20.201  11.466   0.276  1.00 29.93           O  \\nATOM    128  OE2 GLU A  17     -19.127  11.873  -1.643  1.00 34.66           O  \\nATOM    129  N   ASN A  18     -16.094  15.074   3.104  1.00 15.10           N  \\nATOM    130  CA  ASN A  18     -16.029  16.534   3.332  1.00 18.85           C  \\nATOM    131  C   ASN A  18     -14.703  17.131   2.954  1.00 18.46           C  \\nATOM    132  O   ASN A  18     -14.545  18.377   2.834  1.00 19.68           O  \\nATOM    133  CB  ASN A  18     -16.489  16.934   4.738  1.00 20.66           C  \\nATOM    134  CG  ASN A  18     -17.868  16.338   5.142  1.00 29.79           C  \\nATOM    135  OD1 ASN A  18     -18.813  16.053   4.382  1.00 34.48           O  \\nATOM    136  ND2 ASN A  18     -17.991  16.168   6.452  1.00 36.00           N  \\nATOM    137  N   TYR A  19     -13.697  16.327   2.738  1.00 15.68           N  \\nATOM    138  CA  TYR A  19     -12.358  16.724   2.380  1.00 14.19           C  \\nATOM    139  C   TYR A  19     -12.154  16.695   0.899  1.00 13.20           C  \\nATOM    140  O   TYR A  19     -11.010  17.038   0.480  1.00 16.12           O  \\nATOM    141  CB  TYR A  19     -11.364  15.840   3.178  1.00 14.35           C  \\nATOM    142  CG  TYR A  19     -11.586  16.223   4.634  1.00 21.24           C  \\nATOM    143  CD1 TYR A  19     -10.853  17.300   5.129  1.00 26.61           C  \\nATOM    144  CD2 TYR A  19     -12.562  15.703   5.445  1.00 19.21           C  \\nATOM    145  CE1 TYR A  19     -11.084  17.801   6.393  1.00 27.80           C  \\nATOM    146  CE2 TYR A  19     -12.833  16.207   6.714  1.00 23.98           C  \\nATOM    147  CZ  TYR A  19     -12.081  17.267   7.187  1.00 34.08           C  \\nATOM    148  OH  TYR A  19     -12.227  17.849   8.400  1.00 37.96           O  \\nATOM    149  N   CYS A  20     -13.057  16.313   0.077  1.00 13.05           N  \\nATOM    150  CA  CYS A  20     -12.838  16.309  -1.389  1.00 18.69           C  \\nATOM    151  C   CYS A  20     -12.984  17.799  -1.802  1.00 19.09           C  \\nATOM    152  O   CYS A  20     -13.588  18.579  -1.084  1.00 19.31           O  \\nATOM    153  CB  CYS A  20     -13.850  15.490  -2.157  1.00 15.99           C  \\nATOM    154  SG  CYS A  20     -13.923  13.761  -1.584  1.00 12.90           S  \\nATOM    155  N   ASN A  21     -12.380  18.063  -2.909  1.00 17.63           N  \\nATOM    156  CA  ASN A  21     -12.404  19.399  -3.608  1.00 25.23           C  \\nATOM    157  C   ASN A  21     -13.642  19.696  -4.447  1.00 34.82           C  \\nATOM    158  O   ASN A  21     -14.146  18.703  -4.956  1.00 31.24           O  \\nATOM    159  CB  ASN A  21     -11.228  19.392  -4.521  1.00 19.06           C  \\nATOM    160  CG  ASN A  21     -10.020  20.283  -4.456  1.00 40.71           C  \\nATOM    161  OD1 ASN A  21     -10.067  21.380  -5.083  1.00 68.22           O  \\nATOM    162  ND2 ASN A  21      -9.004  19.667  -3.808  1.00 39.69           N  \\nATOM    163  OXT ASN A  21     -13.881  20.890  -4.604  1.00 41.83           O  \\nTER     164      ASN A  21                                                      \\nATOM    165  N   PHE B   1     -21.768   1.132   3.577  1.00 25.87           N  \\nATOM    166  CA  PHE B   1     -20.374   1.368   4.053  1.00 24.30           C  \\nATOM    167  C   PHE B   1     -20.341   1.145   5.585  1.00 39.74           C  \\nATOM    168  O   PHE B   1     -21.423   1.141   6.173  1.00 38.10           O  \\nATOM    169  CB  PHE B   1     -19.806   2.718   3.624  1.00 22.51           C  \\nATOM    170  CG  PHE B   1     -19.924   2.916   2.131  1.00 16.52           C  \\nATOM    171  CD1 PHE B   1     -20.067   4.204   1.618  1.00 35.58           C  \\nATOM    172  CD2 PHE B   1     -19.709   1.873   1.262  1.00 20.86           C  \\nATOM    173  CE1 PHE B   1     -20.093   4.444   0.243  1.00 52.66           C  \\nATOM    174  CE2 PHE B   1     -19.824   2.067  -0.123  1.00 51.46           C  \\nATOM    175  CZ  PHE B   1     -20.011   3.332  -0.631  1.00 42.63           C  \\nATOM    176  N   VAL B   2     -19.104   0.899   6.027  1.00 21.12           N  \\nATOM    177  CA  VAL B   2     -18.754   0.598   7.406  1.00 36.74           C  \\nATOM    178  C   VAL B   2     -17.780   1.656   7.965  1.00 23.52           C  \\nATOM    179  O   VAL B   2     -17.104   2.328   7.197  1.00 19.56           O  \\nATOM    180  CB  VAL B   2     -18.048  -0.765   7.638  1.00 30.58           C  \\nATOM    181  CG1 VAL B   2     -18.993  -1.953   7.609  1.00 25.73           C  \\nATOM    182  CG2 VAL B   2     -16.776  -0.916   6.799  1.00 22.31           C  \\nATOM    183  N   ASN B   3     -17.741   1.753   9.278  1.00 13.38           N  \\nATOM    184  CA  ASN B   3     -16.872   2.691   9.950  1.00 13.94           C  \\nATOM    185  C   ASN B   3     -15.457   2.100   9.881  1.00 15.03           C  \\nATOM    186  O   ASN B   3     -15.312   0.857   9.926  1.00 24.85           O  \\nATOM    187  CB  ASN B   3     -17.272   3.010  11.382  1.00 25.01           C  \\nATOM    188  CG  ASN B   3     -18.513   3.844  11.511  1.00 49.04           C  \\nATOM    189  OD1 ASN B   3     -18.658   4.774  10.724  1.00 34.50           O  \\nATOM    190  ND2 ASN B   3     -19.333   3.415  12.473  1.00 35.00           N  \\nATOM    191  N   GLN B   4     -14.509   3.031   9.767  1.00 12.52           N  \\nATOM    192  CA  GLN B   4     -13.137   2.542   9.571  1.00 22.69           C  \\nATOM    193  C   GLN B   4     -12.213   3.224  10.580  1.00 13.29           C  \\nATOM    194  O   GLN B   4     -12.347   4.333  11.118  1.00 20.53           O  \\nATOM    195  CB  GLN B   4     -12.666   2.760   8.116  1.00 39.18           C  \\nATOM    196  CG AGLN B   4     -13.007   1.731   7.035  0.60 11.45           C  \\nATOM    197  CG BGLN B   4     -12.978   1.763   6.996  0.40 37.30           C  \\nATOM    198  CD AGLN B   4     -12.270   0.520   6.830  0.60 12.42           C  \\nATOM    199  CD BGLN B   4     -14.070   2.781   6.746  0.40 32.97           C  \\nATOM    200  OE1AGLN B   4     -12.812  -0.612   6.494  0.60 17.67           O  \\nATOM    201  OE1BGLN B   4     -14.059   3.957   7.112  0.40 40.00           O  \\nATOM    202  NE2AGLN B   4     -10.898   0.624   6.949  0.60 28.94           N  \\nATOM    203  NE2BGLN B   4     -15.108   2.179   6.165  0.40 35.67           N  \\nATOM    204  N   HIS B   5     -11.158   2.442  10.837  1.00 13.02           N  \\nATOM    205  CA  HIS B   5     -10.083   3.000  11.779  1.00 17.05           C  \\nATOM    206  C   HIS B   5      -8.855   3.149  10.899  1.00 10.95           C  \\nATOM    207  O   HIS B   5      -8.284   2.166  10.380  1.00 17.14           O  \\nATOM    208  CB  HIS B   5      -9.982   1.956  12.877  1.00 22.24           C  \\nATOM    209  CG  HIS B   5      -8.934   2.400  13.860  1.00 25.74           C  \\nATOM    210  ND1 HIS B   5      -8.072   1.535  14.436  1.00 35.32           N  \\nATOM    211  CD2 HIS B   5      -8.637   3.596  14.329  1.00 28.02           C  \\nATOM    212  CE1 HIS B   5      -7.275   2.240  15.211  1.00 28.73           C  \\nATOM    213  NE2 HIS B   5      -7.571   3.509  15.150  1.00 30.21           N  \\nATOM    214  N   LEU B   6      -8.529   4.400  10.604  1.00 11.30           N  \\nATOM    215  CA  LEU B   6      -7.468   4.709   9.611  1.00 11.13           C  \\nATOM    216  C   LEU B   6      -6.399   5.604  10.158  1.00 11.03           C  \\nATOM    217  O   LEU B   6      -6.695   6.779  10.484  1.00 13.66           O  \\nATOM    218  CB  LEU B   6      -8.231   5.398   8.411  1.00 14.13           C  \\nATOM    219  CG  LEU B   6      -9.251   4.634   7.563  1.00 13.39           C  \\nATOM    220  CD1 LEU B   6     -10.017   5.598   6.671  1.00 14.70           C  \\nATOM    221  CD2 LEU B   6      -8.620   3.517   6.767  1.00 18.25           C  \\nATOM    222  N   CYS B   7      -5.180   5.069  10.115  1.00 10.06           N  \\nATOM    223  CA  CYS B   7      -4.058   5.835  10.569  1.00 10.70           C  \\nATOM    224  C   CYS B   7      -3.033   5.982   9.484  1.00 13.26           C  \\nATOM    225  O   CYS B   7      -2.955   5.198   8.573  1.00 19.10           O  \\nATOM    226  CB  CYS B   7      -3.434   5.105  11.762  1.00 15.88           C  \\nATOM    227  SG  CYS B   7      -4.523   5.099  13.246  1.00 16.40           S  \\nATOM    228  N   GLY B   8      -2.181   6.993   9.540  1.00 12.37           N  \\nATOM    229  CA  GLY B   8      -1.070   7.261   8.632  1.00 12.72           C  \\nATOM    230  C   GLY B   8      -1.465   7.317   7.204  1.00 13.24           C  \\nATOM    231  O   GLY B   8      -2.470   7.884   6.744  1.00 11.92           O  \\nATOM    232  N   SER B   9      -0.609   6.582   6.429  1.00 11.74           N  \\nATOM    233  CA  SER B   9      -0.863   6.544   4.980  1.00 15.89           C  \\nATOM    234  C   SER B   9      -2.183   5.870   4.578  1.00  9.73           C  \\nATOM    235  O   SER B   9      -2.649   6.111   3.528  1.00 10.43           O  \\nATOM    236  CB  SER B   9       0.309   5.921   4.206  1.00 17.74           C  \\nATOM    237  OG  SER B   9       0.534   4.626   4.735  1.00 17.37           O  \\nATOM    238  N   HIS B  10      -2.721   5.100   5.451  1.00 10.19           N  \\nATOM    239  CA  HIS B  10      -3.940   4.379   5.188  1.00  7.66           C  \\nATOM    240  C   HIS B  10      -5.081   5.431   5.075  1.00 10.17           C  \\nATOM    241  O   HIS B  10      -6.021   5.163   4.291  1.00 10.92           O  \\nATOM    242  CB  HIS B  10      -4.234   3.316   6.228  1.00  9.55           C  \\nATOM    243  CG  HIS B  10      -3.192   2.269   6.364  1.00  9.55           C  \\nATOM    244  ND1 HIS B  10      -3.043   1.310   5.423  1.00 15.86           N  \\nATOM    245  CD2 HIS B  10      -2.289   1.991   7.311  1.00  8.47           C  \\nATOM    246  CE1 HIS B  10      -2.078   0.573   5.774  1.00 10.65           C  \\nATOM    247  NE2 HIS B  10      -1.589   0.939   6.878  1.00  9.41           N  \\nATOM    248  N   LEU B  11      -5.016   6.497   5.810  1.00  8.93           N  \\nATOM    249  CA  LEU B  11      -6.071   7.518   5.617  1.00  9.64           C  \\nATOM    250  C   LEU B  11      -5.967   8.182   4.279  1.00  7.89           C  \\nATOM    251  O   LEU B  11      -6.969   8.462   3.666  1.00  9.74           O  \\nATOM    252  CB  LEU B  11      -5.860   8.541   6.740  1.00  6.93           C  \\nATOM    253  CG  LEU B  11      -6.949   9.607   6.783  1.00 14.50           C  \\nATOM    254  CD1 LEU B  11      -8.376   9.229   6.627  1.00 18.34           C  \\nATOM    255  CD2 LEU B  11      -6.742  10.309   8.115  1.00 20.70           C  \\nATOM    256  N   VAL B  12      -4.751   8.449   3.799  1.00 10.12           N  \\nATOM    257  CA  VAL B  12      -4.579   9.057   2.495  1.00  8.05           C  \\nATOM    258  C   VAL B  12      -5.050   8.131   1.372  1.00  8.14           C  \\nATOM    259  O   VAL B  12      -5.595   8.653   0.398  1.00 11.63           O  \\nATOM    260  CB  VAL B  12      -3.153   9.538   2.230  1.00 11.54           C  \\nATOM    261  CG1AVAL B  12      -2.822   9.786   0.799  0.50  4.68           C  \\nATOM    262  CG1BVAL B  12      -2.809  10.670   3.148  0.50 17.75           C  \\nATOM    263  CG2AVAL B  12      -2.809  10.670   3.148  0.50 17.75           C  \\nATOM    264  CG2BVAL B  12      -1.963   8.655   2.123  0.50 10.87           C  \\nATOM    265  N   GLU B  13      -4.906   6.880   1.502  1.00  6.12           N  \\nATOM    266  CA  GLU B  13      -5.432   5.946   0.542  1.00  8.88           C  \\nATOM    267  C   GLU B  13      -6.966   6.014   0.472  1.00 12.22           C  \\nATOM    268  O   GLU B  13      -7.578   6.035  -0.614  1.00 11.15           O  \\nATOM    269  CB  GLU B  13      -4.996   4.506   0.854  1.00 12.65           C  \\nATOM    270  CG  GLU B  13      -3.497   4.444   0.582  1.00 15.60           C  \\nATOM    271  CD  GLU B  13      -3.246   3.857  -0.794  1.00 53.85           C  \\nATOM    272  OE1 GLU B  13      -4.238   3.643  -1.500  1.00 33.68           O  \\nATOM    273  OE2 GLU B  13      -2.114   3.611  -1.126  1.00 47.24           O  \\nATOM    274  N   ALA B  14      -7.659   6.004   1.637  1.00  7.15           N  \\nATOM    275  CA  ALA B  14      -9.061   6.164   1.719  1.00  7.29           C  \\nATOM    276  C   ALA B  14      -9.563   7.482   1.051  1.00  6.80           C  \\nATOM    277  O   ALA B  14     -10.595   7.468   0.346  1.00 11.10           O  \\nATOM    278  CB  ALA B  14      -9.604   6.039   3.106  1.00 12.06           C  \\nATOM    279  N   LEU B  15      -8.876   8.580   1.321  1.00  6.72           N  \\nATOM    280  CA  LEU B  15      -9.224   9.854   0.717  1.00 13.51           C  \\nATOM    281  C   LEU B  15      -9.111   9.815  -0.829  1.00 14.62           C  \\nATOM    282  O   LEU B  15      -9.956  10.390  -1.496  1.00 12.32           O  \\nATOM    283  CB  LEU B  15      -8.317  10.981   1.327  1.00  9.71           C  \\nATOM    284  CG  LEU B  15      -8.755  11.581   2.649  1.00  8.92           C  \\nATOM    285  CD1 LEU B  15      -7.682  12.475   3.236  1.00 14.49           C  \\nATOM    286  CD2 LEU B  15     -10.096  12.235   2.460  1.00 12.03           C  \\nATOM    287  N   TYR B  16      -8.050   9.147  -1.297  1.00  8.65           N  \\nATOM    288  CA  TYR B  16      -7.838   8.961  -2.686  1.00  8.75           C  \\nATOM    289  C   TYR B  16      -8.999   8.175  -3.284  1.00 11.14           C  \\nATOM    290  O   TYR B  16      -9.508   8.504  -4.371  1.00 14.34           O  \\nATOM    291  CB  TYR B  16      -6.494   8.247  -3.047  1.00  7.72           C  \\nATOM    292  CG  TYR B  16      -6.271   8.027  -4.522  1.00 10.81           C  \\nATOM    293  CD1 TYR B  16      -6.450   6.784  -5.047  1.00 17.09           C  \\nATOM    294  CD2 TYR B  16      -6.009   9.104  -5.338  1.00 12.64           C  \\nATOM    295  CE1 TYR B  16      -6.354   6.581  -6.467  1.00 17.76           C  \\nATOM    296  CE2 TYR B  16      -5.898   8.958  -6.741  1.00 13.94           C  \\nATOM    297  CZ  TYR B  16      -6.110   7.692  -7.259  1.00 17.34           C  \\nATOM    298  OH  TYR B  16      -5.925   7.520  -8.594  1.00 25.34           O  \\nATOM    299  N   LEU B  17      -9.428   7.109  -2.664  1.00  8.68           N  \\nATOM    300  CA  LEU B  17     -10.566   6.290  -3.167  1.00  8.83           C  \\nATOM    301  C   LEU B  17     -11.861   7.087  -3.142  1.00 10.95           C  \\nATOM    302  O   LEU B  17     -12.650   7.046  -4.073  1.00 15.67           O  \\nATOM    303  CB  LEU B  17     -10.665   5.052  -2.327  1.00 10.82           C  \\nATOM    304  CG  LEU B  17      -9.594   4.104  -2.924  1.00 28.76           C  \\nATOM    305  CD1 LEU B  17      -9.136   3.067  -1.933  1.00 30.52           C  \\nATOM    306  CD2 LEU B  17     -10.280   3.540  -4.157  1.00 34.91           C  \\nATOM    307  N   VAL B  18     -12.123   7.786  -2.036  1.00  8.85           N  \\nATOM    308  CA  VAL B  18     -13.351   8.545  -1.933  1.00  8.77           C  \\nATOM    309  C   VAL B  18     -13.433   9.713  -2.873  1.00  9.77           C  \\nATOM    310  O   VAL B  18     -14.472   9.937  -3.457  1.00 16.86           O  \\nATOM    311  CB  VAL B  18     -13.604   8.974  -0.463  1.00 14.39           C  \\nATOM    312  CG1 VAL B  18     -14.784   9.899  -0.282  1.00 11.72           C  \\nATOM    313  CG2 VAL B  18     -13.862   7.763   0.393  1.00 12.58           C  \\nATOM    314  N   CYS B  19     -12.422  10.518  -2.958  1.00  9.03           N  \\nATOM    315  CA  CYS B  19     -12.433  11.758  -3.756  1.00  8.88           C  \\nATOM    316  C   CYS B  19     -11.994  11.555  -5.212  1.00 14.69           C  \\nATOM    317  O   CYS B  19     -12.410  12.303  -6.126  1.00 16.46           O  \\nATOM    318  CB  CYS B  19     -11.558  12.719  -3.005  1.00 11.19           C  \\nATOM    319  SG  CYS B  19     -12.040  13.127  -1.344  1.00 10.10           S  \\nATOM    320  N   GLY B  20     -11.149  10.609  -5.463  1.00 17.12           N  \\nATOM    321  CA  GLY B  20     -10.685  10.359  -6.851  1.00 21.59           C  \\nATOM    322  C   GLY B  20     -10.275  11.650  -7.524  1.00 21.38           C  \\nATOM    323  O   GLY B  20      -9.494  12.483  -7.053  1.00 20.41           O  \\nATOM    324  N   GLU B  21     -10.784  11.844  -8.710  1.00 29.76           N  \\nATOM    325  CA  GLU B  21     -10.398  13.043  -9.501  1.00 24.44           C  \\nATOM    326  C   GLU B  21     -10.898  14.356  -9.065  1.00 19.21           C  \\nATOM    327  O   GLU B  21     -10.430  15.331  -9.665  1.00 19.00           O  \\nATOM    328  CB  GLU B  21     -10.776  12.724 -10.968  1.00 28.66           C  \\nATOM    329  CG AGLU B  21     -12.310  12.519 -11.045  0.50 56.11           C  \\nATOM    330  CG BGLU B  21      -9.804  13.415 -11.966  0.50 75.35           C  \\nATOM    331  CD AGLU B  21     -12.707  11.349 -11.901  0.50 62.47           C  \\nATOM    332  CD BGLU B  21      -9.689  13.292 -13.466  0.50 52.68           C  \\nATOM    333  OE1AGLU B  21     -12.515  10.193 -11.564  0.50 48.34           O  \\nATOM    334  OE1BGLU B  21     -10.540  12.768 -14.159  0.50 50.28           O  \\nATOM    335  OE2AGLU B  21     -13.225  11.772 -12.958  0.50 49.92           O  \\nATOM    336  OE2BGLU B  21      -8.505  13.537 -13.781  0.50 27.33           O  \\nATOM    337  N   ARG B  22     -11.703  14.491  -8.034  1.00 15.49           N  \\nATOM    338  CA  ARG B  22     -12.089  15.732  -7.456  1.00 16.44           C  \\nATOM    339  C   ARG B  22     -10.797  16.222  -6.745  1.00 17.70           C  \\nATOM    340  O   ARG B  22     -10.636  17.458  -6.608  1.00 21.36           O  \\nATOM    341  CB  ARG B  22     -13.234  15.678  -6.464  1.00 21.99           C  \\nATOM    342  CG  ARG B  22     -14.645  15.427  -7.037  1.00 57.89           C  \\nATOM    343  CD  ARG B  22     -15.675  15.167  -5.960  1.00 31.23           C  \\nATOM    344  NE AARG B  22     -15.739  16.404  -5.124  0.50 16.46           N  \\nATOM    345  NE BARG B  22     -15.629  13.808  -5.271  0.50 17.69           N  \\nATOM    346  CZ AARG B  22     -16.608  16.581  -4.143  0.50 39.57           C  \\nATOM    347  CZ BARG B  22     -16.379  13.225  -4.283  0.50 33.09           C  \\nATOM    348  NH1AARG B  22     -16.743  17.672  -3.416  0.50 20.14           N  \\nATOM    349  NH1BARG B  22     -16.987  14.046  -3.392  0.50 51.50           N  \\nATOM    350  NH2AARG B  22     -17.405  15.551  -3.871  0.50 35.40           N  \\nATOM    351  NH2BARG B  22     -16.705  11.943  -4.184  0.50 19.59           N  \\nATOM    352  N   GLY B  23     -10.007  15.246  -6.287  1.00 19.18           N  \\nATOM    353  CA  GLY B  23      -8.844  15.673  -5.491  1.00 11.89           C  \\nATOM    354  C   GLY B  23      -9.339  15.932  -4.075  1.00 12.83           C  \\nATOM    355  O   GLY B  23     -10.524  15.922  -3.626  1.00 14.47           O  \\nATOM    356  N   PHE B  24      -8.343  16.165  -3.187  1.00 12.54           N  \\nATOM    357  CA  PHE B  24      -8.584  16.432  -1.765  1.00 10.08           C  \\nATOM    358  C   PHE B  24      -7.488  17.220  -1.123  1.00 10.77           C  \\nATOM    359  O   PHE B  24      -6.411  17.409  -1.657  1.00 10.93           O  \\nATOM    360  CB  PHE B  24      -8.754  15.111  -1.032  1.00  3.80           C  \\nATOM    361  CG  PHE B  24      -7.638  14.114  -1.034  1.00  5.98           C  \\nATOM    362  CD1 PHE B  24      -7.488  13.202  -2.069  1.00  5.61           C  \\nATOM    363  CD2 PHE B  24      -6.667  14.205  -0.036  1.00  8.93           C  \\nATOM    364  CE1 PHE B  24      -6.375  12.338  -2.106  1.00 14.64           C  \\nATOM    365  CE2 PHE B  24      -5.573  13.387   0.027  1.00 11.74           C  \\nATOM    366  CZ  PHE B  24      -5.457  12.470  -1.008  1.00  9.78           C  \\nATOM    367  N   PHE B  25      -7.717  17.612   0.116  1.00 14.20           N  \\nATOM    368  CA  PHE B  25      -6.813  18.302   1.052  1.00 12.03           C  \\nATOM    369  C   PHE B  25      -6.569  17.356   2.221  1.00 12.69           C  \\nATOM    370  O   PHE B  25      -7.485  16.788   2.757  1.00 15.22           O  \\nATOM    371  CB  PHE B  25      -7.387  19.633   1.684  1.00 17.25           C  \\nATOM    372  CG  PHE B  25      -7.105  20.689   0.637  1.00 30.38           C  \\nATOM    373  CD1 PHE B  25      -7.842  20.802  -0.543  1.00 61.20           C  \\nATOM    374  CD2 PHE B  25      -6.003  21.541   0.896  1.00 56.90           C  \\nATOM    375  CE1 PHE B  25      -7.445  21.790  -1.461  1.00 29.52           C  \\nATOM    376  CE2 PHE B  25      -5.648  22.564  -0.027  1.00 40.31           C  \\nATOM    377  CZ  PHE B  25      -6.382  22.681  -1.235  1.00 30.38           C  \\nATOM    378  N   TYR B  26      -5.345  17.202   2.583  1.00 11.25           N  \\nATOM    379  CA  TYR B  26      -4.996  16.333   3.717  1.00 10.42           C  \\nATOM    380  C   TYR B  26      -4.445  17.350   4.714  1.00 15.08           C  \\nATOM    381  O   TYR B  26      -3.350  17.906   4.518  1.00 14.52           O  \\nATOM    382  CB  TYR B  26      -3.949  15.288   3.319  1.00  9.51           C  \\nATOM    383  CG  TYR B  26      -3.404  14.530   4.474  1.00 12.61           C  \\nATOM    384  CD1 TYR B  26      -4.243  13.688   5.178  1.00 20.50           C  \\nATOM    385  CD2 TYR B  26      -2.105  14.676   4.857  1.00 13.88           C  \\nATOM    386  CE1 TYR B  26      -3.652  12.967   6.246  1.00 16.38           C  \\nATOM    387  CE2 TYR B  26      -1.577  14.010   5.941  1.00 10.97           C  \\nATOM    388  CZ  TYR B  26      -2.347  13.149   6.642  1.00 11.71           C  \\nATOM    389  OH  TYR B  26      -1.853  12.447   7.734  1.00 16.39           O  \\nATOM    390  N   THR B  27      -5.287  17.537   5.752  1.00 15.47           N  \\nATOM    391  CA  THR B  27      -4.902  18.555   6.772  1.00 20.05           C  \\nATOM    392  C   THR B  27      -4.834  18.053   8.250  1.00 15.09           C  \\nATOM    393  O   THR B  27      -5.825  18.282   8.943  1.00 22.56           O  \\nATOM    394  CB  THR B  27      -5.856  19.825   6.753  1.00 26.34           C  \\nATOM    395  OG1ATHR B  27      -7.228  19.328   6.558  0.50 39.91           O  \\nATOM    396  OG1BTHR B  27      -5.691  20.478   5.511  0.50 29.43           O  \\nATOM    397  CG2ATHR B  27      -5.505  20.781   5.606  0.50 34.53           C  \\nATOM    398  CG2BTHR B  27      -5.413  20.850   7.858  0.50 34.13           C  \\nATOM    399  N   PRO B  28      -3.702  17.548   8.603  1.00 18.26           N  \\nATOM    400  CA  PRO B  28      -3.494  17.055   9.954  1.00 21.56           C  \\nATOM    401  C   PRO B  28      -3.306  18.220  10.892  1.00 22.68           C  \\nATOM    402  O   PRO B  28      -3.072  19.330  10.484  1.00 21.93           O  \\nATOM    403  CB  PRO B  28      -2.249  16.209   9.808  1.00 21.60           C  \\nATOM    404  CG  PRO B  28      -1.544  16.617   8.595  1.00 21.39           C  \\nATOM    405  CD  PRO B  28      -2.526  17.320   7.778  1.00 14.32           C  \\nATOM    406  N   LYS B  29      -3.452  17.975  12.175  1.00 26.27           N  \\nATOM    407  CA  LYS B  29      -3.227  18.941  13.307  1.00 23.17           C  \\nATOM    408  C   LYS B  29      -1.707  18.995  13.459  1.00 52.81           C  \\nATOM    409  O   LYS B  29      -1.026  17.919  13.406  1.00 39.37           O  \\nATOM    410  CB  LYS B  29      -3.764  18.417  14.615  1.00 22.26           C  \\nATOM    411  CG  LYS B  29      -3.990  19.385  15.801  1.00 48.01           C  \\nATOM    412  CD  LYS B  29      -5.153  18.811  16.622  1.00 37.36           C  \\nATOM    413  CE  LYS B  29      -5.067  18.493  18.087  1.00 53.09           C  \\nATOM    414  NZ  LYS B  29      -4.208  19.418  18.841  1.00 61.16           N  \\nATOM    415  N   ALA B  30      -1.166  20.052  13.779  1.00 53.30           N  \\nATOM    416  CA  ALA B  30       0.148  20.539  13.902  1.00 45.30           C  \\nATOM    417  C   ALA B  30       0.991  20.467  15.167  1.00 50.30           C  \\nATOM    418  O   ALA B  30       0.427  20.710  16.268  1.00 62.63           O  \\nATOM    419  CB  ALA B  30       0.033  22.113  13.690  1.00 53.30           C  \\nATOM    420  OXT ALA B  30       2.226  20.205  15.000  1.00 76.30           O  \\nTER     421      ALA B  30                                                      \\nATOM    422  N   GLY C   1      -0.643  19.956 -14.073  1.00 26.16           N  \\nATOM    423  CA  GLY C   1      -0.389  20.033 -12.615  1.00 30.96           C  \\nATOM    424  C   GLY C   1       0.447  18.825 -12.180  1.00 33.76           C  \\nATOM    425  O   GLY C   1       1.216  18.311 -13.006  1.00 21.35           O  \\nATOM    426  N   ILE C   2       0.244  18.434 -10.942  1.00 23.96           N  \\nATOM    427  CA  ILE C   2       1.003  17.290 -10.393  1.00 15.36           C  \\nATOM    428  C   ILE C   2       0.946  16.025 -11.185  1.00 13.59           C  \\nATOM    429  O   ILE C   2       1.971  15.359 -11.278  1.00 17.19           O  \\nATOM    430  CB  ILE C   2       0.491  17.013  -8.931  1.00 16.47           C  \\nATOM    431  CG1 ILE C   2       1.539  16.143  -8.164  1.00 15.91           C  \\nATOM    432  CG2 ILE C   2      -0.969  16.533  -8.863  1.00 17.06           C  \\nATOM    433  CD1 ILE C   2       1.081  15.828  -6.720  1.00 19.56           C  \\nATOM    434  N   VAL C   3      -0.179  15.655 -11.786  1.00 13.55           N  \\nATOM    435  CA  VAL C   3      -0.278  14.455 -12.591  1.00 17.58           C  \\nATOM    436  C   VAL C   3       0.590  14.454 -13.881  1.00 21.18           C  \\nATOM    437  O   VAL C   3       1.245  13.451 -14.197  1.00 19.26           O  \\nATOM    438  CB  VAL C   3      -1.709  14.136 -12.915  1.00 26.49           C  \\nATOM    439  CG1 VAL C   3      -1.808  12.868 -13.745  1.00 30.71           C  \\nATOM    440  CG2 VAL C   3      -2.567  13.989 -11.662  1.00 16.85           C  \\nATOM    441  N   GLU C   4       0.621  15.590 -14.497  1.00 24.23           N  \\nATOM    442  CA  GLU C   4       1.481  15.743 -15.686  1.00 23.06           C  \\nATOM    443  C   GLU C   4       2.966  15.744 -15.343  1.00 19.78           C  \\nATOM    444  O   GLU C   4       3.805  15.198 -16.061  1.00 29.30           O  \\nATOM    445  CB  GLU C   4       1.063  17.059 -16.361  1.00 24.43           C  \\nATOM    446  CG  GLU C   4      -0.349  16.900 -16.964  1.00 19.87           C  \\nATOM    447  CD  GLU C   4      -1.541  17.024 -16.030  1.00 27.93           C  \\nATOM    448  OE1 GLU C   4      -2.429  16.169 -16.211  1.00 37.08           O  \\nATOM    449  OE2 GLU C   4      -1.527  17.967 -15.181  1.00 27.92           O  \\nATOM    450  N   GLN C   5       3.289  16.386 -14.236  1.00 17.90           N  \\nATOM    451  CA  GLN C   5       4.695  16.445 -13.800  1.00 17.90           C  \\nATOM    452  C   GLN C   5       5.206  15.104 -13.307  1.00 23.62           C  \\nATOM    453  O   GLN C   5       6.331  14.684 -13.578  1.00 26.25           O  \\nATOM    454  CB  GLN C   5       4.820  17.553 -12.780  1.00 22.64           C  \\nATOM    455  CG  GLN C   5       4.373  18.969 -13.199  1.00 32.56           C  \\nATOM    456  CD  GLN C   5       5.405  19.755 -12.404  1.00 59.85           C  \\nATOM    457  OE1 GLN C   5       6.478  19.979 -12.947  1.00 50.22           O  \\nATOM    458  NE2 GLN C   5       4.976  19.940 -11.148  1.00 56.46           N  \\nATOM    459  N   CYS C   6       4.386  14.408 -12.514  1.00 20.45           N  \\nATOM    460  CA  CYS C   6       4.949  13.166 -11.921  1.00 20.95           C  \\nATOM    461  C   CYS C   6       4.472  11.875 -12.436  1.00 18.67           C  \\nATOM    462  O   CYS C   6       5.127  10.851 -12.293  1.00 17.98           O  \\nATOM    463  CB  CYS C   6       4.732  13.264 -10.354  1.00 16.76           C  \\nATOM    464  SG  CYS C   6       5.416  14.719  -9.509  1.00 20.00           S  \\nATOM    465  N   CYS C   7       3.347  11.802 -13.063  1.00 15.93           N  \\nATOM    466  CA  CYS C   7       2.858  10.515 -13.612  1.00 10.31           C  \\nATOM    467  C   CYS C   7       3.090  10.422 -15.112  1.00 27.34           C  \\nATOM    468  O   CYS C   7       3.601   9.455 -15.603  1.00 24.77           O  \\nATOM    469  CB  CYS C   7       1.348  10.332 -13.329  1.00 17.60           C  \\nATOM    470  SG  CYS C   7       0.602   8.899 -13.976  1.00 19.30           S  \\nATOM    471  N   THR C   8       2.691  11.395 -15.898  1.00 22.31           N  \\nATOM    472  CA  THR C   8       2.912  11.356 -17.382  1.00 20.22           C  \\nATOM    473  C   THR C   8       4.408  11.508 -17.641  1.00 29.24           C  \\nATOM    474  O   THR C   8       5.051  10.716 -18.357  1.00 25.27           O  \\nATOM    475  CB  THR C   8       1.993  12.487 -17.997  1.00 28.76           C  \\nATOM    476  OG1 THR C   8       0.590  12.071 -17.868  1.00 41.93           O  \\nATOM    477  CG2 THR C   8       2.536  12.795 -19.370  1.00 39.10           C  \\nATOM    478  N   SER C   9       4.957  12.549 -16.969  1.00 23.04           N  \\nATOM    479  CA  SER C   9       6.438  12.783 -17.031  1.00 23.81           C  \\nATOM    480  C   SER C   9       7.038  12.104 -15.796  1.00 22.97           C  \\nATOM    481  O   SER C   9       6.505  11.080 -15.351  1.00 28.36           O  \\nATOM    482  CB  SER C   9       6.838  14.249 -17.258  1.00 22.96           C  \\nATOM    483  OG  SER C   9       8.266  14.523 -17.319  1.00 30.96           O  \\nATOM    484  N   ILE C  10       8.157  12.532 -15.274  1.00 18.48           N  \\nATOM    485  CA  ILE C  10       8.820  12.068 -14.116  1.00 14.18           C  \\nATOM    486  C   ILE C  10       9.162  13.268 -13.217  1.00 22.06           C  \\nATOM    487  O   ILE C  10       9.398  14.336 -13.807  1.00 21.37           O  \\nATOM    488  CB  ILE C  10      10.082  11.219 -14.296  1.00 14.27           C  \\nATOM    489  CG1 ILE C  10      11.168  12.058 -14.941  1.00 17.05           C  \\nATOM    490  CG2 ILE C  10       9.771   9.901 -15.075  1.00 23.12           C  \\nATOM    491  CD1 ILE C  10      12.447  11.250 -15.181  1.00 28.86           C  \\nATOM    492  N   CYS C  11       9.158  13.001 -11.897  1.00 17.73           N  \\nATOM    493  CA  CYS C  11       9.488  14.178 -11.063  1.00 13.63           C  \\nATOM    494  C   CYS C  11      10.301  13.733  -9.869  1.00 24.12           C  \\nATOM    495  O   CYS C  11      10.238  12.518  -9.595  1.00 27.63           O  \\nATOM    496  CB  CYS C  11       8.284  14.934 -10.501  1.00 24.47           C  \\nATOM    497  SG  CYS C  11       7.324  13.971  -9.301  1.00 22.60           S  \\nATOM    498  N   SER C  12      10.941  14.682  -9.241  1.00 20.64           N  \\nATOM    499  CA  SER C  12      11.739  14.434  -8.095  1.00 19.89           C  \\nATOM    500  C   SER C  12      10.978  14.612  -6.784  1.00 26.49           C  \\nATOM    501  O   SER C  12       9.925  15.241  -6.843  1.00 17.94           O  \\nATOM    502  CB  SER C  12      12.896  15.478  -7.911  1.00 22.13           C  \\nATOM    503  OG  SER C  12      12.280  16.737  -7.889  1.00 25.55           O  \\nATOM    504  N   LEU C  13      11.522  14.165  -5.654  1.00 20.92           N  \\nATOM    505  CA  LEU C  13      10.827  14.392  -4.422  1.00 18.59           C  \\nATOM    506  C   LEU C  13      10.811  15.860  -4.099  1.00 17.62           C  \\nATOM    507  O   LEU C  13       9.865  16.311  -3.427  1.00 17.56           O  \\nATOM    508  CB  LEU C  13      11.451  13.616  -3.261  1.00 24.54           C  \\nATOM    509  CG  LEU C  13      11.620  12.137  -3.475  1.00 37.60           C  \\nATOM    510  CD1 LEU C  13      12.386  11.586  -2.273  1.00 42.94           C  \\nATOM    511  CD2 LEU C  13      10.209  11.562  -3.628  1.00 33.13           C  \\nATOM    512  N   TYR C  14      11.756  16.642  -4.538  1.00 19.39           N  \\nATOM    513  CA  TYR C  14      11.756  18.053  -4.253  1.00 15.81           C  \\nATOM    514  C   TYR C  14      10.632  18.665  -4.977  1.00 16.49           C  \\nATOM    515  O   TYR C  14      10.098  19.673  -4.517  1.00 24.31           O  \\nATOM    516  CB  TYR C  14      13.061  18.792  -4.698  1.00 23.42           C  \\nATOM    517  CG  TYR C  14      14.244  18.334  -3.884  1.00 22.68           C  \\nATOM    518  CD1 TYR C  14      14.406  18.926  -2.643  1.00 37.43           C  \\nATOM    519  CD2 TYR C  14      15.065  17.294  -4.321  1.00 44.52           C  \\nATOM    520  CE1 TYR C  14      15.469  18.455  -1.806  1.00 35.18           C  \\nATOM    521  CE2 TYR C  14      16.138  16.837  -3.558  1.00 39.13           C  \\nATOM    522  CZ  TYR C  14      16.305  17.440  -2.304  1.00 43.24           C  \\nATOM    523  OH  TYR C  14      17.325  17.067  -1.472  1.00 40.09           O  \\nATOM    524  N   GLN C  15      10.286  18.189  -6.156  1.00 16.44           N  \\nATOM    525  CA  GLN C  15       9.175  18.706  -6.908  1.00 13.87           C  \\nATOM    526  C   GLN C  15       7.940  18.402  -6.108  1.00 16.45           C  \\nATOM    527  O   GLN C  15       7.042  19.228  -6.047  1.00 17.74           O  \\nATOM    528  CB  GLN C  15       9.021  18.202  -8.378  1.00 19.83           C  \\nATOM    529  CG  GLN C  15      10.045  18.695  -9.368  1.00 27.50           C  \\nATOM    530  CD  GLN C  15      10.002  18.165 -10.771  1.00 46.46           C  \\nATOM    531  OE1 GLN C  15      10.459  17.057 -11.040  1.00 43.32           O  \\nATOM    532  NE2 GLN C  15       9.456  18.889 -11.761  1.00 35.38           N  \\nATOM    533  N   LEU C  16       7.841  17.245  -5.491  1.00 18.47           N  \\nATOM    534  CA  LEU C  16       6.580  16.984  -4.783  1.00 17.18           C  \\nATOM    535  C   LEU C  16       6.311  17.904  -3.587  1.00 21.36           C  \\nATOM    536  O   LEU C  16       5.195  18.069  -3.079  1.00 17.34           O  \\nATOM    537  CB  LEU C  16       6.542  15.527  -4.227  1.00 17.15           C  \\nATOM    538  CG  LEU C  16       6.500  14.520  -5.341  1.00 17.53           C  \\nATOM    539  CD1 LEU C  16       6.477  13.227  -4.593  1.00 22.82           C  \\nATOM    540  CD2 LEU C  16       5.117  14.493  -5.991  1.00 18.33           C  \\nATOM    541  N   GLU C  17       7.389  18.427  -3.040  1.00 16.47           N  \\nATOM    542  CA  GLU C  17       7.353  19.319  -1.959  1.00 10.93           C  \\nATOM    543  C   GLU C  17       6.614  20.601  -2.282  1.00 11.20           C  \\nATOM    544  O   GLU C  17       6.183  21.310  -1.386  1.00 13.69           O  \\nATOM    545  CB  GLU C  17       8.751  19.800  -1.584  1.00 16.99           C  \\nATOM    546  CG  GLU C  17       9.178  18.980  -0.408  1.00 22.69           C  \\nATOM    547  CD  GLU C  17      10.397  19.524   0.269  1.00 26.38           C  \\nATOM    548  OE1 GLU C  17      11.270  20.013  -0.342  1.00 26.42           O  \\nATOM    549  OE2 GLU C  17      10.358  19.291   1.470  1.00 30.68           O  \\nATOM    550  N   ASN C  18       6.466  20.844  -3.575  1.00 13.42           N  \\nATOM    551  CA  ASN C  18       5.707  22.010  -4.006  1.00 12.91           C  \\nATOM    552  C   ASN C  18       4.238  21.879  -3.683  1.00 14.96           C  \\nATOM    553  O   ASN C  18       3.506  22.887  -3.698  1.00 17.72           O  \\nATOM    554  CB  ASN C  18       5.894  22.279  -5.506  1.00 16.62           C  \\nATOM    555  CG  ASN C  18       7.286  22.755  -5.884  1.00 20.64           C  \\nATOM    556  OD1 ASN C  18       7.761  22.375  -6.927  1.00 28.41           O  \\nATOM    557  ND2 ASN C  18       7.836  23.540  -4.999  1.00 26.53           N  \\nATOM    558  N   TYR C  19       3.683  20.717  -3.420  1.00 10.42           N  \\nATOM    559  CA  TYR C  19       2.315  20.489  -3.110  1.00 11.44           C  \\nATOM    560  C   TYR C  19       2.035  20.284  -1.641  1.00 19.22           C  \\nATOM    561  O   TYR C  19       0.884  20.008  -1.330  1.00 14.52           O  \\nATOM    562  CB  TYR C  19       1.745  19.342  -3.938  1.00 13.89           C  \\nATOM    563  CG  TYR C  19       1.995  19.579  -5.394  1.00 13.15           C  \\nATOM    564  CD1 TYR C  19       1.176  20.360  -6.185  1.00 15.85           C  \\nATOM    565  CD2 TYR C  19       3.101  19.022  -6.004  1.00 20.18           C  \\nATOM    566  CE1 TYR C  19       1.482  20.590  -7.526  1.00 15.80           C  \\nATOM    567  CE2 TYR C  19       3.427  19.257  -7.342  1.00 20.94           C  \\nATOM    568  CZ  TYR C  19       2.600  20.077  -8.111  1.00 16.07           C  \\nATOM    569  OH  TYR C  19       2.930  20.198  -9.438  1.00 28.20           O  \\nATOM    570  N   CYS C  20       2.997  20.523  -0.779  1.00 15.85           N  \\nATOM    571  CA  CYS C  20       2.816  20.515   0.660  1.00 12.75           C  \\nATOM    572  C   CYS C  20       2.167  21.848   1.080  1.00 18.29           C  \\nATOM    573  O   CYS C  20       2.424  22.842   0.449  1.00 26.63           O  \\nATOM    574  CB  CYS C  20       4.116  20.329   1.465  1.00 12.02           C  \\nATOM    575  SG  CYS C  20       5.109  18.925   1.079  1.00 13.70           S  \\nATOM    576  N   ASN C  21       1.396  21.899   2.168  1.00 17.14           N  \\nATOM    577  CA  ASN C  21       0.814  23.152   2.677  1.00 22.18           C  \\nATOM    578  C   ASN C  21       1.879  23.650   3.681  1.00 31.46           C  \\nATOM    579  O   ASN C  21       2.937  22.991   3.836  1.00 44.77           O  \\nATOM    580  CB  ASN C  21      -0.433  22.757   3.387  1.00 21.67           C  \\nATOM    581  CG  ASN C  21      -1.636  22.166   2.742  1.00 29.17           C  \\nATOM    582  OD1 ASN C  21      -2.062  22.468   1.647  1.00 33.63           O  \\nATOM    583  ND2 ASN C  21      -2.242  21.277   3.591  1.00 33.21           N  \\nATOM    584  OXT ASN C  21       1.575  24.585   4.398  1.00 52.38           O  \\nTER     585      ASN C  21                                                      \\nATOM    586  N   PHE D   1      18.330  11.816  -3.893  1.00 61.50           N  \\nATOM    587  CA  PHE D   1      17.047  11.272  -4.371  1.00 28.75           C  \\nATOM    588  C   PHE D   1      17.165  10.885  -5.854  1.00 45.77           C  \\nATOM    589  O   PHE D   1      18.169  11.137  -6.550  1.00 49.85           O  \\nATOM    590  CB  PHE D   1      15.801  12.158  -4.041  1.00 30.01           C  \\nATOM    591  CG  PHE D   1      16.035  12.424  -2.571  1.00 31.74           C  \\nATOM    592  CD1 PHE D   1      15.980  11.294  -1.745  1.00 39.79           C  \\nATOM    593  CD2 PHE D   1      16.425  13.629  -2.070  1.00 31.26           C  \\nATOM    594  CE1 PHE D   1      16.219  11.375  -0.405  1.00 31.66           C  \\nATOM    595  CE2 PHE D   1      16.706  13.719  -0.715  1.00 53.74           C  \\nATOM    596  CZ  PHE D   1      16.609  12.612   0.115  1.00 37.15           C  \\nATOM    597  N   VAL D   2      16.117  10.168  -6.274  1.00 33.71           N  \\nATOM    598  CA  VAL D   2      16.006   9.666  -7.682  1.00 24.16           C  \\nATOM    599  C   VAL D   2      14.648  10.218  -8.239  1.00 33.79           C  \\nATOM    600  O   VAL D   2      13.733  10.588  -7.495  1.00 22.26           O  \\nATOM    601  CB  VAL D   2      16.128   8.144  -7.911  1.00 27.80           C  \\nATOM    602  CG1 VAL D   2      17.529   7.562  -7.875  1.00 85.81           C  \\nATOM    603  CG2 VAL D   2      15.239   7.326  -6.976  1.00 36.43           C  \\nATOM    604  N   ASN D   3      14.564  10.315  -9.558  1.00 19.91           N  \\nATOM    605  CA  ASN D   3      13.383  10.694 -10.332  1.00 33.51           C  \\nATOM    606  C   ASN D   3      12.449   9.525 -10.618  1.00 27.19           C  \\nATOM    607  O   ASN D   3      12.946   8.479 -11.058  1.00 35.28           O  \\nATOM    608  CB  ASN D   3      13.966  11.427 -11.571  1.00 28.03           C  \\nATOM    609  CG  ASN D   3      14.214  12.869 -11.102  1.00 36.72           C  \\nATOM    610  OD1 ASN D   3      14.425  13.813 -11.829  1.00 71.83           O  \\nATOM    611  ND2 ASN D   3      14.114  13.118  -9.791  1.00 69.95           N  \\nATOM    612  N   GLN D   4      11.128   9.652 -10.488  1.00 38.71           N  \\nATOM    613  CA  GLN D   4      10.224   8.523 -10.782  1.00 30.37           C  \\nATOM    614  C   GLN D   4       8.833   8.864 -11.345  1.00 13.92           C  \\nATOM    615  O   GLN D   4       8.361  10.028 -11.197  1.00 25.56           O  \\nATOM    616  CB  GLN D   4       9.983   7.921  -9.378  1.00 38.84           C  \\nATOM    617  CG  GLN D   4      10.350   6.429  -9.194  1.00 40.71           C  \\nATOM    618  CD  GLN D   4      10.515   6.248  -7.678  1.00 31.29           C  \\nATOM    619  OE1 GLN D   4      10.490   5.138  -7.268  1.00 47.62           O  \\nATOM    620  NE2 GLN D   4      10.697   7.383  -6.997  1.00 30.17           N  \\nATOM    621  N   HIS D   5       8.291   7.803 -12.000  1.00 13.87           N  \\nATOM    622  CA  HIS D   5       6.903   7.919 -12.482  1.00 16.22           C  \\nATOM    623  C   HIS D   5       6.029   7.528 -11.252  1.00 16.61           C  \\nATOM    624  O   HIS D   5       6.115   6.355 -10.822  1.00 21.77           O  \\nATOM    625  CB  HIS D   5       6.427   7.086 -13.663  1.00 24.93           C  \\nATOM    626  CG  HIS D   5       6.855   7.450 -15.054  1.00 32.91           C  \\nATOM    627  ND1 HIS D   5       6.285   8.338 -15.876  1.00 31.86           N  \\nATOM    628  CD2 HIS D   5       7.867   6.966 -15.822  1.00 31.55           C  \\nATOM    629  CE1 HIS D   5       6.866   8.458 -17.068  1.00 28.50           C  \\nATOM    630  NE2 HIS D   5       7.838   7.587 -17.015  1.00 18.29           N  \\nATOM    631  N   LEU D   6       5.259   8.452 -10.792  1.00 11.23           N  \\nATOM    632  CA  LEU D   6       4.302   8.232  -9.639  1.00 22.97           C  \\nATOM    633  C   LEU D   6       2.854   8.566 -10.048  1.00 11.62           C  \\nATOM    634  O   LEU D   6       2.505   9.705 -10.335  1.00 14.32           O  \\nATOM    635  CB  LEU D   6       4.815   9.069  -8.489  1.00 16.96           C  \\nATOM    636  CG  LEU D   6       6.253   8.828  -7.985  1.00 18.92           C  \\nATOM    637  CD1 LEU D   6       6.828  10.000  -7.271  1.00 17.11           C  \\nATOM    638  CD2 LEU D   6       6.091   7.594  -7.091  1.00 18.40           C  \\nATOM    639  N   CYS D   7       2.119   7.495 -10.108  1.00 13.70           N  \\nATOM    640  CA  CYS D   7       0.745   7.570 -10.582  1.00 18.64           C  \\nATOM    641  C   CYS D   7      -0.288   7.058  -9.564  1.00 17.90           C  \\nATOM    642  O   CYS D   7       0.016   6.138  -8.853  1.00 18.63           O  \\nATOM    643  CB  CYS D   7       0.607   6.723 -11.869  1.00 31.44           C  \\nATOM    644  SG  CYS D   7       1.635   7.350 -13.232  1.00 20.00           S  \\nATOM    645  N   GLY D   8      -1.440   7.618  -9.644  1.00 19.94           N  \\nATOM    646  CA  GLY D   8      -2.580   7.155  -8.776  1.00 18.12           C  \\nATOM    647  C   GLY D   8      -2.305   7.237  -7.329  1.00  8.63           C  \\nATOM    648  O   GLY D   8      -1.914   8.246  -6.763  1.00 13.74           O  \\nATOM    649  N   SER D   9      -2.656   6.101  -6.694  1.00 11.65           N  \\nATOM    650  CA  SER D   9      -2.483   5.997  -5.227  1.00 12.39           C  \\nATOM    651  C   SER D   9      -1.032   6.120  -4.867  1.00  8.86           C  \\nATOM    652  O   SER D   9      -0.760   6.584  -3.762  1.00  9.84           O  \\nATOM    653  CB  SER D   9      -3.264   4.870  -4.568  1.00 14.54           C  \\nATOM    654  OG  SER D   9      -2.756   3.670  -5.015  1.00 21.40           O  \\nATOM    655  N   HIS D  10      -0.166   5.698  -5.800  1.00  8.91           N  \\nATOM    656  CA  HIS D  10       1.270   5.805  -5.452  1.00  9.09           C  \\nATOM    657  C   HIS D  10       1.755   7.227  -5.317  1.00  7.69           C  \\nATOM    658  O   HIS D  10       2.658   7.481  -4.533  1.00 11.42           O  \\nATOM    659  CB  HIS D  10       2.092   5.048  -6.482  1.00  8.95           C  \\nATOM    660  CG  HIS D  10       1.707   3.599  -6.615  1.00  8.78           C  \\nATOM    661  ND1 HIS D  10       1.951   2.701  -5.608  1.00 13.99           N  \\nATOM    662  CD2 HIS D  10       1.059   2.922  -7.561  1.00 11.16           C  \\nATOM    663  CE1 HIS D  10       1.536   1.530  -6.025  1.00  8.79           C  \\nATOM    664  NE2 HIS D  10       0.948   1.628  -7.159  1.00 11.60           N  \\nATOM    665  N   LEU D  11       1.142   8.109  -6.070  1.00  9.45           N  \\nATOM    666  CA  LEU D  11       1.434   9.541  -6.053  1.00  9.98           C  \\nATOM    667  C   LEU D  11       1.027  10.155  -4.729  1.00  5.91           C  \\nATOM    668  O   LEU D  11       1.818  10.853  -4.084  1.00  8.83           O  \\nATOM    669  CB  LEU D  11       0.815  10.196  -7.249  1.00 10.51           C  \\nATOM    670  CG  LEU D  11       1.105  11.705  -7.294  1.00 13.37           C  \\nATOM    671  CD1 LEU D  11       2.526  12.118  -7.270  1.00 11.72           C  \\nATOM    672  CD2 LEU D  11       0.404  12.225  -8.564  1.00 16.22           C  \\nATOM    673  N   VAL D  12      -0.188   9.816  -4.302  1.00  6.52           N  \\nATOM    674  CA  VAL D  12      -0.534  10.395  -2.938  1.00  6.95           C  \\nATOM    675  C   VAL D  12       0.207   9.758  -1.787  1.00  5.55           C  \\nATOM    676  O   VAL D  12       0.467  10.475  -0.761  1.00  8.60           O  \\nATOM    677  CB  VAL D  12      -2.065  10.355  -2.790  1.00 14.07           C  \\nATOM    678  CG1 VAL D  12      -2.687  11.271  -3.899  1.00 16.64           C  \\nATOM    679  CG2 VAL D  12      -2.636   9.010  -2.737  1.00 14.03           C  \\nATOM    680  N   GLU D  13       0.681   8.537  -1.890  1.00  8.61           N  \\nATOM    681  CA  GLU D  13       1.509   7.903  -0.866  1.00  7.25           C  \\nATOM    682  C   GLU D  13       2.815   8.631  -0.804  1.00  7.02           C  \\nATOM    683  O   GLU D  13       3.403   8.915   0.274  1.00 11.89           O  \\nATOM    684  CB  GLU D  13       1.770   6.435  -1.051  1.00 11.84           C  \\nATOM    685  CG  GLU D  13       0.458   5.634  -0.816  1.00 27.71           C  \\nATOM    686  CD  GLU D  13       0.536   4.937   0.492  1.00 62.15           C  \\nATOM    687  OE1 GLU D  13       1.499   5.094   1.172  1.00 28.12           O  \\nATOM    688  OE2 GLU D  13      -0.429   4.258   0.726  1.00 32.97           O  \\nATOM    689  N   ALA D  14       3.390   9.046  -1.888  1.00  9.58           N  \\nATOM    690  CA  ALA D  14       4.643   9.836  -1.985  1.00 10.00           C  \\nATOM    691  C   ALA D  14       4.495  11.218  -1.377  1.00 10.02           C  \\nATOM    692  O   ALA D  14       5.373  11.718  -0.675  1.00 12.34           O  \\nATOM    693  CB  ALA D  14       5.151   9.884  -3.366  1.00  9.23           C  \\nATOM    694  N   LEU D  15       3.340  11.836  -1.609  1.00  8.93           N  \\nATOM    695  CA  LEU D  15       3.014  13.121  -1.053  1.00  9.97           C  \\nATOM    696  C   LEU D  15       2.968  13.020   0.450  1.00 16.23           C  \\nATOM    697  O   LEU D  15       3.462  13.914   1.142  1.00 11.63           O  \\nATOM    698  CB  LEU D  15       1.719  13.654  -1.641  1.00  9.29           C  \\nATOM    699  CG  LEU D  15       1.771  14.340  -2.979  1.00 10.49           C  \\nATOM    700  CD1 LEU D  15       0.398  14.483  -3.556  1.00 12.99           C  \\nATOM    701  CD2 LEU D  15       2.487  15.663  -2.804  1.00 15.87           C  \\nATOM    702  N   TYR D  16       2.313  11.974   0.938  1.00 11.90           N  \\nATOM    703  CA  TYR D  16       2.254  11.791   2.374  1.00  8.84           C  \\nATOM    704  C   TYR D  16       3.662  11.710   3.013  1.00 11.08           C  \\nATOM    705  O   TYR D  16       3.925  12.338   4.014  1.00 10.74           O  \\nATOM    706  CB  TYR D  16       1.409  10.545   2.657  1.00  8.68           C  \\nATOM    707  CG  TYR D  16       1.408  10.201   4.150  1.00  9.93           C  \\nATOM    708  CD1 TYR D  16       0.611  10.879   5.091  1.00 13.27           C  \\nATOM    709  CD2 TYR D  16       2.232   9.163   4.535  1.00 10.34           C  \\nATOM    710  CE1 TYR D  16       0.657  10.554   6.429  1.00 11.61           C  \\nATOM    711  CE2 TYR D  16       2.284   8.846   5.894  1.00 17.86           C  \\nATOM    712  CZ  TYR D  16       1.518   9.554   6.843  1.00 13.23           C  \\nATOM    713  OH  TYR D  16       1.664   9.081   8.177  1.00 17.80           O  \\nATOM    714  N   LEU D  17       4.478  10.910   2.349  1.00  9.51           N  \\nATOM    715  CA  LEU D  17       5.832  10.719   2.819  1.00  8.94           C  \\nATOM    716  C   LEU D  17       6.672  11.993   2.879  1.00 12.51           C  \\nATOM    717  O   LEU D  17       7.395  12.298   3.853  1.00 13.68           O  \\nATOM    718  CB  LEU D  17       6.573   9.683   1.932  1.00 12.01           C  \\nATOM    719  CG  LEU D  17       7.784   9.141   2.606  1.00 26.75           C  \\nATOM    720  CD1 LEU D  17       7.274   8.084   3.613  1.00 27.77           C  \\nATOM    721  CD2 LEU D  17       8.533   8.579   1.435  1.00 36.25           C  \\nATOM    722  N   VAL D  18       6.585  12.773   1.784  1.00 10.21           N  \\nATOM    723  CA  VAL D  18       7.348  14.029   1.661  1.00  8.25           C  \\nATOM    724  C   VAL D  18       6.789  15.099   2.514  1.00  8.01           C  \\nATOM    725  O   VAL D  18       7.560  15.914   3.063  1.00 14.17           O  \\nATOM    726  CB  VAL D  18       7.357  14.409   0.176  1.00  8.24           C  \\nATOM    727  CG1 VAL D  18       7.683  15.903   0.006  1.00 18.17           C  \\nATOM    728  CG2 VAL D  18       8.321  13.582  -0.638  1.00 16.22           C  \\nATOM    729  N   CYS D  19       5.469  15.224   2.565  1.00  8.80           N  \\nATOM    730  CA  CYS D  19       4.790  16.327   3.282  1.00 10.89           C  \\nATOM    731  C   CYS D  19       4.550  16.076   4.760  1.00 20.41           C  \\nATOM    732  O   CYS D  19       4.771  17.057   5.513  1.00 17.13           O  \\nATOM    733  CB  CYS D  19       3.540  16.778   2.546  1.00 11.71           C  \\nATOM    734  SG  CYS D  19       3.846  17.363   0.909  1.00 11.30           S  \\nATOM    735  N   GLY D  20       4.108  14.860   5.076  1.00 15.31           N  \\nATOM    736  CA  GLY D  20       3.841  14.553   6.475  1.00 18.10           C  \\nATOM    737  C   GLY D  20       2.815  15.479   7.177  1.00 19.19           C  \\nATOM    738  O   GLY D  20       1.669  15.721   6.770  1.00 18.72           O  \\nATOM    739  N   GLU D  21       3.387  15.976   8.278  1.00 20.01           N  \\nATOM    740  CA  GLU D  21       2.659  16.847   9.244  1.00 23.69           C  \\nATOM    741  C   GLU D  21       2.237  18.138   8.636  1.00 19.09           C  \\nATOM    742  O   GLU D  21       1.246  18.727   9.109  1.00 24.03           O  \\nATOM    743  CB  GLU D  21       3.575  17.120  10.476  1.00 22.76           C  \\nATOM    744  CG  GLU D  21       5.075  17.192  10.046  1.00 53.97           C  \\nATOM    745  CD  GLU D  21       6.099  17.832  10.938  1.00 78.86           C  \\nATOM    746  OE1 GLU D  21       5.775  18.563  11.863  1.00 73.07           O  \\nATOM    747  OE2 GLU D  21       7.300  17.568  10.645  1.00 72.74           O  \\nATOM    748  N   ARG D  22       2.952  18.555   7.566  1.00 15.75           N  \\nATOM    749  CA  ARG D  22       2.552  19.835   6.996  1.00 21.02           C  \\nATOM    750  C   ARG D  22       1.231  19.779   6.228  1.00 30.79           C  \\nATOM    751  O   ARG D  22       0.558  20.740   5.955  1.00 22.02           O  \\nATOM    752  CB  ARG D  22       3.613  20.247   5.973  1.00 29.63           C  \\nATOM    753  CG  ARG D  22       5.024  20.325   6.450  1.00 16.81           C  \\nATOM    754  CD  ARG D  22       5.952  20.706   5.309  1.00 32.57           C  \\nATOM    755  NE AARG D  22       5.584  21.912   4.554  0.50 23.88           N  \\nATOM    756  NE BARG D  22       6.663  19.518   4.827  0.50 24.77           N  \\nATOM    757  CZ AARG D  22       6.163  22.431   3.476  0.50 34.85           C  \\nATOM    758  CZ BARG D  22       7.482  19.684   3.777  0.50 11.98           C  \\nATOM    759  NH1AARG D  22       5.819  23.547   2.845  0.50 18.26           N  \\nATOM    760  NH1BARG D  22       7.547  20.848   3.124  0.50 20.33           N  \\nATOM    761  NH2AARG D  22       7.190  21.770   2.930  0.50 20.80           N  \\nATOM    762  NH2BARG D  22       8.196  18.646   3.318  0.50 20.38           N  \\nATOM    763  N   GLY D  23       0.914  18.555   5.836  1.00 17.93           N  \\nATOM    764  CA  GLY D  23      -0.237  18.302   4.993  1.00 12.82           C  \\nATOM    765  C   GLY D  23       0.087  18.615   3.514  1.00 11.10           C  \\nATOM    766  O   GLY D  23       1.137  19.102   3.198  1.00 12.49           O  \\nATOM    767  N   PHE D  24      -0.903  18.290   2.672  1.00 11.76           N  \\nATOM    768  CA  PHE D  24      -0.774  18.497   1.209  1.00 10.21           C  \\nATOM    769  C   PHE D  24      -2.127  18.688   0.607  1.00  9.65           C  \\nATOM    770  O   PHE D  24      -3.142  18.470   1.204  1.00 12.03           O  \\nATOM    771  CB  PHE D  24       0.000  17.400   0.578  1.00  9.41           C  \\nATOM    772  CG  PHE D  24      -0.602  16.046   0.597  1.00  9.89           C  \\nATOM    773  CD1 PHE D  24      -0.280  15.182   1.648  1.00 12.55           C  \\nATOM    774  CD2 PHE D  24      -1.497  15.657  -0.387  1.00 12.14           C  \\nATOM    775  CE1 PHE D  24      -0.863  13.901   1.714  1.00 19.03           C  \\nATOM    776  CE2 PHE D  24      -2.049  14.388  -0.368  1.00 12.53           C  \\nATOM    777  CZ  PHE D  24      -1.737  13.486   0.692  1.00 14.23           C  \\nATOM    778  N   PHE D  25      -2.070  19.069  -0.693  1.00  8.89           N  \\nATOM    779  CA  PHE D  25      -3.285  19.185  -1.469  1.00 10.03           C  \\nATOM    780  C   PHE D  25      -3.021  18.326  -2.724  1.00  9.84           C  \\nATOM    781  O   PHE D  25      -1.918  18.362  -3.216  1.00 12.46           O  \\nATOM    782  CB  PHE D  25      -3.686  20.613  -1.925  1.00 10.32           C  \\nATOM    783  CG  PHE D  25      -2.567  21.455  -2.479  1.00 14.41           C  \\nATOM    784  CD1 PHE D  25      -2.435  21.546  -3.850  1.00 20.47           C  \\nATOM    785  CD2 PHE D  25      -1.647  22.070  -1.651  1.00 14.83           C  \\nATOM    786  CE1 PHE D  25      -1.470  22.321  -4.476  1.00 13.89           C  \\nATOM    787  CE2 PHE D  25      -0.608  22.775  -2.271  1.00 20.12           C  \\nATOM    788  CZ  PHE D  25      -0.558  22.880  -3.660  1.00 13.72           C  \\nATOM    789  N   TYR D  26      -3.962  17.546  -3.192  1.00 10.77           N  \\nATOM    790  CA  TYR D  26      -3.915  16.723  -4.366  1.00  8.24           C  \\nATOM    791  C   TYR D  26      -5.013  17.222  -5.272  1.00 11.68           C  \\nATOM    792  O   TYR D  26      -6.209  17.040  -5.011  1.00 12.49           O  \\nATOM    793  CB  TYR D  26      -4.090  15.233  -4.010  1.00 12.91           C  \\nATOM    794  CG  TYR D  26      -4.159  14.291  -5.196  1.00 13.37           C  \\nATOM    795  CD1 TYR D  26      -5.379  13.599  -5.455  1.00 12.94           C  \\nATOM    796  CD2 TYR D  26      -3.138  14.227  -6.110  1.00 12.82           C  \\nATOM    797  CE1 TYR D  26      -5.491  12.770  -6.582  1.00 18.45           C  \\nATOM    798  CE2 TYR D  26      -3.289  13.401  -7.216  1.00 14.97           C  \\nATOM    799  CZ  TYR D  26      -4.390  12.679  -7.390  1.00 16.87           C  \\nATOM    800  OH  TYR D  26      -4.411  11.849  -8.508  1.00 19.99           O  \\nATOM    801  N   THR D  27      -4.639  17.933  -6.342  1.00 10.07           N  \\nATOM    802  CA  THR D  27      -5.563  18.555  -7.264  1.00 15.24           C  \\nATOM    803  C   THR D  27      -5.254  18.120  -8.704  1.00 16.10           C  \\nATOM    804  O   THR D  27      -4.545  18.821  -9.404  1.00 18.07           O  \\nATOM    805  CB  THR D  27      -5.536  20.114  -7.048  1.00 16.40           C  \\nATOM    806  OG1 THR D  27      -4.178  20.585  -6.974  1.00 19.93           O  \\nATOM    807  CG2 THR D  27      -6.181  20.408  -5.709  1.00 16.74           C  \\nATOM    808  N   PRO D  28      -5.766  17.035  -9.149  1.00 11.98           N  \\nATOM    809  CA  PRO D  28      -5.571  16.537 -10.477  1.00 13.55           C  \\nATOM    810  C   PRO D  28      -6.105  17.452 -11.585  1.00 21.64           C  \\nATOM    811  O   PRO D  28      -5.471  17.453 -12.635  1.00 24.41           O  \\nATOM    812  CB  PRO D  28      -6.192  15.163 -10.479  1.00 21.77           C  \\nATOM    813  CG  PRO D  28      -6.813  14.874  -9.135  1.00 18.60           C  \\nATOM    814  CD  PRO D  28      -6.687  16.109  -8.343  1.00 11.54           C  \\nATOM    815  N   LYS D  29      -7.103  18.249 -11.346  1.00 18.85           N  \\nATOM    816  CA  LYS D  29      -7.647  19.148 -12.397  1.00 21.86           C  \\nATOM    817  C   LYS D  29      -6.782  20.313 -12.719  1.00 25.32           C  \\nATOM    818  O   LYS D  29      -6.944  20.988 -13.739  1.00 28.67           O  \\nATOM    819  CB  LYS D  29      -9.020  19.689 -11.971  1.00 37.73           C  \\nATOM    820  CG ALYS D  29     -10.118  18.678 -12.365  0.50 26.24           C  \\nATOM    821  CG BLYS D  29      -9.869  20.535 -12.933  0.50 51.60           C  \\nATOM    822  CD ALYS D  29     -11.261  18.659 -11.369  0.50 25.69           C  \\nATOM    823  CD BLYS D  29     -11.094  21.211 -12.318  0.50 26.73           C  \\nATOM    824  CE ALYS D  29     -12.315  17.615 -11.670  0.50 22.42           C  \\nATOM    825  CE BLYS D  29     -12.076  21.723 -13.345  0.50 41.26           C  \\nATOM    826  NZ ALYS D  29     -13.648  18.108 -11.182  0.50 39.07           N  \\nATOM    827  NZ BLYS D  29     -13.280  22.348 -12.731  0.50 38.89           N  \\nATOM    828  N   ALA D  30      -5.874  20.552 -11.809  1.00 17.54           N  \\nATOM    829  CA  ALA D  30      -5.022  21.709 -11.876  1.00 15.58           C  \\nATOM    830  C   ALA D  30      -3.942  21.534 -12.910  1.00 35.26           C  \\nATOM    831  O   ALA D  30      -3.347  20.451 -13.025  1.00 37.09           O  \\nATOM    832  CB  ALA D  30      -4.392  21.978 -10.527  1.00 32.86           C  \\nATOM    833  OXT ALA D  30      -3.822  22.597 -13.538  1.00 43.22           O  \\nTER     834      ALA D  30                                                      \\nHETATM  835 ZN    ZN B 101      -0.002  -0.004   7.891  0.33 10.40          ZN  \\nHETATM  836 ZN    ZN D 101       0.000   0.000  -8.039  0.33 11.00          ZN  \\nHETATM  837  O   HOH A 101      -9.600  22.800   0.340  1.00 86.30           O  \\nHETATM  838  O   HOH A 102     -11.312  20.399   0.771  1.00 76.27           O  \\nHETATM  839  O   HOH A 103     -12.203  23.328   2.558  0.50 53.25           O  \\nHETATM  840  O   HOH A 104     -10.414  20.888   3.177  1.00 46.17           O  \\nHETATM  841  O   HOH A 105      -9.814  18.858  10.172  0.50 36.40           O  \\nHETATM  842  O   HOH A 106      -5.367   9.301  10.390  1.00 29.29           O  \\nHETATM  843  O   HOH A 107      -7.077  17.022  11.114  1.00 24.63           O  \\nHETATM  844  O   HOH A 108      -1.805  14.585  13.563  1.00 36.72           O  \\nHETATM  845  O   HOH A 109      -7.400  20.900  13.300  1.00 82.49           O  \\nHETATM  846  O   HOH A 110     -10.436  21.800  14.703  1.00 82.89           O  \\nHETATM  847  O   HOH A 111      -8.811  19.800  14.800  1.00 84.33           O  \\nHETATM  848  O   HOH A 112      -1.885  13.663  16.148  1.00 61.43           O  \\nHETATM  849  O   HOH A 113     -11.760  21.100  16.700  0.50 40.14           O  \\nHETATM  850  O   HOH A 114      -8.270  20.622  17.537  0.50 29.13           O  \\nHETATM  851  O   HOH A 115      -7.800  16.300  17.668  1.00 75.77           O  \\nHETATM  852  O   HOH A 116      -8.894  18.338  17.943  1.00 51.71           O  \\nHETATM  853  O   HOH A 117     -11.769  21.611  10.650  0.50 56.71           O  \\nHETATM  854  O   HOH A 118      -8.631  22.492  11.840  0.50 43.99           O  \\nHETATM  855  O   HOH A 119      -7.614  21.771  17.400  0.50 39.65           O  \\nHETATM  856  O   HOH A 120      -4.837  15.671  19.009  1.00 33.67           O  \\nHETATM  857  O   HOH A 121      -5.300  12.855  19.148  1.00 51.00           O  \\nHETATM  858  O   HOH A 122      -9.409  23.116  19.633  1.00 66.30           O  \\nHETATM  859  O   HOH A 123      -5.648  12.110  21.371  1.00 36.62           O  \\nHETATM  860  O   HOH A 124      -6.414  11.027  24.091  1.00 36.40           O  \\nHETATM  861  O   HOH A 125     -16.809  20.696   1.745  1.00 71.89           O  \\nHETATM  862  O   HOH A 126     -20.717  13.454   2.846  1.00 29.91           O  \\nHETATM  863  O   HOH A 127     -10.096  20.400  -7.963  1.00 73.01           O  \\nHETATM  864  O   HOH A 128     -16.384  20.545   4.220  1.00 62.82           O  \\nHETATM  865  O   HOH A 129     -13.362  23.086   0.118  1.00 64.26           O  \\nHETATM  866  O   HOH A 130      -8.031  12.739  18.649  0.50 27.91           O  \\nHETATM  867  O   HOH A 131     -16.226  18.618  -0.171  1.00 38.87           O  \\nHETATM  868  O   HOH B 201      -5.500  23.300   3.200  1.00 76.30           O  \\nHETATM  869  O   HOH B 202      -8.655  24.051   2.000  1.00 56.40           O  \\nHETATM  870  O   HOH B 203      -0.463   3.085   2.990  1.00 35.04           O  \\nHETATM  871  O   HOH B 204      -9.400  22.600   4.900  0.50 52.19           O  \\nHETATM  872  O   HOH B 205      -7.624  16.101   5.527  1.00 25.99           O  \\nHETATM  873  O   HOH B 206      -9.100  20.600   6.100  0.50 60.00           O  \\nHETATM  874  O   HOH B 207      -8.768  19.973   5.675  0.50 51.16           O  \\nHETATM  875  O   HOH B 208      -7.991  22.590   6.582  0.50 57.20           O  \\nHETATM  876  O   HOH B 209     -12.272  21.574   6.100  1.00 55.48           O  \\nHETATM  877  O   HOH B 210      -9.484  23.456   7.067  0.50 41.28           O  \\nHETATM  878  O   HOH B 211      -3.193  10.008   8.356  1.00 20.33           O  \\nHETATM  879  O   HOH B 212      -9.443  20.240   8.418  0.50 46.63           O  \\nHETATM  880  O   HOH B 213      -0.074   1.535   9.452  1.00 25.97           O  \\nHETATM  881  O   HOH B 214       0.475  13.399   9.000  1.00 61.33           O  \\nHETATM  882  O   HOH B 215      -8.473  18.398   9.204  0.50 34.10           O  \\nHETATM  883  O   HOH B 216       4.832  22.586  10.697  0.50 41.13           O  \\nHETATM  884  O   HOH B 217      -9.503  22.800  10.020  0.50 40.98           O  \\nHETATM  885  O   HOH B 218       3.402  22.242  10.235  0.50 38.69           O  \\nHETATM  886  O   HOH B 219      -9.990  20.281  10.186  0.50 56.19           O  \\nHETATM  887  O   HOH B 220      -6.059  22.694  11.492  1.00 61.96           O  \\nHETATM  888  O   HOH B 221       2.400   0.700  11.139  1.00 63.37           O  \\nHETATM  889  O   HOH B 222       0.589  13.731  11.296  0.50 60.20           O  \\nHETATM  890  O   HOH B 223      -0.216  11.635  11.240  0.50 52.12           O  \\nHETATM  891  O   HOH B 224      -0.023  -0.033  11.206  0.33 21.05           O  \\nHETATM  892  O   HOH B 225      -5.435  20.514  11.630  1.00 51.70           O  \\nHETATM  893  O   HOH B 226      -1.821   8.684  12.044  1.00 36.04           O  \\nHETATM  894  O   HOH B 227       4.207  21.400  12.867  0.50 42.63           O  \\nHETATM  895  O   HOH B 228      -3.059  10.929  12.998  0.50 54.91           O  \\nHETATM  896  O   HOH B 229      -2.700  22.200  12.300  1.00 69.61           O  \\nHETATM  897  O   HOH B 230       1.000  16.700  13.200  1.00 45.00           O  \\nHETATM  898  O   HOH B 231       3.947  18.704  13.926  0.50 87.22           O  \\nHETATM  899  O   HOH B 232       0.063   7.371  13.477  0.50 44.43           O  \\nHETATM  900  O   HOH B 233       4.331  21.351  15.200  1.00 48.37           O  \\nHETATM  901  O   HOH B 234      -3.119  22.865  15.776  1.00 59.24           O  \\nHETATM  902  O   HOH B 235      -5.600  22.740  16.755  0.50 60.98           O  \\nHETATM  903  O   HOH B 236      -1.700  21.227  17.206  1.00 50.32           O  \\nHETATM  904  O   HOH B 237       4.677  20.377  17.882  1.00 49.25           O  \\nHETATM  905  O   HOH B 238      -1.199  22.838  19.138  1.00 36.10           O  \\nHETATM  906  O   HOH B 239       1.686  21.194  18.890  1.00 36.62           O  \\nHETATM  907  O   HOH B 240       3.848  22.050  20.436  1.00 73.68           O  \\nHETATM  908  O   HOH B 241      -2.438  17.554  21.488  1.00 27.54           O  \\nHETATM  909  O   HOH B 242       1.801  22.097  22.791  1.00 33.66           O  \\nHETATM  910  O   HOH B 243      -8.670  18.367  -8.735  1.00 27.41           O  \\nHETATM  911  O   HOH B 244       1.208   0.917  -0.239  1.00 44.11           O  \\nHETATM  912  O   HOH B 245       0.000   0.000   2.721  0.33 78.41           O  \\nHETATM  913  O   HOH B 246       0.397   4.490  11.763  0.50 23.04           O  \\nHETATM  914  O   HOH B 247       2.938   2.622  12.785  1.00 45.76           O  \\nHETATM  915  O   HOH B 248      -8.497  11.582  -5.037  1.00 65.08           O  \\nHETATM  916  O   HOH B 249     -16.177  12.121  -3.599  0.50 17.96           O  \\nHETATM  917  O   HOH C 101      13.443  19.181   0.629  1.00 37.18           O  \\nHETATM  918  O   HOH C 102      14.910  22.149   0.107  1.00 64.34           O  \\nHETATM  919  O   HOH C 103      15.228  18.988   2.773  1.00 47.95           O  \\nHETATM  920  O   HOH C 104      -2.652  21.385   6.534  1.00 64.51           O  \\nHETATM  921  O   HOH C 105      -2.600  23.100   8.100  0.50 37.93           O  \\nHETATM  922  O   HOH C 106       6.358  23.611  -0.018  1.00 94.84           O  \\nHETATM  923  O   HOH C 107       2.469  24.709  -2.126  1.00 29.22           O  \\nHETATM  924  O   HOH D 201       1.228   4.480   7.603  1.00 34.42           O  \\nHETATM  925  O   HOH D 202       5.794  10.580   7.551  0.50 18.82           O  \\nHETATM  926  O   HOH D 203       3.540   5.186   7.514  1.00 50.94           O  \\nHETATM  927  O   HOH D 204       3.162  12.350   8.534  1.00 60.02           O  \\nHETATM  928  O   HOH D 205       5.402   7.037   8.018  1.00 76.30           O  \\nHETATM  929  O   HOH D 206       4.499   3.274   9.801  1.00 21.36           O  \\nHETATM  930  O   HOH D 207       3.575  13.150  11.256  0.50 53.19           O  \\nHETATM  931  O   HOH D 208       6.250  11.753  12.889  1.00 50.22           O  \\nHETATM  932  O   HOH D 209       4.989  14.660  13.137  1.00 52.11           O  \\nHETATM  933  O   HOH D 210       7.840  13.593  13.240  1.00 36.77           O  \\nHETATM  934  O   HOH D 211      -1.841   2.626  14.104  1.00 47.47           O  \\nHETATM  935  O   HOH D 212      -0.629   5.214  14.504  1.00 52.63           O  \\nHETATM  936  O   HOH D 213      -3.496  10.462  15.323  1.00 41.57           O  \\nHETATM  937  O   HOH D 214      -3.814  12.169  17.184  1.00 40.38           O  \\nHETATM  938  O   HOH D 215      -0.459  10.100  16.800  1.00 96.03           O  \\nHETATM  939  O   HOH D 216      -2.265   8.874  17.970  1.00 48.81           O  \\nHETATM  940  O   HOH D 217       4.304  23.913   8.300  1.00 56.36           O  \\nHETATM  941  O   HOH D 218      -3.499   9.530  21.143  1.00 58.40           O  \\nHETATM  942  O   HOH D 219      -2.158  10.172  22.900  1.00 34.73           O  \\nHETATM  943  O   HOH D 220     -12.014  22.141  -9.704  1.00 66.63           O  \\nHETATM  944  O   HOH D 221      -1.763  20.077  -8.993  1.00 21.36           O  \\nHETATM  945  O   HOH D 222      -2.718  10.300  25.150  1.00 73.39           O  \\nHETATM  946  O   HOH D 223      -1.739  18.453  -6.295  1.00 12.21           O  \\nHETATM  947  O   HOH D 224      27.252   2.224   9.433  1.00 83.01           O  \\nHETATM  948  O   HOH D 225      14.487  19.051   4.963  0.50 47.26           O  \\nHETATM  949  O   HOH D 226      15.038  21.545   5.268  1.00 55.92           O  \\nHETATM  950  O   HOH D 227       8.897  22.662   0.833  0.50 72.59           O  \\nHETATM  951  O   HOH D 228       5.430   4.632   0.353  0.50 68.27           O  \\nHETATM  952  O   HOH D 229      10.779  23.846   1.400  0.50 44.66           O  \\nHETATM  953  O   HOH D 230       3.208   6.718   2.188  1.00 44.14           O  \\nHETATM  954  O   HOH D 231       1.948   1.415   2.000  1.00 83.31           O  \\nHETATM  955  O   HOH D 232       3.289   3.437   2.898  1.00 29.40           O  \\nHETATM  956  O   HOH D 233      10.311  22.834   3.121  0.50 53.93           O  \\nHETATM  957  O   HOH D 234      10.183  24.198   3.002  0.50 47.80           O  \\nHETATM  958  O   HOH D 235      23.650   0.900   3.000  1.00 86.30           O  \\nHETATM  959  O   HOH D 236       8.256  18.783   3.330  0.50 14.13           O  \\nHETATM  960  O   HOH D 237       5.786   4.688   3.296  1.00 79.24           O  \\nHETATM  961  O   HOH D 238      12.041  23.097   3.478  0.50 31.09           O  \\nHETATM  962  O   HOH D 239       9.781  13.999   4.347  1.00 38.00           O  \\nHETATM  963  O   HOH D 240      24.800   2.000   4.680  1.00 50.76           O  \\nHETATM  964  O   HOH D 241       1.700  14.430   4.490  1.00 60.70           O  \\nHETATM  965  O   HOH D 242       4.071   5.888   5.145  1.00 60.78           O  \\nHETATM  966  O   HOH D 243      12.744  23.426   5.604  0.50 40.09           O  \\nHETATM  967  O   HOH D 244      10.354  21.084   4.829  0.50 33.95           O  \\nHETATM  968  O   HOH D 245      12.369  21.475   5.162  0.50 61.18           O  \\nHETATM  969  O   HOH D 246      13.100  13.724   6.029  1.00 21.23           O  \\nHETATM  970  O   HOH D 247      25.212   3.031   6.890  1.00 49.58           O  \\nHETATM  971  O   HOH D 248      10.600  10.100   6.800  0.50 50.00           O  \\nHETATM  972  O   HOH D 249      11.191  23.508   6.621  0.50 37.26           O  \\nHETATM  973  O   HOH D 250      14.528  16.192   7.070  1.00 51.29           O  \\nHETATM  974  O   HOH D 251       1.492  23.231   6.892  0.50 27.00           O  \\nHETATM  975  O   HOH D 252      14.502  18.882   6.980  0.50 59.76           O  \\nHETATM  976  O   HOH D 253       0.172  22.959   7.609  0.50 29.31           O  \\nHETATM  977  O   HOH D 254      10.079  20.626   8.067  0.52 33.28           O  \\nHETATM  978  O   HOH D 255       8.276  22.353   6.634  0.50 71.16           O  \\nHETATM  979  O   HOH D 256      11.200  20.200   6.480  0.50 43.34           O  \\nHETATM  980  O   HOH D 257       9.467  21.709   6.550  0.50 60.52           O  \\nHETATM  981  O   HOH D 258      12.911  19.857   7.882  0.50 41.83           O  \\nHETATM  982  O   HOH D 259      24.906   0.400   8.400  1.00 59.59           O  \\nHETATM  983  O   HOH D 260      22.962   4.308   7.971  1.00 75.46           O  \\nHETATM  984  O   HOH D 261       6.986  23.533   8.608  0.50 42.86           O  \\nHETATM  985  O   HOH D 262       7.747  22.514   8.736  0.50 51.31           O  \\nHETATM  986  O   HOH D 263      11.350  23.300   9.170  1.00 50.61           O  \\nHETATM  987  O   HOH D 264      13.640  22.500   8.400  1.00 61.09           O  \\nHETATM  988  O   HOH D 265      21.200   5.800   9.000  1.00 46.66           O  \\nHETATM  989  O   HOH D 266      -0.559  20.337   9.227  0.50 31.04           O  \\nHETATM  990  O   HOH D 267       2.956   7.356   9.255  1.00 37.48           O  \\nHETATM  991  O   HOH D 268      11.907  19.705   9.066  0.50 35.59           O  \\nHETATM  992  O   HOH D 269      16.094  16.073   9.932  1.00 48.31           O  \\nHETATM  993  O   HOH D 270      13.482  17.840   9.054  1.00 66.41           O  \\nHETATM  994  O   HOH D 271       9.900  19.400  10.176  1.00 57.45           O  \\nHETATM  995  O   HOH D 272       8.300  21.900   9.770  0.50 62.82           O  \\nHETATM  996  O   HOH D 273       5.531  14.046   9.251  1.00 53.12           O  \\nHETATM  997  O   HOH D 274       0.500  21.500   9.739  0.50 40.73           O  \\nHETATM  998  O   HOH D 275      24.500   1.839  10.261  0.50 44.08           O  \\nHETATM  999  O   HOH D 276      14.403  20.407   9.881  0.25 62.98           O  \\nHETATM 1000  O   HOH D 277       5.099  10.014  10.625  1.00 39.93           O  \\nHETATM 1001  O   HOH D 278       0.786  10.457  10.424  0.50 27.11           O  \\nHETATM 1002  O   HOH D 279       7.099  20.641  10.117  0.50 45.52           O  \\nHETATM 1003  O   HOH D 280       1.327   5.583  11.052  0.50 56.24           O  \\nHETATM 1004  O   HOH D 281      10.114  17.143  11.048  1.00 38.47           O  \\nHETATM 1005  O   HOH D 282      13.122  21.921  10.724  0.50 29.51           O  \\nHETATM 1006  O   HOH D 283      24.000   4.400  11.200  1.00 63.61           O  \\nHETATM 1007  O   HOH D 284      20.200   6.301  11.100  1.00 62.49           O  \\nHETATM 1008  O   HOH D 285       1.431   8.001  10.900  1.00 61.63           O  \\nHETATM 1009  O   HOH D 286       3.725   4.937  12.004  1.00 60.07           O  \\nHETATM 1010  O   HOH D 287       9.246  23.287  11.503  1.00 45.58           O  \\nHETATM 1011  O   HOH D 288      13.713  17.818  11.963  1.00 66.30           O  \\nHETATM 1012  O   HOH D 289      16.125  12.139  12.240  1.00 50.03           O  \\nHETATM 1013  O   HOH D 290      12.904  20.420  11.491  0.50 30.82           O  \\nHETATM 1014  O   HOH D 291      22.345   2.177  11.300  1.00 53.69           O  \\nHETATM 1015  O   HOH D 292       3.200   8.800  12.200  1.00 58.95           O  \\nHETATM 1016  O   HOH D 293      12.623  23.254  12.208  0.50 36.01           O  \\nHETATM 1017  O   HOH D 294       6.083  21.437  12.952  0.50 45.78           O  \\nHETATM 1018  O   HOH D 295      15.629  16.552  13.704  1.00 54.02           O  \\nHETATM 1019  O   HOH D 296       1.400  10.000  13.205  1.00 62.38           O  \\nHETATM 1020  O   HOH D 297      10.681  15.953  13.460  1.00 37.62           O  \\nHETATM 1021  O   HOH D 298       3.526  17.510  13.393  0.50 58.59           O  \\nHETATM 1022  O   HOH D 299      17.589  14.886  12.994  0.50 46.92           O  \\nHETATM 1023  O   HOH D 300       8.436  16.103  13.124  0.50 66.40           O  \\nHETATM 1024  O   HOH D 301      20.767   2.758  14.127  0.50 49.50           O  \\nHETATM 1025  O   HOH D 302      19.461   3.175  13.236  0.50 43.30           O  \\nHETATM 1026  O   HOH D 303      10.600  21.200  12.800  1.00 65.41           O  \\nHETATM 1027  O   HOH D 304      17.091  12.037  14.424  1.00 66.30           O  \\nHETATM 1028  O   HOH D 305      26.230   1.151  12.982  0.50 42.97           O  \\nHETATM 1029  O   HOH D 306       8.200  19.886  13.338  1.00 59.62           O  \\nHETATM 1030  O   HOH D 307       3.000   8.250  14.500  1.00 39.38           O  \\nHETATM 1031  O   HOH D 308      23.669   1.157  13.223  1.00 49.56           O  \\nHETATM 1032  O   HOH D 309       6.706  17.656  14.200  1.00 62.44           O  \\nHETATM 1033  O   HOH D 310      21.660   8.100  14.400  1.00 78.85           O  \\nHETATM 1034  O   HOH D 311      16.002  14.165  14.204  0.50 36.21           O  \\nHETATM 1035  O   HOH D 312      18.624   9.971  14.472  1.00 68.09           O  \\nHETATM 1036  O   HOH D 313      12.345  18.363  14.320  1.00 61.52           O  \\nHETATM 1037  O   HOH D 314      15.375  13.331  15.234  0.50 38.87           O  \\nHETATM 1038  O   HOH D 315       8.900  16.074  15.111  0.50 56.02           O  \\nHETATM 1039  O   HOH D 316       9.800  20.500  15.000  0.50 37.74           O  \\nHETATM 1040  O   HOH D 317      20.471   6.314  15.326  0.50 42.47           O  \\nHETATM 1041  O   HOH D 318      15.956  10.347  15.577  1.00 73.09           O  \\nHETATM 1042  O   HOH D 319       4.166  16.122  15.117  1.00 64.59           O  \\nHETATM 1043  O   HOH D 320      22.179   1.588  16.322  1.00 60.14           O  \\nHETATM 1044  O   HOH D 321       7.227  21.575  16.034  1.00 79.51           O  \\nHETATM 1045  O   HOH D 322      24.987   3.024  15.000  0.50 77.50           O  \\nHETATM 1046  O   HOH D 323       0.000   0.000  15.940  0.33 48.19           O  \\nHETATM 1047  O   HOH D 324       2.058   1.124  15.886  0.50 57.48           O  \\nHETATM 1048  O   HOH D 325      19.200   4.000  16.256  1.00 52.18           O  \\nHETATM 1049  O   HOH D 326      14.044  21.773  15.730  0.50 35.08           O  \\nHETATM 1050  O   HOH D 327       1.900   3.400  16.729  1.00 42.63           O  \\nHETATM 1051  O   HOH D 328      14.400  17.800  15.886  0.50 52.96           O  \\nHETATM 1052  O   HOH D 329      13.700  15.600  15.000  1.00 65.01           O  \\nHETATM 1053  O   HOH D 330       2.100   6.200  15.800  1.00 73.63           O  \\nHETATM 1054  O   HOH D 331      13.105  18.675  16.692  0.50 57.75           O  \\nHETATM 1055  O   HOH D 332       5.384   1.947  16.328  1.00 52.83           O  \\nHETATM 1056  O   HOH D 333      19.387   1.649  16.538  1.00 38.74           O  \\nHETATM 1057  O   HOH D 334       6.570  18.429  16.505  1.00 50.65           O  \\nHETATM 1058  O   HOH D 335      18.269   7.536  16.565  1.00 60.44           O  \\nHETATM 1059  O   HOH D 336      16.961  12.601  17.169  1.00 81.43           O  \\nHETATM 1060  O   HOH D 337      19.266  10.576  17.790  1.00 63.02           O  \\nHETATM 1061  O   HOH D 338      -2.564   5.638  17.137  1.00 48.33           O  \\nHETATM 1062  O   HOH D 339      21.336   8.539  16.849  1.00 46.39           O  \\nHETATM 1063  O   HOH D 340      11.361  23.312  16.935  1.00 56.56           O  \\nHETATM 1064  O   HOH D 341      11.482  20.115  16.180  0.50 36.42           O  \\nHETATM 1065  O   HOH D 342      10.431  19.608  16.704  0.50 61.59           O  \\nHETATM 1066  O   HOH D 343      22.701   4.882  16.533  0.50 59.04           O  \\nHETATM 1067  O   HOH D 344      -0.559   7.125  16.712  1.00 55.39           O  \\nHETATM 1068  O   HOH D 345      22.349   3.895  16.628  0.50 44.55           O  \\nHETATM 1069  O   HOH D 346      24.254   2.529  17.221  0.50 64.98           O  \\nHETATM 1070  O   HOH D 347      17.313  15.074  17.581  1.00 53.72           O  \\nHETATM 1071  O   HOH D 348      16.509   9.562  17.812  1.00 54.16           O  \\nHETATM 1072  O   HOH D 349      17.535   3.747  17.792  1.00 51.88           O  \\nHETATM 1073  O   HOH D 350       0.500   3.000  18.600  0.50 35.00           O  \\nHETATM 1074  O   HOH D 351       6.000   4.100  17.800  1.00 50.00           O  \\nHETATM 1075  O   HOH D 352       2.700   3.900  19.000  0.50 40.00           O  \\nHETATM 1076  O   HOH D 353       2.992   5.257  18.400  0.50 40.00           O  \\nHETATM 1077  O   HOH D 354      22.320   6.616  18.376  1.00 76.30           O  \\nHETATM 1078  O   HOH D 355      14.300  15.147  17.400  0.50 41.41           O  \\nHETATM 1079  O   HOH D 356      12.379  14.000  18.485  1.00 69.21           O  \\nHETATM 1080  O   HOH D 357      26.585   1.034  18.302  1.00 72.39           O  \\nHETATM 1081  O   HOH D 358      24.664   3.165  18.640  0.50 55.98           O  \\nHETATM 1082  O   HOH D 359      16.000   6.400  18.500  1.00 75.03           O  \\nHETATM 1083  O   HOH D 360       7.587  19.898  18.900  1.00 74.10           O  \\nHETATM 1084  O   HOH D 361      23.636   0.400  18.433  0.50 38.58           O  \\nHETATM 1085  O   HOH D 362       7.200   0.000  19.364  1.00 80.00           O  \\nHETATM 1086  O   HOH D 363      12.325  20.187  18.584  1.00 76.30           O  \\nHETATM 1087  O   HOH D 364       8.868  21.918  18.717  1.00 88.44           O  \\nHETATM 1088  O   HOH D 365       4.000   0.889  19.161  1.00 56.19           O  \\nHETATM 1089  O   HOH D 366      11.500  16.200  18.500  1.00 54.59           O  \\nHETATM 1090  O   HOH D 367      15.033  14.600  19.135  0.50 40.58           O  \\nHETATM 1091  O   HOH D 368      16.700  11.300  19.500  0.50 53.18           O  \\nHETATM 1092  O   HOH D 369      17.595   8.307  19.856  1.00 43.95           O  \\nHETATM 1093  O   HOH D 370       8.460  17.052  19.897  1.00 58.87           O  \\nHETATM 1094  O   HOH D 371      11.600  18.400  20.000  1.00 60.00           O  \\nHETATM 1095  O   HOH D 372       7.200   3.000  19.668  1.00 79.22           O  \\nHETATM 1096  O   HOH D 373      21.000   4.000  19.956  1.00 59.41           O  \\nHETATM 1097  O   HOH D 374      -0.800   4.800  19.700  1.00 50.00           O  \\nHETATM 1098  O   HOH D 375      13.592   4.017  20.054  1.00 43.20           O  \\nHETATM 1099  O   HOH D 376      -0.400   2.600  20.000  0.50 40.00           O  \\nHETATM 1100  O   HOH D 377      19.480   6.217  19.797  1.00 62.99           O  \\nHETATM 1101  O   HOH D 378      20.876   1.574  20.116  0.50 41.62           O  \\nHETATM 1102  O   HOH D 379      18.870   3.203  20.295  1.00 65.25           O  \\nHETATM 1103  O   HOH D 380      22.043   8.448  20.602  0.50 38.02           O  \\nHETATM 1104  O   HOH D 381      15.420   7.740  20.813  1.00 61.77           O  \\nHETATM 1105  O   HOH D 382      14.400  19.900  20.800  1.00 46.45           O  \\nHETATM 1106  O   HOH D 383       8.900   4.300  20.900  1.00 66.62           O  \\nHETATM 1107  O   HOH D 384      12.900   7.700  20.600  1.00 50.81           O  \\nHETATM 1108  O   HOH D 385      23.373   3.907  20.784  0.50 37.81           O  \\nHETATM 1109  O   HOH D 386      22.600   5.200  21.000  0.50 50.00           O  \\nHETATM 1110  O   HOH D 387      11.000  20.300  21.100  0.50 50.00           O  \\nHETATM 1111  O   HOH D 388      15.900  13.400  20.500  0.50 37.02           O  \\nHETATM 1112  O   HOH D 389      15.000  15.900  20.409  0.50 51.72           O  \\nHETATM 1113  O   HOH D 390      11.455   5.672  21.400  0.50 45.30           O  \\nHETATM 1114  O   HOH D 391      19.300  11.200  21.251  1.00 93.71           O  \\nHETATM 1115  O   HOH D 392      12.867  15.311  21.383  1.00104.59           O  \\nHETATM 1116  O   HOH D 393      16.200  18.500  20.516  0.50 57.42           O  \\nHETATM 1117  O   HOH D 394      17.900  14.200  21.400  0.50 54.11           O  \\nHETATM 1118  O   HOH D 395       2.426   2.789  21.667  0.50 37.21           O  \\nHETATM 1119  O   HOH D 396       5.591   3.514  22.644  1.00 66.58           O  \\nHETATM 1120  O   HOH D 397      21.490   9.784  21.789  0.50 37.24           O  \\nHETATM 1121  O   HOH D 398      22.459   7.099  21.906  0.50 54.11           O  \\nHETATM 1122  O   HOH D 399      23.362   5.246  22.116  0.50 53.14           O  \\nHETATM 1123  O   HOH D 400      10.026   5.995  22.154  0.50 36.85           O  \\nHETATM 1124  O   HOH D 401      16.230  16.756  22.900  1.00 51.60           O  \\nHETATM 1125  O   HOH D 402       7.950   1.500  22.400  1.00 74.32           O  \\nHETATM 1126  O   HOH D 403       0.000   0.000  22.203  0.33 26.53           O  \\nHETATM 1127  O   HOH D 404       6.000   0.331  22.809  1.00 81.99           O  \\nHETATM 1128  O   HOH D 405       2.400   1.900  22.950  0.50 53.75           O  \\nHETATM 1129  O   HOH D 406      16.826   9.360  22.915  1.00 64.50           O  \\nHETATM 1130  O   HOH D 407      10.028   0.600  22.667  1.00 50.88           O  \\nHETATM 1131  O   HOH D 408       8.566  22.210  23.275  1.00 52.52           O  \\nHETATM 1132  O   HOH D 409      19.041   7.317  22.965  1.00 45.49           O  \\nHETATM 1133  O   HOH D 410      13.154  17.896  23.121  0.50 58.99           O  \\nHETATM 1134  O   HOH D 411      25.120   2.792  23.670  1.00 49.60           O  \\nHETATM 1135  O   HOH D 412       0.848   2.630  23.300  0.50 51.96           O  \\nHETATM 1136  O   HOH D 413      12.799   4.914  23.300  1.00 45.19           O  \\nHETATM 1137  O   HOH D 414      14.100   1.511  23.000  1.00 54.91           O  \\nHETATM 1138  O   HOH D 415      18.000   5.200  23.400  0.50 40.00           O  \\nHETATM 1139  O   HOH D 416      20.460   5.246  23.195  0.50 42.74           O  \\nHETATM 1140  O   HOH D 417      18.500  13.238  23.926  0.50 60.34           O  \\nHETATM 1141  O   HOH D 418       9.469   4.007  23.263  0.50 65.39           O  \\nHETATM 1142  O   HOH D 419       3.330   4.805  23.757  1.00 24.03           O  \\nHETATM 1143  O   HOH D 420      -1.798   3.543  24.064  1.00 80.64           O  \\nHETATM 1144  O   HOH D 421       1.468   0.225  24.184  1.00 35.46           O  \\nHETATM 1145  O   HOH D 422      13.052  19.509  23.128  0.50 62.84           O  \\nHETATM 1146  O   HOH D 423       9.900  22.890  25.259  1.00 65.26           O  \\nHETATM 1147  O   HOH D 424      16.495  17.126  26.000  1.00 40.00           O  \\nHETATM 1148  O   HOH D 425      21.700   5.400  24.514  0.50 50.22           O  \\nHETATM 1149  O   HOH D 426       6.642  20.679  24.898  1.00 58.56           O  \\nHETATM 1150  O   HOH D 427      20.692   8.311  24.457  0.50 48.03           O  \\nHETATM 1151  O   HOH D 428       7.700   4.223  25.055  1.00 33.74           O  \\nHETATM 1152  O   HOH D 429      19.581  11.070  25.093  1.00 65.09           O  \\nHETATM 1153  O   HOH D 430      22.450   8.700  25.159  0.50 48.15           O  \\nHETATM 1154  O   HOH D 431      12.210  21.699  25.848  0.50 44.27           O  \\nHETATM 1155  O   HOH D 432      11.500  21.600  24.157  0.50 70.69           O  \\nHETATM 1156  O   HOH D 433      25.803   0.095  26.600  1.00 56.59           O  \\nHETATM 1157  O   HOH D 434      23.392   1.295  25.732  1.00 48.54           O  \\nHETATM 1158  O   HOH D 435      12.194   3.114  25.000  1.00 76.30           O  \\nHETATM 1159  O   HOH D 436       4.708   1.340  26.039  1.00 30.59           O  \\nHETATM 1160  O   HOH D 437      14.388  19.737  25.636  1.00 64.76           O  \\nHETATM 1161  O   HOH D 438       5.419   3.667  26.434  1.00 31.12           O  \\nHETATM 1162  O   HOH D 439      23.512   4.500  26.629  1.00 66.30           O  \\nHETATM 1163  O   HOH D 440      16.389  14.064  27.171  1.00 74.53           O  \\nHETATM 1164  O   HOH D 441      18.700  14.000  26.532  1.00 79.38           O  \\nHETATM 1165  O   HOH D 442      10.926  10.684  -7.060  1.00 44.80           O  \\nHETATM 1166  O   HOH D 443      12.500  21.800  27.254  0.50 47.63           O  \\nHETATM 1167  O   HOH D 444      27.000   1.000  28.300  1.00 70.92           O  \\nHETATM 1168  O   HOH D 445      20.879   8.934  27.397  0.50 49.09           O  \\nHETATM 1169  O   HOH D 446       5.301   3.962  28.700  0.60 31.54           O  \\nHETATM 1170  O   HOH D 447      13.794  13.317  28.059  1.00 74.41           O  \\nHETATM 1171  O   HOH D 448       6.578   4.370  28.779  0.40 28.91           O  \\nHETATM 1172  O   HOH D 449      19.254  14.024  28.832  1.00 55.11           O  \\nHETATM 1173  O   HOH D 450      14.080   9.849  -4.521  1.00 27.58           O  \\nHETATM 1174  O   HOH D 451      23.137   6.636  29.293  0.50 55.90           O  \\nHETATM 1175  O   HOH D 452      10.475  22.332  30.034  1.00 41.35           O  \\nHETATM 1176  O   HOH D 453      14.200  22.300  30.301  1.00 67.11           O  \\nHETATM 1177  O   HOH D 454       4.564   5.544  30.131  1.00 37.24           O  \\nHETATM 1178  O   HOH D 455       0.000   0.000  30.941  0.33 24.09           O  \\nHETATM 1179  O   HOH D 456       3.559   3.107  30.844  1.00 34.61           O  \\nHETATM 1180  O   HOH D 457      -1.317   2.461  30.833  1.00 27.74           O  \\nHETATM 1181  O   HOH D 458       6.900   3.468  31.640  1.00 38.60           O  \\nHETATM 1182  O   HOH D 459       1.484   2.379  31.639  1.00 81.34           O  \\nHETATM 1183  O   HOH D 460      12.200  21.561  31.604  1.00 55.36           O  \\nHETATM 1184  O   HOH D 461     -11.275  25.048 -12.333  1.00 77.73           O  \\nHETATM 1185  O   HOH D 462      11.090  22.753  33.200  1.00 96.30           O  \\nHETATM 1186  O   HOH D 463       3.296   3.651  33.200  1.00 86.30           O  \\nCONECT   43   76                                                                \\nCONECT   49  227                                                                \\nCONECT   76   43                                                                \\nCONECT  154  319                                                                \\nCONECT  227   49                                                                \\nCONECT  247  835                                                                \\nCONECT  319  154                                                                \\nCONECT  464  497                                                                \\nCONECT  470  644                                                                \\nCONECT  497  464                                                                \\nCONECT  575  734                                                                \\nCONECT  644  470                                                                \\nCONECT  664  836                                                                \\nCONECT  734  575                                                                \\nCONECT  835  247  880                                                           \\nCONECT  836  664                                                                \\nCONECT  880  835                                                                \\nMASTER      477    0    2    6    2    0   14    9 1158    4   17   10          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewer_17539308553244367.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewer_17539308553244367.zoomTo();\n",
       "viewer_17539308553244367.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "\n",
    "from Bio.PDB import PDBList\n",
    "pdbl = PDBList()\n",
    "pdbl.retrieve_pdb_file('4INS', pdir='.', file_format='pdb')\n",
    "import py3Dmol\n",
    "\n",
    "with open('pdb4INS.ent', 'r') as file:\n",
    "    pdb_data = file.read()\n",
    "\n",
    "view = py3Dmol.view(width=1000, height=300)\n",
    "view.addModel(pdb_data, 'pdb')\n",
    "view.setStyle({'cartoon': {'color': 'spectrum'}})\n",
    "view.zoomTo()\n",
    "view.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "344c6897-fcf9-4022-a706-3df7e8e95fc6",
   "metadata": {},
   "source": [
    "#### Functions of Insulin\n",
    "    \n",
    "|Molecular Function|\n",
    "|----------|\n",
    "|DNA binding|\n",
    "|DNA binding transcription factor activity|\n",
    "|RNA binding|\n",
    "|Catalytic activity|\n",
    "|Signaling receptor binding|\n",
    "|Cytoskeleton protein binding|\n",
    "|Transcription factor binding|\n",
    "|Carbohydrate binding|\n",
    "|Small molecule binding|\n",
    "|Enzyme regulator activity|\n",
    "\n",
    "\n",
    "|Biological Process|\n",
    "|-------------|\n",
    "|Immune system process|\n",
    "|Lipd metabolic process|\n",
    "|catabolic process|\n",
    "|DNA metabolic process|\n",
    "|Cell population poliferation|\n",
    "|Signaling|\n",
    "|Developmental process|\n",
    "|Programmed cell death|\n",
    "|RNA metabolic process|\n",
    "|Protein metabolic process|\n",
    "|Homeostatic process|\n",
    "|Nervous system process|"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "944f10c1-130b-407d-8b50-9115dcc1c34c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('DNA binding', 'DNA binding transcription factor activity', 'RNA binding', 'Catalytic activity', 'Signaling receptor binding', 'Cytoskeleton protein binding', 'Transcription factor binding', 'Carbohydrate binding', 'Small molecule binding', 'Enzyme regulator activity')\n"
     ]
    }
   ],
   "source": [
    "# let's create a variable ~Molecular function~\n",
    "Molecular_function=(\"DNA binding\",\"DNA binding transcription factor activity\",\"RNA binding\",\"Catalytic activity\",\"Signaling receptor binding\",\"Cytoskeleton protein binding\",\"Transcription factor binding\",\"Carbohydrate binding\",\"Small molecule binding\",\"Enzyme regulator activity\")\n",
    "print(Molecular_function)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "96d195db-b182-4528-ac45-2e60418775bc",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Catalytic activity'"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Molecular_function[3]\n",
    "#to fetch out the a string"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "78e8fc04-7f12-4b83-a047-e0945bd8f0b9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "('Catalytic activity',\n",
       " 'Signaling receptor binding',\n",
       " 'Cytoskeleton protein binding',\n",
       " 'Transcription factor binding',\n",
       " 'Carbohydrate binding')"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#string slicing--\n",
    "Molecular_function[3:8]\n",
    "# 3:8 print string 4,5,6,7,8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "ebaf8d02-70be-4031-aa1f-7a859fb1c4f3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'carbohydrate binding'"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#lower() method converts uppercase string to lowercase\n",
    "'CARBOHYDRATE BINDING'.lower()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "9b0475ae-380f-4818-8167-62bc741c3acf",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "e5701b76-e362-409c-b8d3-50426ea4c987",
   "metadata": {},
   "outputs": [],
   "source": [
    "#lets perform some regular expression with the strings./ first import re, which was done above\n",
    "#I am going to replace 'Catalytic'  with 'metal ion binding'\n",
    "A_Molecular_function='Catalytic activity'\n",
    "pattern=r'Catalytic'\n",
    "replace='metal ion binding'\n",
    "new_molecular_function=re.sub(pattern,replace,A_Molecular_function)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "d6896c03-45a6-42d7-82f6-3f81f9c2d053",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "metal ion binding activity\n"
     ]
    }
   ],
   "source": [
    "print(new_molecular_function)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "789f5cd7-474b-477f-9bfb-d099781935c1",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Enzyme regulator activity'"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#In search, the index -1 denotes the final position\n",
    "Molecular_function[-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "e0e8af12-ca7f-4a79-add4-78511b68bf61",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('DNA binding', 'DNA binding transcription factor activity', 'RNA binding', 'Catalytic activity', 'Signaling receptor binding', 'Cytoskeleton protein binding', 'Transcription factor binding', 'Carbohydrate binding', 'Small molecule binding', 'Enzyme regulator activity')\n"
     ]
    }
   ],
   "source": [
    "print(Molecular_function)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "51025b3e-a7c0-4300-8fe0-1ec004ccb6a6",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Catalytic activityCatalytic activityCatalytic activity'"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#print a string three times\n",
    "\"Catalytic activity\"*3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "a3aa56a3-0082-4e01-8f17-76a1e15c2401",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "41\n"
     ]
    }
   ],
   "source": [
    "#find the length of string index 1\n",
    "print(len(\"DNA binding transcription factor activity\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "5d352281-76de-4d3a-9150-04e43412045f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "('DNA binding',\n",
       " 'RNA binding',\n",
       " 'Signaling receptor binding',\n",
       " 'Transcription factor binding',\n",
       " 'Small molecule binding')"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Molecular_function[0:10:2]\n",
    "# stride activity. Here means print every 2nd string from 0 to 10"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5c41b058-ea2a-4972-b270-5650b89f3f16",
   "metadata": {},
   "source": [
    "### Disease and variants\n",
    "#### Hyperproinsulinemia (HPRI)\n",
    "----------\n",
    "*Some variants causing HPRI affect the gene INS.* Let's take a look at the variants\n",
    "|VariantID|Position|Change|Desription|\n",
    "|-------|------|-----|-------|\n",
    "|VAR_003971|34|H>D|A probably deleterious missense mutation--Amino acid change from histidine to aspartic acid|\n",
    "|VAR_003972|48|F>S|Amino acid change from phenylalanine to serine at position 48|\n",
    "|VAR_003974|89|R>H|Missense mutation, probably deleterious, from arginine to histidine, position 89|\n",
    "|VAR_003975|89|R>L|Missense mutation, amino acid change arginine to leucine| "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "bbb84b2a-4b08-4719-b5b7-8606683ddab1",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Create an empty dictionary with the disease, Hyperproinsulinemia(HPRI)\n",
    "HPRI={}\n",
    "#Now, insert variant1 in HPRI\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "2910f1f2-420d-4ae1-aa66-593351494485",
   "metadata": {},
   "outputs": [],
   "source": [
    "#create dictionaries with the variants causing diseases using each disease as a dictionary\n",
    "# let's turn each variant into a variable\n",
    "Variant1_name_HPRI=\"VAR_003971\"\n",
    "Variant1_position_HPRI=34\n",
    "Variant1_change_HPRI=\"H>D\"\n",
    "Variant1_description_HPRI=\"A probably deleterious missense mutation--Amino acid change from histidine to aspartic acid\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "b9ae3304-a706-43a8-98cc-f057b64a9388",
   "metadata": {},
   "outputs": [],
   "source": [
    "HPRI[\"Variant 1 HPRI\"]=Variant1_name_HPRI\n",
    "HPRI[\"Variant 1 position HPRI\"]=Variant1_position_HPRI\n",
    "HPRI[\"Variant 1 position change HPRI\"]=Variant1_change_HPRI\n",
    "HPRI[\"Variant 1 position change description HPRI\"]=Variant1_description_HPRI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "6f8e1d85-cf5d-4d34-aec7-5aa1c732b9db",
   "metadata": {},
   "outputs": [],
   "source": [
    "#do the same with other variants of HPRI\n",
    "#turn the second variant role into variable\n",
    "Variant2_name_HPRI=\"VAR_003972\"\n",
    "Variant2_position_HPRI=48\n",
    "Variant2_change_HPRI=\"F>S\"\n",
    "Variant2_description_HPRI=\"Amino acid change from phenylalanine to serine at position 48\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "6f4ec74e-18b5-4bb3-b021-1348d31d971c",
   "metadata": {},
   "outputs": [],
   "source": [
    "#turn the variant 3 role into variable\n",
    "Variant3_name_HPRI=\"VAR_003974\"\n",
    "Variant3_position_HPRI=89\n",
    "Variant3_change_HPRI=\"R>H\"\n",
    "Variant3_description_HPRI=\"Missense mutation, probably deleterious, from arginine to histidine, position 89\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "788a7d2e-c1e4-4492-8a86-e5d695be3324",
   "metadata": {},
   "outputs": [],
   "source": [
    "Variant4_name_HPRI=\"VAR_003975\"\n",
    "Variant4_position_HPRI=89\n",
    "Variant4_change_HPRI=\"R>L\"\n",
    "Variant4_description_HPRI=\"Missense mutation, amino acid change arginine to leucine\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "6d6528ac-e827-4fe6-b140-52d5169c54c1",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Now, insert the new variants into the dictionary HPRI\n",
    "HPRI[\"Variant 2 HPRI\"]=Variant2_name_HPRI\n",
    "HPRI[\"Variant 2 position HPRI\"]=Variant2_position_HPRI\n",
    "HPRI[\"Variant 2 position change HPRI\"]=Variant2_change_HPRI\n",
    "HPRI[\"Variant 2 position change description HPRI\"]=Variant2_description_HPRI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "0c27cf82-ab6c-45c9-854c-be4e09e5cffc",
   "metadata": {},
   "outputs": [],
   "source": [
    "HPRI[\"Variant 3 HPRI\"]=Variant3_name_HPRI\n",
    "HPRI[\"Variant 3 position HPRI\"]=Variant3_position_HPRI\n",
    "HPRI[\"Variant 3 position change HPRI\"]=Variant3_change_HPRI\n",
    "HPRI[\"Variant 3 position change description HPRI\"]=Variant3_description_HPRI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "fab0adc5-f3a1-4cab-9749-09e00a8cb323",
   "metadata": {},
   "outputs": [],
   "source": [
    "HPRI[\"Variant 4 HPRI\"]=Variant4_name_HPRI\n",
    "HPRI[\"Variant 4 position HPRI\"]=Variant4_position_HPRI\n",
    "HPRI[\"Variant 4 position change HPRI\"]=Variant4_change_HPRI\n",
    "HPRI[\"Variant 4 position change description HPRI\"]=Variant4_description_HPRI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "34144834-90df-4c07-ae19-9fa2e7918560",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Variant 1 HPRI': 'VAR_003971', 'Variant 1 position HPRI': 34, 'Variant 1 position change HPRI': 'H>D', 'Variant 1 position change description HPRI': 'A probably deleterious missense mutation--Amino acid change from histidine to aspartic acid', 'Variant 2 HPRI': 'VAR_003972', 'Variant 2 position HPRI': 48, 'Variant 2 position change HPRI': 'F>S', 'Variant 2 position change description HPRI': 'Amino acid change from phenylalanine to serine at position 48', 'Variant 3 HPRI': 'VAR_003974', 'Variant 3 position HPRI': 89, 'Variant 3 position change HPRI': 'R>H', 'Variant 3 position change description HPRI': 'Missense mutation, probably deleterious, from arginine to histidine, position 89', 'Variant 4 HPRI': 'VAR_003975', 'Variant 4 position HPRI': 89, 'Variant 4 position change HPRI': 'R>L', 'Variant 4 position change description HPRI': 'Missense mutation, amino acid change arginine to leucine'}\n"
     ]
    }
   ],
   "source": [
    "print(HPRI)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "534b31f6-f582-424a-9e8d-aac90ea20bdf",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['Variant 1 HPRI', 'Variant 1 position HPRI', 'Variant 1 position change HPRI', 'Variant 1 position change description HPRI', 'Variant 2 HPRI', 'Variant 2 position HPRI', 'Variant 2 position change HPRI', 'Variant 2 position change description HPRI', 'Variant 3 HPRI', 'Variant 3 position HPRI', 'Variant 3 position change HPRI', 'Variant 3 position change description HPRI', 'Variant 4 HPRI', 'Variant 4 position HPRI', 'Variant 4 position change HPRI', 'Variant 4 position change description HPRI'])"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Fetch out the keys and values in the dictionary, HPRI using the keys() and values() functions\n",
    "HPRI.keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "18847495-d458-4645-b87a-4bcbb45cd6d7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_values(['VAR_003971', 34, 'H>D', 'A probably deleterious missense mutation--Amino acid change from histidine to aspartic acid', 'VAR_003972', 48, 'F>S', 'Amino acid change from phenylalanine to serine at position 48', 'VAR_003974', 89, 'R>H', 'Missense mutation, probably deleterious, from arginine to histidine, position 89', 'VAR_003975', 89, 'R>L', 'Missense mutation, amino acid change arginine to leucine'])"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "HPRI.values()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "7c240ddb-7828-4316-992c-0b9a9dce6d4c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'H>D'"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# search out some keys info from the dictionary\n",
    "HPRI['Variant 1 position change HPRI']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "7cd3e390-a913-4062-9289-a687421d1fd0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"Variant 2 position change description HPRI\" in HPRI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "260deb3d-9a7f-4109-8240-9350a3cf0156",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "89"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# What chromosomal position would you find the fourth variant causing HPRI\n",
    "HPRI[\"Variant 4 position HPRI\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "30d66e5f-d5f1-46c0-8027-5eb75b6019ce",
   "metadata": {},
   "source": [
    "#### Diabetes mellitus, permanent neonatal, 4(PNDM4)\n",
    "----------------\n",
    "*Some variants affect the INS gene resulting into this disease. It is characterized by the onset of persistent hyperglycemia within the first six months of life. Some clinical symtoms with PNDM4 include hyperglycemia, glycosuria, osmotic polyuria, severe dehydration, and growth retardation.*\n",
    "|VariantID|Position|Change|Desription|\n",
    "|-------|------|-----|-------|\n",
    "|VAR_063723|24|A>D|Misssense mutation, alanine to aspartic acid|\n",
    "|VAR_063724|29|H>D|Missense mutation, Histidine to aspartic acid|\n",
    "|VAR_063725|32|G>R|Missense mutation, glycine to arginine|\n",
    "|VAR_063726|32|G>S|glycine to serine|\n",
    "|VAR_063727|35|L>P|leucine to proline leading to probably deleterious missense mutation|\n",
    "#We still have more variants that affect INS leading to PNDM4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1ade4501-ed19-41eb-98f6-ec2fc867cedb",
   "metadata": {},
   "outputs": [],
   "source": [
    "# create an empty dictionary named PNDM4 (Diabetes mellitus, permanent neonatal"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "36d3276d-b557-4d06-be12-7cdc5f8895d2",
   "metadata": {},
   "outputs": [],
   "source": [
    "PNDM4={}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "041052b7-20e2-446b-ab82-208b7743a026",
   "metadata": {},
   "outputs": [],
   "source": [
    "#turn the five variants causing PNDM4 to variable\n",
    "Variant1_name_PNDM4=\"VAR_063723\"\n",
    "Variant1_position_PNDM4=24\n",
    "Variant1_change_PNDM4=\"A>D\"\n",
    "Variant1_description_PNDM4=\"Missense mutation, alanine to aspartic acid\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "4b5f2f86-44a3-45f2-99a4-31621d9b7d9e",
   "metadata": {},
   "outputs": [],
   "source": [
    "Variant2_name_PNDM4=\"VAR_063724\"\n",
    "Variant2_position_PNDM4=29\n",
    "Variant2_change_PNDM4=\"H>D\"\n",
    "Variant2_description_PNDM4=\"Missense mutation, Histidine to aspartic acid\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "4c0ff5b0-3ac1-4536-a426-e5abe450ebb4",
   "metadata": {},
   "outputs": [],
   "source": [
    "Variant3_name_PNDM4=\"VAR_063725\"\n",
    "Variant3_position_PNDM4=32\n",
    "Variant3_change_PNDM4=\"G>R\"\n",
    "Variant3_description_PNDM4=\"Missense mutation, glycine to arginine\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "f63e2020-f4de-4f41-8aee-9d1a3cfff79c",
   "metadata": {},
   "outputs": [],
   "source": [
    "Variant4_name_PNDM4=\"VAR_063726\"\n",
    "Variant4_position_PNDM4=32\n",
    "Variant4_change_PNDM4=\"G>S\"\n",
    "Variant4_description_PNDM4=\"Missense mutation, glycine to serine\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "919286c4-5b38-4ff7-8925-d365ec878755",
   "metadata": {},
   "outputs": [],
   "source": [
    "Variant5_name_PNDM4=\"VAR_063727\"\n",
    "Variant5_position_PNDM4=35\n",
    "Variant5_change_PNDM4=\"L>P\"\n",
    "Variant5_description_PNDM4=\"leucine to proline leading to probably deleterious missense mutation\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "d855bd8a-4bfa-459e-8577-b4ca6a5a24ce",
   "metadata": {},
   "outputs": [],
   "source": [
    "PNDM4[\"Variant 1 PNDM4\"]=Variant1_name_PNDM4\n",
    "PNDM4[\"Variant 1 position PNDM4\"]=Variant1_position_PNDM4\n",
    "PNDM4[\"Variant 1 position change PNDM4\"]=Variant1_change_PNDM4\n",
    "PNDM4[\"Variant 1 position change description PNDM4\"]=Variant1_description_PNDM4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "ba61892e-d706-43c2-8743-6329b794175b",
   "metadata": {},
   "outputs": [],
   "source": [
    "PNDM4[\"Variant 2 PNDM4\"]=Variant2_name_PNDM4\n",
    "PNDM4[\"Variant 2 position PNDM4\"]=Variant2_position_PNDM4\n",
    "PNDM4[\"Variant 2 position change PNDM4\"]=Variant2_change_PNDM4\n",
    "PNDM4[\"Variant 2 position change description PNDM4\"]=Variant2_description_PNDM4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "8f28cb95-801e-42c2-87b2-961b775b21e8",
   "metadata": {},
   "outputs": [],
   "source": [
    "PNDM4[\"Variant 3 PNDM4\"]=Variant3_name_PNDM4\n",
    "PNDM4[\"Variant 3 position PNDM4\"]=Variant3_position_PNDM4\n",
    "PNDM4[\"Variant 3 position change PNDM4\"]=Variant3_change_PNDM4\n",
    "PNDM4[\"Variant 3 position change description PNDM4\"]=Variant3_description_PNDM4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "603a55ec-e25c-4e1c-919f-a42752d6a3d0",
   "metadata": {},
   "outputs": [],
   "source": [
    "PNDM4[\"Variant 4 PNDM4\"]=Variant4_name_PNDM4\n",
    "PNDM4[\"Variant 4 position PNDM4\"]=Variant4_position_PNDM4\n",
    "PNDM4[\"Variant 4 position change PNDM4\"]=Variant4_change_PNDM4\n",
    "PNDM4[\"Variant 4 position change description PNDM4\"]=Variant4_description_PNDM4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "e57f01aa-bcc6-4cd1-8678-cd1f12291292",
   "metadata": {},
   "outputs": [],
   "source": [
    "PNDM4[\"Variant 5 PNDM4\"]=Variant5_name_PNDM4\n",
    "PNDM4[\"Variant 5 position PNDM4\"]=Variant5_position_PNDM4\n",
    "PNDM4[\"Variant 5 position change PNDM4\"]=Variant5_change_PNDM4\n",
    "PNDM4[\"Variant 5 position change description PNDM4\"]=Variant5_description_PNDM4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "3be4ef21-3c22-4529-963e-df52a3749b5f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Variant 3 PNDM4': 'VAR_063725', 'Variant 3 position PNDM4': 32, 'Variant 3 position change PNDM4': 'G>R', 'Variant 3 position change description PNDM4': 'Missense mutation, glycine to arginine', 'Variant 4 PNDM4': 'VAR_063726', 'Variant 4 position PNDM4': 32, 'Variant 4 position change PNDM4': 'G>S', 'Variant 4 position change description PNDM4': 'Missense mutation, glycine to serine', 'Variant 5 PNDM4': 'VAR_063727', 'Variant 5 position PNDM4': 35, 'Variant 5 position change PNDM4': 'L>P', 'Variant 5 position change description PNDM4': 'leucine to proline leading to probably deleterious missense mutation', 'Variant 1 PNDM4': 'VAR_063723', 'Variant 1 position PNDM4': 24, 'Variant 1 position change PNDM4': 'A>D', 'Variant 1 position change description PNDM4': 'Missense mutation, alanine to aspartic acid', 'Variant 2 PNDM4': 'VAR_063724', 'Variant 2 position PNDM4': 29, 'Variant 2 position change PNDM4': 'H>D', 'Variant 2 position change description PNDM4': 'Missense mutation, Histidine to aspartic acid'}\n"
     ]
    }
   ],
   "source": [
    "print(PNDM4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "2f6fd2a4-8961-4425-b01e-bac0db416fce",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "# let's merge the two dictionaries together using {**dict1, **dict2}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "eb9910df-1812-4ece-937e-c9b7106cd5c2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Variant 1 HPRI': 'VAR_003971', 'Variant 1 position HPRI': 34, 'Variant 1 position change HPRI': 'H>D', 'Variant 1 position change description HPRI': 'A probably deleterious missense mutation--Amino acid change from histidine to aspartic acid', 'Variant 2 HPRI': 'VAR_003972', 'Variant 2 position HPRI': 48, 'Variant 2 position change HPRI': 'F>S', 'Variant 2 position change description HPRI': 'Amino acid change from phenylalanine to serine at position 48', 'Variant 3 HPRI': 'VAR_003974', 'Variant 3 position HPRI': 89, 'Variant 3 position change HPRI': 'R>H', 'Variant 3 position change description HPRI': 'Missense mutation, probably deleterious, from arginine to histidine, position 89', 'Variant 4 HPRI': 'VAR_003975', 'Variant 4 position HPRI': 89, 'Variant 4 position change HPRI': 'R>L', 'Variant 4 position change description HPRI': 'Missense mutation, amino acid change arginine to leucine', 'Variant 3 PNDM4': 'VAR_063725', 'Variant 3 position PNDM4': 32, 'Variant 3 position change PNDM4': 'G>R', 'Variant 3 position change description PNDM4': 'Missense mutation, glycine to arginine', 'Variant 4 PNDM4': 'VAR_063726', 'Variant 4 position PNDM4': 32, 'Variant 4 position change PNDM4': 'G>S', 'Variant 4 position change description PNDM4': 'Missense mutation, glycine to serine', 'Variant 5 PNDM4': 'VAR_063727', 'Variant 5 position PNDM4': 35, 'Variant 5 position change PNDM4': 'L>P', 'Variant 5 position change description PNDM4': 'leucine to proline leading to probably deleterious missense mutation', 'Variant 1 PNDM4': 'VAR_063723', 'Variant 1 position PNDM4': 24, 'Variant 1 position change PNDM4': 'A>D', 'Variant 1 position change description PNDM4': 'Missense mutation, alanine to aspartic acid', 'Variant 2 PNDM4': 'VAR_063724', 'Variant 2 position PNDM4': 29, 'Variant 2 position change PNDM4': 'H>D', 'Variant 2 position change description PNDM4': 'Missense mutation, Histidine to aspartic acid'}\n"
     ]
    }
   ],
   "source": [
    "INS_disease={**HPRI, **PNDM4}\n",
    "print(INS_disease)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "id": "c97a06fa-6ce4-478e-adec-41c5eaf61f1d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'L>P'"
      ]
     },
     "execution_count": 63,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#search could be applied with the new dict INS_disease \n",
    "INS_disease[\"Variant 5 position change PNDM4\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "id": "508306b6-f57e-4170-a6b0-65017c118add",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Variant 4 causing PNDM4 position change description: Missense mutation, glycine to serine\n"
     ]
    }
   ],
   "source": [
    "print('Variant 4 causing PNDM4 position change description:', INS_disease[\"Variant 4 position change description PNDM4\"]) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "id": "0a3cbd2b-f038-47cb-b6ce-e6a5cd5f6dc1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Variant 1 HPRI', 'Variant 1 position HPRI', 'Variant 1 position change HPRI', 'Variant 1 position change description HPRI', 'Variant 2 HPRI', 'Variant 2 position HPRI', 'Variant 2 position change HPRI', 'Variant 2 position change description HPRI', 'Variant 3 HPRI', 'Variant 3 position HPRI', 'Variant 3 position change HPRI', 'Variant 3 position change description HPRI', 'Variant 4 HPRI', 'Variant 4 position HPRI', 'Variant 4 position change HPRI', 'Variant 4 position change description HPRI', 'Variant 3 PNDM4', 'Variant 3 position PNDM4', 'Variant 3 position change PNDM4', 'Variant 3 position change description PNDM4', 'Variant 4 PNDM4', 'Variant 4 position PNDM4', 'Variant 4 position change PNDM4', 'Variant 4 position change description PNDM4', 'Variant 5 PNDM4', 'Variant 5 position PNDM4', 'Variant 5 position change PNDM4', 'Variant 5 position change description PNDM4', 'Variant 1 PNDM4', 'Variant 1 position PNDM4', 'Variant 1 position change PNDM4', 'Variant 1 position change description PNDM4', 'Variant 2 PNDM4', 'Variant 2 position PNDM4', 'Variant 2 position change PNDM4', 'Variant 2 position change description PNDM4']\n"
     ]
    }
   ],
   "source": [
    "#List can be applied to dictionary by using the list() function\n",
    "#print out the keys in the dict. The curly braces would change to square bracket because the square bracket [] is used to create list in python\n",
    "INS_disease_Keys_List=list(INS_disease.keys())\n",
    "print(INS_disease_Keys_List)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "id": "c700e172-80a8-40b0-b2d1-61c3f064cfe0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Variant 1 HPRI',\n",
       " 'Variant 1 position HPRI',\n",
       " 'Variant 1 position change HPRI',\n",
       " 'Variant 1 position change description HPRI',\n",
       " 'Variant 2 HPRI',\n",
       " 'Variant 2 position HPRI',\n",
       " 'Variant 2 position change HPRI',\n",
       " 'Variant 2 position change description HPRI',\n",
       " 'Variant 3 HPRI',\n",
       " 'Variant 3 position HPRI',\n",
       " 'Variant 3 position change HPRI',\n",
       " 'Variant 3 position change description HPRI',\n",
       " 'Variant 4 HPRI',\n",
       " 'Variant 4 position HPRI',\n",
       " 'Variant 4 position change HPRI',\n",
       " 'Variant 4 position change description HPRI',\n",
       " 'Variant 3 PNDM4',\n",
       " 'Variant 3 position PNDM4',\n",
       " 'Variant 3 position change PNDM4',\n",
       " 'Variant 3 position change description PNDM4',\n",
       " 'Variant 4 PNDM4',\n",
       " 'Variant 4 position PNDM4',\n",
       " 'Variant 4 position change PNDM4',\n",
       " 'Variant 4 position change description PNDM4',\n",
       " 'Variant 5 PNDM4',\n",
       " 'Variant 5 position PNDM4',\n",
       " 'Variant 5 position change PNDM4',\n",
       " 'Variant 5 position change description PNDM4',\n",
       " 'Variant 1 PNDM4',\n",
       " 'Variant 1 position PNDM4',\n",
       " 'Variant 1 position change PNDM4',\n",
       " 'Variant 1 position change description PNDM4',\n",
       " 'Variant 2 PNDM4',\n",
       " 'Variant 2 position PNDM4',\n",
       " 'Variant 2 position change PNDM4',\n",
       " 'Variant 2 position change description PNDM4']"
      ]
     },
     "execution_count": 84,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "INS_disease_Keys_List[0:36]\n",
    "#index o as the first string to index 36, the last "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "id": "55411fe7-9cf3-40c9-aca2-cfd7fc2fe636",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Variant 1 HPRI']"
      ]
     },
     "execution_count": 77,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# negative indexes can be printed also\n",
    "INS_disease_Keys_List[-36]\n",
    "#-36 being first string and -1 the last string"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "61ebfdd2-83ef-4d0c-9d13-5ebfc3fd4dd3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Variant 2 position change description PNDM4'"
      ]
     },
     "execution_count": 79,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "INS_disease_Keys_List[-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "id": "906ab0fc-858f-4b1d-a459-3053705c005b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Variant 1 HPRI',\n",
       " 'Variant 1 position HPRI',\n",
       " 'Variant 1 position change HPRI',\n",
       " 'Variant 1 position change description HPRI',\n",
       " 'Variant 2 HPRI',\n",
       " 'Variant 2 position HPRI',\n",
       " 'Variant 2 position change HPRI',\n",
       " 'Variant 2 position change description HPRI',\n",
       " 'Variant 3 HPRI',\n",
       " 'Variant 3 position HPRI',\n",
       " 'Variant 3 position change HPRI',\n",
       " 'Variant 3 position change description HPRI',\n",
       " 'Variant 4 HPRI',\n",
       " 'Variant 4 position HPRI',\n",
       " 'Variant 4 position change HPRI',\n",
       " 'Variant 4 position change description HPRI',\n",
       " 'Variant 3 PNDM4',\n",
       " 'Variant 3 position PNDM4',\n",
       " 'Variant 3 position change PNDM4',\n",
       " 'Variant 3 position change description PNDM4',\n",
       " 'Variant 4 PNDM4',\n",
       " 'Variant 4 position PNDM4',\n",
       " 'Variant 4 position change PNDM4',\n",
       " 'Variant 4 position change description PNDM4',\n",
       " 'Variant 5 PNDM4',\n",
       " 'Variant 5 position PNDM4',\n",
       " 'Variant 5 position change PNDM4',\n",
       " 'Variant 5 position change description PNDM4',\n",
       " 'Variant 1 PNDM4',\n",
       " 'Variant 1 position PNDM4',\n",
       " 'Variant 1 position change PNDM4',\n",
       " 'Variant 1 position change description PNDM4',\n",
       " 'Variant 2 PNDM4',\n",
       " 'Variant 2 position PNDM4',\n",
       " 'Variant 2 position change PNDM4']"
      ]
     },
     "execution_count": 80,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "INS_disease_Keys_List[-36:-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "id": "aaa1ef4d-9a31-4020-b9a2-47002478c411",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "same element using +ve and -ve indexing: \n",
      " positive index: Variant 2 position change description PNDM4 \n",
      " negative index: Variant 2 position change description PNDM4\n"
     ]
    }
   ],
   "source": [
    "#print out both positive and negative indexing\n",
    "print(\"same element using +ve and -ve indexing:\", \"\\n positive index:\", INS_disease_Keys_List[35], '\\n negative index:', INS_disease_Keys_List[-1]) "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "73f5f6a9-647a-4638-bb67-60f74fd7cdfb",
   "metadata": {},
   "source": [
    "#### Maturity-onset diabetes of the young 10 (MODY10)\n",
    "-----------\n",
    "*An autosomal dominant mode of inheritance diabetes occur in onset in childhood or early adulthood-__usually before 25 years of age__, a primary defect in insulin secretion and frequent insulin-independence at the beginning of the disease.*\n",
    "|VariantID|Position|Change|Desription|\n",
    "|-------|------|-----|-------|\n",
    "|VAR_063721|6|R>C|argine to cysteine, missense mutation|\n",
    "|VAR_063722|6|R>H|arginine to histidine, missense mutation|\n",
    "|VAR_063729|46|R>Q|arginine to glutamine, missense mutation|"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4eb32ee5-5a6c-4aba-b19d-2a621e5f3f54",
   "metadata": {},
   "source": [
    "###### Tuple Explanation with MODY10"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "id": "69c10f3a-643b-4c7c-92f1-847e60974301",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('VAR_063721', 6, 6.0, 'R>C', 'missense mutation')\n"
     ]
    }
   ],
   "source": [
    "# Here, a tuple would be created from the table\n",
    "MODY10 = ('VAR_063721', 6, 6.0, 'R>C', 'missense mutation')\n",
    "#This is a simple tuple containing strings, a float and an integer.\n",
    "print(MODY10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "id": "e92f0bd5-aae9-4bd2-be51-522656c2ebc8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "5"
      ]
     },
     "execution_count": 94,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# To print the length of the tuple, use len() function\n",
    "len(MODY10)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "id": "edffcecc-c915-4c58-ae6c-c60ff6b81a7e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "VAR_063721\n",
      "6\n",
      "6.0\n",
      "R>C\n",
      "missense mutation\n"
     ]
    }
   ],
   "source": [
    "# print out the elements using indexing\n",
    "print(MODY10[0])\n",
    "print(MODY10[1])\n",
    "print(MODY10[2])\n",
    "print(MODY10[3])\n",
    "print(MODY10[4])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "id": "1c6826c8-c242-4789-bdcb-3683499724db",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "missense mutation\n",
      "R>C\n",
      "6.0\n",
      "6\n",
      "VAR_063721\n"
     ]
    }
   ],
   "source": [
    "# let's try the above using negative indexing\n",
    "print(MODY10[-1])\n",
    "print(MODY10[-2])\n",
    "print(MODY10[-3])\n",
    "print(MODY10[-4])\n",
    "print(MODY10[-5])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "id": "06bc7a15-fe9f-4fc4-b200-f17c98a395f7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'str'>\n",
      "<class 'int'>\n",
      "<class 'float'>\n",
      "<class 'str'>\n",
      "<class 'str'>\n"
     ]
    }
   ],
   "source": [
    "# To check the types of elements, use type() function\n",
    "print(type(MODY10[0]))\n",
    "print(type(MODY10[1]))\n",
    "print(type(MODY10[2]))\n",
    "print(type(MODY10[3]))\n",
    "print(type(MODY10[4]))\n",
    "# The type of each element in the tuple would be printed.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "id": "8e922ea0-96de-4fca-851d-ed369ba9233d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('VAR_063721', 6, 6.0, 'R>C', 'missense mutation', 'arginine to cysteine', 6)\n"
     ]
    }
   ],
   "source": [
    "# Concatenation adds new tuple using '+' sign\n",
    "# Create a new tuple MODY10_new containing MODY10 and a new tuple: (\"arginine to cysteine\", 6)\n",
    "MODY10_new = MODY10 + (\"arginine to cysteine\", 6)\n",
    "print(MODY10_new)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c1d64020-e227-46d6-983a-58849724a0bd",
   "metadata": {},
   "source": [
    "###### Nested Tuple"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "id": "92d7374b-df2d-438f-8f09-846f497bff2b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('VAR_063721', 6, 46, (6, 6.0), ('R>C', 46), ('missense mutation', ('R>Q', 6), ('R>H', 'VAR_063729')))\n"
     ]
    }
   ],
   "source": [
    "# A nested tuple contains another tuples and complex data)\n",
    "# a nested tuple MODY10_nested would be created from the data in the table above (MODY10)\n",
    "MODY10_nested = (\"VAR_063721\", 6, 46, (6, 6.0), ('R>C', 46), (\"missense mutation\", (\"R>Q\", 6), (\"R>H\", \"VAR_063729\")))\n",
    "print(MODY10_nested)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "id": "5e75b05b-0040-457a-8f71-76d08eaf33d8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "6\n"
     ]
    }
   ],
   "source": [
    "# To check the number of elements, use the len()\n",
    "print(len(MODY10_nested))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "id": "2c6446b6-1d43-49e3-8889-e7283606ec99",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "element 0 of MODY10_nested: VAR_063721\n",
      "element 1 of MODY10_nested: 6\n",
      "element 2 of MODY10_nested: 46\n",
      "element 3 of MODY10_nested: (6, 6.0)\n",
      "element 4 of MODY10_nested: ('R>C', 46)\n",
      "element 5 of MODY10_nested: ('missense mutation', ('R>Q', 6), ('R>H', 'VAR_063729'))\n"
     ]
    }
   ],
   "source": [
    "# print out the elements using indexing. \n",
    "print(\"element 0 of MODY10_nested:\", MODY10_nested[0])\n",
    "print(\"element 1 of MODY10_nested:\", MODY10_nested[1])\n",
    "print(\"element 2 of MODY10_nested:\", MODY10_nested[2])\n",
    "print(\"element 3 of MODY10_nested:\", MODY10_nested[3])\n",
    "print(\"element 4 of MODY10_nested:\", MODY10_nested[4])\n",
    "print(\"element 5 of MODY10_nested:\", MODY10_nested[5])\n",
    "# Index 0 being the first element and 5, the last element in the nested tuple\n",
    "# The nested tuple is complex containing strings, integers,tuples and complex tuple"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "id": "d5347084-633a-4c57-a8f9-2bda29f3935c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'str'>\n",
      "<class 'int'>\n",
      "<class 'int'>\n",
      "<class 'tuple'>\n",
      "<class 'tuple'>\n",
      "<class 'tuple'>\n"
     ]
    }
   ],
   "source": [
    "# let's print out the type of the elements\n",
    "\n",
    "print(type(MODY10_nested[0]))\n",
    "print(type(MODY10_nested[1]))\n",
    "print(type(MODY10_nested[2]))\n",
    "print(type(MODY10_nested[3]))\n",
    "print(type(MODY10_nested[4]))\n",
    "print(type(MODY10_nested[5]))\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "id": "a8a3b8b2-e249-4d92-9944-f3d3edf5018c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "element 3, 0 of MODY10_nested: 6\n",
      "element 3, 1 of MODY10_nested: 6.0\n",
      "element 4, 0 of MODY10_nested: R>C\n",
      "element 4, 1 of MODY10_nested: 46\n",
      "element 5, 0 of MODY10_nested: missense mutation\n",
      "element 5, 1 of MODY10_nested: ('R>Q', 6)\n",
      "element 5, 2 of MODY10_nested: ('R>H', 'VAR_063729')\n"
     ]
    }
   ],
   "source": [
    "# The elements in the 'class tuple' of MODY10_nested could be fetched out \n",
    "# index 3 to 5 are tuples. Let's fetch out the elements. \n",
    "print(\"element 3, 0 of MODY10_nested:\", MODY10_nested[3][0])\n",
    "print(\"element 3, 1 of MODY10_nested:\", MODY10_nested[3][1])\n",
    "print(\"element 4, 0 of MODY10_nested:\", MODY10_nested[4][0])\n",
    "print(\"element 4, 1 of MODY10_nested:\", MODY10_nested[4][1])\n",
    "print(\"element 5, 0 of MODY10_nested:\", MODY10_nested[5][0])\n",
    "print(\"element 5, 1 of MODY10_nested:\", MODY10_nested[5][1])\n",
    "print(\"element 5, 2 of MODY10_nested:\", MODY10_nested[5][2])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "id": "996c8030-f4c6-4103-9e2c-a5b835516413",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "element 5, 1, 0 of MODY10_nested: R>Q\n",
      "element 5, 1, 1 of MODY10_nested: 6\n",
      "element 5, 2, 0 of MODY10_nested: R>H\n",
      "element 5, 2, 1 of MODY10_nested: VAR_063729\n"
     ]
    }
   ],
   "source": [
    "# From the output above, index 5,1 and 5,2 are tuples. So let's fetch out the elements further\n",
    "print(\"element 5, 1, 0 of MODY10_nested:\", MODY10_nested[5][1][0])\n",
    "print(\"element 5, 1, 1 of MODY10_nested:\", MODY10_nested[5][1][1])\n",
    "print(\"element 5, 2, 0 of MODY10_nested:\", MODY10_nested[5][2][0])\n",
    "print(\"element 5, 2, 1 of MODY10_nested:\", MODY10_nested[5][2][1])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b8a60693-041b-437d-b815-dff72eb97a39",
   "metadata": {},
   "source": [
    "##### _Other molecular functions of Insulin_:\n",
    "+ carbohydrate derivative binding\n",
    "+ metal ion binding\n",
    "+ small molecule binding\n",
    "+ ~~other molecular function~~\n",
    "\n",
    " _unordered list and cross-through line_ created"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b82db252-fbc0-4aed-a905-da2533e98416",
   "metadata": {},
   "source": [
    "##### _Other biological process:_\n",
    "1. regulation of protein localization\n",
    "2. regulation of protein secretion\n",
    "3. vasodilation\n",
    "4. wound healing\n",
    "5. positive regulation of peptide hormone secretion\n",
    "\n",
    "*ordered list*\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8cc1d206-e2ce-42de-81c1-4687dc1941c4",
   "metadata": {},
   "source": [
    "## Author\n",
    "\n",
    "**Oluwafemi, AWE**\n",
    "\n",
    "  _Data obtained from Uniprot and molecular structure from PDB_"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python [conda env:anacondaa3]",
   "language": "python",
   "name": "conda-env-anacondaa3-py"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
